{
 "cells": [
  {
   "metadata": {},
   "cell_type": "raw",
   "source": [
    "Examples from\n",
    "\n",
    "https://wintoncentre.maths.cam.ac.uk/projects/communicating-results-genetic-testing/"
   ],
   "id": "a7a3ef4af11cfde6"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:09.228356Z",
     "start_time": "2025-06-07T05:57:03.187183Z"
    }
   },
   "cell_type": "code",
   "source": [
    "import tabula\n",
    "\n",
    "\n",
    "pdf_file = \"PDF/d_pos_v6.pdf\"\n",
    "#pdf_file = \"PDF/d_neg_v6.pdf\"\n",
    "dfs = tabula.read_pdf(pdf_file, pages=\"all\", stream=True, encoding='cp1252')\n",
    "tabula.environment_info()\n",
    "print(len(dfs))"
   ],
   "id": "71e66c63cec6b377",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Python version:\n",
      "    3.13.3 (v3.13.3:6280bb54784, Apr  8 2025, 10:47:54) [Clang 15.0.0 (clang-1500.3.9.4)]\n",
      "Java version:\n",
      "    openjdk version \"20.0.2\" 2023-07-18\n",
      "OpenJDK Runtime Environment (build 20.0.2+9-78)\n",
      "OpenJDK 64-Bit Server VM (build 20.0.2+9-78, mixed mode, sharing)\n",
      "tabula-py version: 2.10.0\n",
      "platform: macOS-15.5-x86_64-i386-64bit-Mach-O\n",
      "uname:\n",
      "    uname_result(system='Darwin', node='Freya.local', release='24.5.0', version='Darwin Kernel Version 24.5.0: Tue Apr 22 19:53:26 PDT 2025; root:xnu-11417.121.6~2/RELEASE_X86_64', machine='x86_64')\n",
      "linux_distribution: ('Darwin', '24.5.0', '')\n",
      "mac_ver: ('15.5', ('', '', ''), 'x86_64')\n",
      "3\n"
     ]
    }
   ],
   "execution_count": 19
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:09.267873Z",
     "start_time": "2025-06-07T05:57:09.263879Z"
    }
   },
   "cell_type": "code",
   "source": "print(len(dfs))",
   "id": "8d2c8fcbad3f6ea",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n"
     ]
    }
   ],
   "execution_count": 20
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:09.344684Z",
     "start_time": "2025-06-07T05:57:09.333652Z"
    }
   },
   "cell_type": "code",
   "source": "dfs[0]",
   "id": "1311cc8196b21d49",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "           N ame:       Jane Doe          Name:       Dr Requesta  \\\n",
       "0  Date of birth:  18 March 1970  Organisation:  Chester Hospital   \n",
       "1            Sex:         Female     Telephone:      01223 555555   \n",
       "2     NHS number:        NH00198     Copies to:      Dr A. Nother   \n",
       "3    Sample type:          Blood            NaN         Dr X. Tra   \n",
       "\n",
       "     Laborat o ry:        Gentest UK  \n",
       "0       Telephone:      01223 666555  \n",
       "1  Date receive d:  26 February 2018  \n",
       "2  Date reported :     12 March 2018  \n",
       "3  Authorised  by:           ATester  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>N ame:</th>\n",
       "      <th>Jane Doe</th>\n",
       "      <th>Name:</th>\n",
       "      <th>Dr Requesta</th>\n",
       "      <th>Laborat o ry:</th>\n",
       "      <th>Gentest UK</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Date of birth:</td>\n",
       "      <td>18 March 1970</td>\n",
       "      <td>Organisation:</td>\n",
       "      <td>Chester Hospital</td>\n",
       "      <td>Telephone:</td>\n",
       "      <td>01223 666555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Sex:</td>\n",
       "      <td>Female</td>\n",
       "      <td>Telephone:</td>\n",
       "      <td>01223 555555</td>\n",
       "      <td>Date receive d:</td>\n",
       "      <td>26 February 2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NHS number:</td>\n",
       "      <td>NH00198</td>\n",
       "      <td>Copies to:</td>\n",
       "      <td>Dr A. Nother</td>\n",
       "      <td>Date reported :</td>\n",
       "      <td>12 March 2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Sample type:</td>\n",
       "      <td>Blood</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dr X. Tra</td>\n",
       "      <td>Authorised  by:</td>\n",
       "      <td>ATester</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 21
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:09.441126Z",
     "start_time": "2025-06-07T05:57:09.431157Z"
    }
   },
   "cell_type": "code",
   "source": "dfs[1]",
   "id": "8e87725681d486e1",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "           N ame:       Jane Doe          Name:       Dr Requesta  \\\n",
       "0  Date of birth:  18 March 1970  Organisation:  Chester Hospital   \n",
       "1            Sex:         Female     Telephone:      01223 555555   \n",
       "2     NHS number:        NH00198     Copies to:      Dr A. Nother   \n",
       "3    Sample type:          Blood            NaN         Dr X. Tra   \n",
       "\n",
       "     Laborat o ry:        Gentest UK  \n",
       "0       Telephone:      01223 666555  \n",
       "1  Date receive d:  26 February 2018  \n",
       "2  Date reported :     12 March 2018  \n",
       "3  Authorised  by:           ATester  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>N ame:</th>\n",
       "      <th>Jane Doe</th>\n",
       "      <th>Name:</th>\n",
       "      <th>Dr Requesta</th>\n",
       "      <th>Laborat o ry:</th>\n",
       "      <th>Gentest UK</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Date of birth:</td>\n",
       "      <td>18 March 1970</td>\n",
       "      <td>Organisation:</td>\n",
       "      <td>Chester Hospital</td>\n",
       "      <td>Telephone:</td>\n",
       "      <td>01223 666555</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Sex:</td>\n",
       "      <td>Female</td>\n",
       "      <td>Telephone:</td>\n",
       "      <td>01223 555555</td>\n",
       "      <td>Date receive d:</td>\n",
       "      <td>26 February 2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NHS number:</td>\n",
       "      <td>NH00198</td>\n",
       "      <td>Copies to:</td>\n",
       "      <td>Dr A. Nother</td>\n",
       "      <td>Date reported :</td>\n",
       "      <td>12 March 2018</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Sample type:</td>\n",
       "      <td>Blood</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dr X. Tra</td>\n",
       "      <td>Authorised  by:</td>\n",
       "      <td>ATester</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 22
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:09.557048Z",
     "start_time": "2025-06-07T05:57:09.546317Z"
    }
   },
   "cell_type": "code",
   "source": "dfs[2]",
   "id": "608b70d0414140d6",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "    Gene          Zygosity Inheritance                  HGVS Description  \\\n",
       "0  BRCA1      Heterozygous   Autosomal  c.3143delG p.(Gly1048ValfsTer14)   \n",
       "1    NaN               NaN    dominant                               NaN   \n",
       "2  BRCA2     No clinically         NaN                               NaN   \n",
       "3    NaN        pathogenic         NaN                               NaN   \n",
       "4    NaN  variant detected         NaN                               NaN   \n",
       "5  PALB2     No clinically         NaN                               NaN   \n",
       "6    NaN        pathogenic         NaN                               NaN   \n",
       "7    NaN  variant detected         NaN                               NaN   \n",
       "\n",
       "  Location: GRCh38 (hg38) *Classification  \n",
       "0   17: 43092388 (GRCh38)      Pathogenic  \n",
       "1                     NaN             NaN  \n",
       "2                     NaN             Not  \n",
       "3                     NaN      applicable  \n",
       "4                     NaN             NaN  \n",
       "5                     NaN             Not  \n",
       "6                     NaN      applicable  \n",
       "7                     NaN             NaN  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Zygosity</th>\n",
       "      <th>Inheritance</th>\n",
       "      <th>HGVS Description</th>\n",
       "      <th>Location: GRCh38 (hg38)</th>\n",
       "      <th>*Classification</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>BRCA1</td>\n",
       "      <td>Heterozygous</td>\n",
       "      <td>Autosomal</td>\n",
       "      <td>c.3143delG p.(Gly1048ValfsTer14)</td>\n",
       "      <td>17: 43092388 (GRCh38)</td>\n",
       "      <td>Pathogenic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>dominant</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>BRCA2</td>\n",
       "      <td>No clinically</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Not</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>applicable</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>variant detected</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>PALB2</td>\n",
       "      <td>No clinically</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Not</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NaN</td>\n",
       "      <td>pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>applicable</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>variant detected</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 23
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "Based on https://build.fhir.org/ig/HL7/genomics-reporting/\n",
    "\n",
    "I think we can build a variant http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
   ],
   "id": "4a767d7f9735a06e"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:10.730092Z",
     "start_time": "2025-06-07T05:57:09.712201Z"
    }
   },
   "cell_type": "code",
   "source": [
    "from fhir.resources.observation import Observation,ObservationComponent\n",
    "from fhir.resources.codeableconcept import CodeableConcept\n",
    "from fhir.resources.coding import Coding\n",
    "\n",
    "df = dfs[len(dfs)-1]\n",
    "df['Gene'] = df['Gene'].astype(str)\n",
    "df['HGVS Description'] = df['HGVS Description'].astype(str)\n",
    "df.dtypes"
   ],
   "id": "33ecc03efbb6769",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Gene                       object\n",
       "Zygosity                   object\n",
       "Inheritance                object\n",
       "HGVS Description           object\n",
       "Location: GRCh38 (hg38)    object\n",
       "*Classification            object\n",
       "dtype: object"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 24
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:10.835983Z",
     "start_time": "2025-06-07T05:57:10.792712Z"
    }
   },
   "cell_type": "code",
   "source": [
    "\n",
    "\n",
    "for i in range(0, len(df)):\n",
    "    if df.loc[i,'Gene'] is not None and df.loc[i,'Gene'] != 'nan':\n",
    "        print('\\n',df.loc[i,'Gene'],'\\n')\n",
    "        observationBase = {\n",
    "            \"status\" : \"final\",\n",
    "            \"category\":[\n",
    "                {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://terminology.hl7.org/CodeSystem/observation-category\",\n",
    "                            \"code\" : \"laboratory\"\n",
    "                        }\n",
    "                    ]\n",
    "                },\n",
    "                {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://terminology.hl7.org/CodeSystem/v2-0074\",\n",
    "                            \"code\" : \"GE\"\n",
    "                        }\n",
    "                    ]\n",
    "                }\n",
    "            ],\n",
    "            \"code\" : {\n",
    "                \"coding\" : [\n",
    "                    {\n",
    "                        \"system\" : \"http://loinc.org\",\n",
    "                        \"code\" : \"69548-6\",\n",
    "                        \"display\" : \"Genetic variant assessment\"\n",
    "                    }\n",
    "                ]\n",
    "            },\n",
    "            \"valueCodeableConcept\" : {\n",
    "                \"coding\" : [\n",
    "                    {\n",
    "                        \"system\" : \"http://loinc.org\",\n",
    "                        \"code\" : \"LA9633-4\",\n",
    "                        \"display\" : \"Present\"\n",
    "                    }\n",
    "                ]\n",
    "            },\n",
    "            \"subject\" : {\n",
    "                \"reference\" : \"urn:uuid:d6faafcf-db64-4c11-9da8-25f36774c1bd\",\n",
    "                \"identifier\" : {\n",
    "                    \"system\" : \"https://fhir.nhs.uk/Id/nhs-number\",\n",
    "                    \"value\" : \"9449305552\"\n",
    "                },\n",
    "                \"display\" : \"Octavia CHISLETT\"\n",
    "            },\n",
    "            \"method\" : {\n",
    "                \"coding\" : [\n",
    "                    {\n",
    "                        \"system\" : \"http://loinc.org\",\n",
    "                        \"code\" : \"LA26398-0\",\n",
    "                        \"display\" : \"Sequencing\"\n",
    "                    }\n",
    "                ]\n",
    "            },\n",
    "            \"component\" : []\n",
    "        }\n",
    "        obs = Observation(**observationBase)\n",
    "        genename = \"NK\"\n",
    "        if df.loc[i,'Gene'] == 'BRCA1':\n",
    "            genename = \"HGNC:1100\"\n",
    "        if df.loc[i,'Gene'] == 'BRCA2':\n",
    "            genename = \"HGNC:1101\"\n",
    "        if df.loc[i,'Gene'] == 'PALB2':\n",
    "            genename = \"HGNC:26144\"\n",
    "        geneData = {\n",
    "            \"code\" : {\n",
    "                \"coding\" : [\n",
    "                    {\n",
    "                        \"system\" : \"http://loinc.org\",\n",
    "                        \"code\" : \"48018-6\"\n",
    "                    }\n",
    "                ]\n",
    "            },\n",
    "            \"valueCodeableConcept\" : {\n",
    "                \"coding\" : [\n",
    "                    {\n",
    "                        \"system\" : \"http://www.genenames.org\",\n",
    "                        \"code\" : genename,\n",
    "                        \"display\" : df.loc[i,'Gene']\n",
    "                    }\n",
    "                ]\n",
    "            }\n",
    "        }\n",
    "        gene = ObservationComponent(**geneData)\n",
    "        obs.component.append(gene)\n",
    "\n",
    "        if df.loc[i,'Zygosity'] == 'Heterozygous':\n",
    "            alleData = {\n",
    "                \"code\" : {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://loinc.org\",\n",
    "                            \"code\" : \"53034-5\",\n",
    "                            \"display\" : \"Allelic state\"\n",
    "                        }\n",
    "                    ]\n",
    "                },\n",
    "                \"valueCodeableConcept\" : {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://loinc.org\",\n",
    "                            \"code\" : \"LA6706-1\",\n",
    "                            \"display\" : \"Heterozygous\"\n",
    "                        }\n",
    "                    ]\n",
    "                }\n",
    "            }\n",
    "            alle = ObservationComponent(**alleData)\n",
    "            obs.component.append(alle)\n",
    "        if df.loc[i,'HGVS Description'] is not None and df.loc[i,'HGVS Description'] != 'nan':\n",
    "            code = df.loc[i,'Gene'] + \":\" + df.loc[i,'HGVS Description']\n",
    "            hgvsData = {\n",
    "                \"code\" : {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://loinc.org\",\n",
    "                            \"code\" : \"48004-6\",\n",
    "                            \"display\": \"DNA change\"\n",
    "                        }\n",
    "                    ]\n",
    "                },\n",
    "                \"valueCodeableConcept\" : {\n",
    "                    \"coding\" : [\n",
    "                        {\n",
    "                            \"system\" : \"http://varnomen.hgvs.org\",\n",
    "                            \"code\" : code\n",
    "                        }\n",
    "                    ]\n",
    "                }\n",
    "            }\n",
    "            hgvs = ObservationComponent(**hgvsData)\n",
    "            obs.component.append(hgvs)\n",
    "\n",
    "        json_str = obs.model_dump_json()\n",
    "        print(json_str)"
   ],
   "id": "d3fe0e424488a4bb",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " BRCA1 \n",
      "\n",
      "{\"resourceType\":\"Observation\",\"status\":\"final\",\"category\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/observation-category\",\"code\":\"laboratory\"}]},{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0074\",\"code\":\"GE\"}]}],\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"69548-6\",\"display\":\"Genetic variant assessment\"}]},\"subject\":{\"reference\":\"urn:uuid:d6faafcf-db64-4c11-9da8-25f36774c1bd\",\"identifier\":{\"system\":\"https://fhir.nhs.uk/Id/nhs-number\",\"value\":\"9449305552\"},\"display\":\"Octavia CHISLETT\"},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA9633-4\",\"display\":\"Present\"}]},\"method\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA26398-0\",\"display\":\"Sequencing\"}]},\"component\":[{\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"48018-6\"}]},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://www.genenames.org\",\"code\":\"HGNC:1100\",\"display\":\"BRCA1\"}]}},{\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"53034-5\",\"display\":\"Allelic state\"}]},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA6706-1\",\"display\":\"Heterozygous\"}]}},{\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"48004-6\",\"display\":\"DNA change\"}]},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://varnomen.hgvs.org\",\"code\":\"BRCA1:c.3143delG p.(Gly1048ValfsTer14)\"}]}}]}\n",
      "\n",
      " BRCA2 \n",
      "\n",
      "{\"resourceType\":\"Observation\",\"status\":\"final\",\"category\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/observation-category\",\"code\":\"laboratory\"}]},{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0074\",\"code\":\"GE\"}]}],\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"69548-6\",\"display\":\"Genetic variant assessment\"}]},\"subject\":{\"reference\":\"urn:uuid:d6faafcf-db64-4c11-9da8-25f36774c1bd\",\"identifier\":{\"system\":\"https://fhir.nhs.uk/Id/nhs-number\",\"value\":\"9449305552\"},\"display\":\"Octavia CHISLETT\"},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA9633-4\",\"display\":\"Present\"}]},\"method\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA26398-0\",\"display\":\"Sequencing\"}]},\"component\":[{\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"48018-6\"}]},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://www.genenames.org\",\"code\":\"HGNC:1101\",\"display\":\"BRCA2\"}]}}]}\n",
      "\n",
      " PALB2 \n",
      "\n",
      "{\"resourceType\":\"Observation\",\"status\":\"final\",\"category\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/observation-category\",\"code\":\"laboratory\"}]},{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0074\",\"code\":\"GE\"}]}],\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"69548-6\",\"display\":\"Genetic variant assessment\"}]},\"subject\":{\"reference\":\"urn:uuid:d6faafcf-db64-4c11-9da8-25f36774c1bd\",\"identifier\":{\"system\":\"https://fhir.nhs.uk/Id/nhs-number\",\"value\":\"9449305552\"},\"display\":\"Octavia CHISLETT\"},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA9633-4\",\"display\":\"Present\"}]},\"method\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"LA26398-0\",\"display\":\"Sequencing\"}]},\"component\":[{\"code\":{\"coding\":[{\"system\":\"http://loinc.org\",\"code\":\"48018-6\"}]},\"valueCodeableConcept\":{\"coding\":[{\"system\":\"http://www.genenames.org\",\"code\":\"HGNC:26144\",\"display\":\"PALB2\"}]}}]}\n"
     ]
    }
   ],
   "execution_count": 25
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-07T05:57:10.936457Z",
     "start_time": "2025-06-07T05:57:10.932801Z"
    }
   },
   "cell_type": "code",
   "source": "",
   "id": "de107e867fa8d296",
   "outputs": [],
   "execution_count": null
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
