{
 "cells": [
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "# PyPDF2",
   "id": "108a82bf70ad30e7"
  },
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:20.300534Z",
     "start_time": "2025-06-05T13:12:20.260133Z"
    }
   },
   "source": [
    "from PyPDF2 import PdfFileReader, PdfReader\n",
    "\n",
    "#pdf_file = \"PDF/Genomics.pdf\"\n",
    "pdf_file = \"PDF/genome-report-example.pdf\"\n",
    "pdf_object = PdfReader(pdf_file)"
   ],
   "outputs": [],
   "execution_count": 19
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:20.350920Z",
     "start_time": "2025-06-05T13:12:20.336905Z"
    }
   },
   "cell_type": "code",
   "source": [
    "page_count = len(pdf_object.pages)\n",
    "# Access a specific page (index starts at 0)\n",
    "page = pdf_object.pages[0]"
   ],
   "id": "3199f4945da5d251",
   "outputs": [],
   "execution_count": 20
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## text",
   "id": "5c27c5db2b58f6a0"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:21.374423Z",
     "start_time": "2025-06-05T13:12:20.378906Z"
    }
   },
   "cell_type": "code",
   "source": [
    "all_text = \"\"\n",
    "for page in pdf_object.pages:\n",
    "    all_text += page.extract_text() + \"\\n\"\n",
    "print(all_text)"
   ],
   "id": "5d9b46f37ab05714",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "_______________________________ _____________________________________________________________________________________________________________________________ __________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine  CLIA#: 22D1005307Name:  LAST , FIRST  MRN: XXXXXXXX  LMM Accession ID: PM-23-NXXXX  \n",
      "DOB: MM /DD/YYYY Referring Facility:  HS Specimen: Blood, Peripheral\n",
      "Sex at birth : Female Referring Physician:  LAST , FIRST Received:   MM /DD/2023\n",
      "Family #:  FXXXXXX Copies To:  HOSPITAL  SYSTEM Page: 1 of 7\n",
      "Test Performed:  Expanded Genome Screening Test Codes:  lmWGS -pnlFv4_L\n",
      "VARIANT SUMMARY  \n",
      "REPORT SECTION  Disease,  \n",
      "Inheritance  Gene  \n",
      "Transcript  Variant  Allele State  Classification  \n",
      "A. MONOGENIC DISEASE FINDINGS None identified.  \n",
      "B. RISK ALLELES None identified.  \n",
      "C. CARRIER STATUS VARIANTSPrimary ciliary \n",
      "dyskinesia , \n",
      "Autosomal Recessive  CCDC40  \n",
      "NM_017950.4  c.248delC\n",
      "p.Ala83ValfsX84Heterozygous  Pathogenic  \n",
      "Primary ciliary \n",
      "dyskinesia , \n",
      "Autosomal Recessive  DNAH11  \n",
      "NM_001277115.2  c.12363C>G  \n",
      "p.Tyr4121XHeterozygous  Pathogenic  \n",
      "Short/branched chain \n",
      "acyl-CoA dehydrogenase \n",
      "deficiency , \n",
      "Autosomal Recessive  ACADSB  \n",
      "NM_001609.4  c.303+1G>A Heterozygous  Pathogenic  \n",
      "D.PHARMACOGENOMIC\n",
      "ASSOCIATIONSSee below.  \n",
      "DETAILED VARIANT INFORMATION \n",
      "A.MONOGENIC DISEASE FINDINGS\n",
      "This test did not identify any variants with the potential to cause monogenic disease in this individual.  \n",
      "B.RISK ALLELES\n",
      " This test did not identify any reportable risk alleles (see methodology for the list of risk alleles included in this analysi s). \n",
      "C.CARRIER STATUS VARIANTS\n",
      "This test identified 3 carrier status variants for autosomal recessive conditions. Autosomal recessive disorders are caused by the presence of \n",
      "pathogenic variants in both copies of the same gene. Being a carrier of these variants does not put this individual at risk for disease  but may \n",
      "impact disease risk in this individual’s children. PLEASE NOTE: The presence of a second pathogenic variant in one of these genes  cannot be \n",
      "definitively ruled out  due to the technical and analytical limitations of this assay.   \n",
      "FAMILIAL AND REPRODUCTIVE RISK:  The risk to this individual’s child or future child of developing an autosomal recessive condition  is \n",
      "dependent on the carrier status of th is individual's r eproductive partner(s). Two carriers have a 25% risk for having a child with the associated \n",
      "disease. First degree relatives of this individual have a 50% chance of  also being carriers of each of these variants . Other biologically related \n",
      "family members may  also be carriers. \n",
      "GENOME SCREENING RESULTS SUMMARY  \n",
      "NEGATIVE - NO MONOGENIC DISEASE FINDINGS . Sequencing of this individual’s genome identified  3 carrier status \n",
      "variants. Pharmacogenomic associations are also included in this report. Result details are listed below.  \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 2 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      " \n",
      "Disease,  \n",
      "Inheritance  Gene  \n",
      "Transcript  Variant  Allele State  \n",
      " Classification  \n",
      "Primary ciliary dyskinesia , \n",
      "Autosomal Recessive  CCDC40  \n",
      "NM_017950.4  c.248delC  \n",
      "p.Ala83ValfsX84  Heterozygous  Pathogenic  \n",
      "Genomic Coordinate  Location  Penetrance  Carrier Phenotype  Gene Coverage  \n",
      "g.80039966delC (chr17, GRCh38)  Exon 3 High  None reported  100% at 15X  \n",
      "VARIANT INTERPRETATION: The p.Ala83ValfsX84 variant in CCDC40  has been previously reported in 19 homozygous and 7 compound \n",
      "heterozygous individuals with primary ciliary dyskinesia (PCD) and segregated with disease in 1 homozygous affected relative (Becker -Heck 2011 \n",
      "PMID: 21131974, Nakhleh 2012 PMID: 22499950, Antony  2013 PMID: 23255504, Zariwala 2013 PMID: 23891469). This variant has been identified \n",
      "in 0.074% (860/1167354) of non -Finnish European chromosomes by gnomAD (http://gnomad.broadinstitute.org). However, this frequency is low \n",
      "enough to be consistent with a re cessive allele frequency. This variant has also been reported in ClinVar (Variation ID 31069). This variant is \n",
      "predicted to cause a frameshift, which alters the protein’s amino acid sequence beginning at position 83 and leads to a prema ture termination \n",
      "codon 84 amino acids downstream. This alteration is then predicted to lead to a truncated or absent protein.  Functional studies indicate that loss \n",
      "of CCDC40 function results in abnormal cilia structure and motility (Becker -Heck 2011 PMID: 21131974). In summar y, this variant meets criteria to \n",
      "be classified as pathogenic for autosomal recessive primary ciliary dyskinesia. ACMG/AMP Criteria applied: PVS1, PM3_VeryStro ng, \n",
      "PM2_Supporting, PP1.  \n",
      "DISEASE INFORMATION:  Primary ciliary dyskinesia is a rare genetic condi tion that is genetically heterogeneous. It is associated with recurrent \n",
      "respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities i n the motility of the cilia \n",
      "and flagella that are found in the linings of organs and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and \n",
      "chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror -image reversal of all visceral org ans, is \n",
      "found in 40 -50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females \n",
      "with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Oth er symptoms can include recurrent ear \n",
      "infections and hydrocephalus in the brain. Pathogenic variants in CCDC40  contribute to  3-4% of primary ciliary dyskinesia (Medline Plus: \n",
      "https://medlineplus.gov/genetics/condition/primary -ciliary -dyskinesia, GeneReview s: https://www.ncbi.nlm.nih.gov/books/NBK1122).  \n",
      "FAMILIAL AND REPRODUCTIVE RISK  \n",
      "Disease Prevalence (Estimated)  Carrier Frequency (Estimated)  Reproductive Risk (Estimated)  \n",
      "1/16000 (https://medlineplus.gov/genetics/condition/primary -\n",
      "ciliary -dyskinesia ) 1/317 to 1/366  (for CCDC40 ) 1/1268 to 1/1464  (for CCDC40 ) \n",
      " \n",
      " \n",
      "Disease,  \n",
      "Inheritance  Gene  \n",
      "Transcript  Variant  Allele State  \n",
      " Classification  \n",
      "Primary ciliary dyskinesia , \n",
      "Autosomal Recessive  DNAH11  \n",
      "NM_001277115.2  c.12363C>G  \n",
      "p.Tyr4121X  Heterozygous  Pathogenic  \n",
      "Genomic Coordinate  Location  Penetrance  Carrier Phenotype  Gene Coverage  \n",
      "g.21880869C>G (chr7, GRCh38)  Exon 75 High  None reported  100% at 15X  \n",
      "VARIANT INTERPRETATION: The p.Tyr4121X variant in DNAH11  has been reported in the compound heterozygous state in 1 individual with \n",
      "primary ciliary dyskinesia and segregated with disease in 5 affected siblings (Schwabe 2008 PMID: 18022865). It has been iden tified in 0.045% \n",
      "(29/63996) of European (Finnish) chromo somes by gnomAD (http://gnomad.broadinstitute.org, v4.0.0). However, this frequency is low enough to \n",
      "be consistent with the carrier frequency. This variant has also been reported in ClinVar (Variation ID 6475). This nonsense v ariant leads to a \n",
      "premature te rmination codon at position 4121, which is predicted to lead to a truncated or absent protein. Biallelic loss -of-function of the \n",
      "DNAH11 gene is an established disease mechanism in autosomal recessive primary ciliary dyskinesia 7. In summary, this variant  meets criteria to \n",
      "be classified as pathogenic for autosomal recessive primary ciliary dyskinesia 7. ACMG/AMP Criteria applied: PVS1, PP1_Strong , PM3, \n",
      "PM2_Supporting.  \n",
      "DISEASE INFORMATION:  Primary ciliary dyskinesia is a rare genetic condition that is geneti cally heterogeneous. It is associated with recurrent \n",
      "respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities i n the motility of the cilia \n",
      "and flagella that are found in the linings of orga ns and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and \n",
      "chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror -image reversal of all visceral organs, is \n",
      "found in 40 -50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females \n",
      "with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Other symptoms can inc lude recurrent ear \n",
      "infections and hydrocephalus in the brain. Pathogenic variants in DNAH11  contribute to  6-9% of primary ciliary dyskinesia (Medline Plus: \n",
      "https://medlineplus.gov/genetics/condition/primary -ciliary -dyskinesia ; GeneReviews: https://www.ncbi .nlm.nih.gov/books/NBK1122).  \n",
      "FAMILIAL AND REPRODUCTIVE RISK  \n",
      "Disease Prevalence (Estimated)  Carrier Frequency (Estimated)  Reproductive Risk (Estimated)  \n",
      "1/16000 (https://medlineplus.gov/genetics/condition/primary -\n",
      "ciliary -dyskinesia ) 1/211 to 1/259  (for DNAH11 ) 1/844 to 1/1036  (for DNAH11 ) \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 3 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      " \n",
      "Disease,  \n",
      "Inheritance  Gene  \n",
      "Transcript  Variant  Allele State  \n",
      " Classification  \n",
      "Short/branched chain acyl -CoA \n",
      "dehydrogenase deficiency , \n",
      "Autosomal Recessive  ACADSB  \n",
      "NM_001609.4  c.303+1G>A  Heterozygous  Pathogenic  \n",
      "Genomic Coordinate  Location  Penetrance  Carrier Phenotype  Gene Coverage  \n",
      "g.123037848G>A (chr10, GRCh38)  Intron 3 Reduced  None reported  100% at 15X  \n",
      "VARIANT INTERPRETATION: The c.303+1G>A variant in ACADSB  has been reported in the homozygous state in 2 individual with short /branched \n",
      "chain acyl -CoA dehydrogenase (SBCAD) deficiency; both showed the biochemical phenotype but only one showed clinical symptoms (Alfardan \n",
      "2010 PMID: 20547083). It has also been reported in the homozygous state in 2 siblings  without clinical symptoms  (Spedicati 2021 PMID: \n",
      "33727708). It has been identified in 0.056% (628/1127804) of European (non -Finnish) chromosome s, including 1 homozygote, by gnomAD \n",
      "(http://gnomad.broadinstitute.org, v4.0.0). This variant occurs within the canonical splice site (+/ - 1,2) and is predicted to cause altered splicing \n",
      "leading to an abnormal or absent protein. Computational prediction to ols and conservation analyses are consistent with pathogenicity. Biallelic \n",
      "loss-of-function of the ACADSB gene is an established disease mechanism in autosomal recessive SBCAD deficiency. SBCAD deficiency res ults in a \n",
      "biochemical phenotype that be detected  from birth; however, the clinical significance of this is unclear. 90% of individuals with SBCAD deficiency \n",
      "have no clinical symptoms , and the remaining 10% may display developmental delay and/or neurological disorders. These 10% of individuals may \n",
      "repres ent extreme end of the clinical spectrum or coincidental findings (Porta 2019 PMID: 30730842, Alfardan 2010 PMID: 20547083). In \n",
      "summary, this variant meets criteria to be classified as pathogenic for autosomal recessive SBCAD deficiency. ACMG/AMP Criter ia applied: PVS1, \n",
      "PM3_Supporting.  \n",
      "DISEASE INFORMATION:  Short/branched chain acyl -CoA dehydrogenase deficiency (SBCAD deficiency) is a rare inborn error of metabolism that is \n",
      "more common among Hmong population in Southeast Asia. Affected individuals can be de tected during newborn screening (NBS) but most have \n",
      "no clinical symptoms . 10% of SBCAD deficiency patients  have developmental delay and neurological disorders ; however, it is unclear if the SBCAD \n",
      "deficiency  is causative  (Porta 2019 PMID: 30730842, Medline Plus: https://medlineplus.gov/genetics/condition/short -branched -chain -acyl-coa-\n",
      "dehydrogenase -deficiency).  \n",
      "FAMILIAL AND REPRODUCTIVE RISK  \n",
      "Disease Prevalence (Estimated)  Carrier Frequency (Estimated)  Reproductive Risk (Estimated)  \n",
      "Unknown  Unknown  Unknown  \n",
      " \n",
      "D. PHARMACOGENOMIC ASSOCIATIONS  \n",
      "Detailed dosing instructions are not provided in the brief interpretation notes below. Extrinsic factors (e.g. diet, smoking status, co -administered \n",
      "medications , drug -drug interactions ) and intrinsic factors (e.g. sex, age, weight, renal or hepatic function) may affect drug response. A *1/*1 or \n",
      "wild type result is assigned if no variants that are included in the test are detected and does not exclude the possibility t hat an individual h as a \n",
      "different phenotype that may alter drug response.  Patients should not use the test results to stop or change any medication unless directed by a \n",
      "qualified clinician. Genetic information to guide prescribing decisions may be found in a drug's FDA -appro ved label . These labels are found at \n",
      "https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm  and a table of current PGx biomarkers at https://www.fda.gov/medical -\n",
      "devices/precision -medicine/table -pharmacogenetic -associations  and https://www.fda.gov/drugs/science -and-research -drugs/table -\n",
      "pharmacogenomic -biomarkers -drug -labeling . The Clinical Pharmacogenetics Implementation Consortium (CPIC®)  website may be consulted for \n",
      "their most current recommendations , at https://cpicpgx.org/guidelines/ ; however, these have NOT received FDA -approval. Always consult a \n",
      "clinician or clinical pharmacologist before changing drug dosage or for additional information. The Pharmacogenomics Clinic at BWH is a \n",
      "resource for patients loo king for a more detailed interpretation of the clinical relevance of the results \n",
      "(https://www.brighamandwomens.org/medicine/genetics/genetics -and-genomic -medicine/pharmacogenomics ).  \n",
      " \n",
      "Genotype  Phenotype  FDA Drugs  with  PGx Labelling  \n",
      "TPMT *1/*1  \n",
      "NUDT15 c.415C/c.415C  TPMT Normal metabolizer  \n",
      "NUDT15 Normal metabolizer  Azathioprine, Mercaptopurine, Thioguanine  \n",
      "IFNL3 c. -3180G/c. -3180G>A  IFNL3 Unfavorable response  Peginterferon Alfa -2b \n",
      "CYP2C9 *1/*1  CYP2C9 Normal Metabolizer  - Activity score 2  Celecoxib, Dronabinol, Erdafitinib, Flibanserin, \n",
      "Flurbiprofen, Lesinurad, Meloxicam, Phenytoin, \n",
      "Piroxicam, Siponimod  \n",
      "CYP2C19 *1/*17  CYP2C19 Rapid metabolizer  Brivaracetam, Carisoprodol, Citalopram, \n",
      "Clobazam, Clopidogrel, Dexlansoprazole, \n",
      "Diazepam, Doxepin, Escitalopram, \n",
      "Esomeprazole, Flibanserin, Omeprazole, \n",
      "Pantoprazole, Rabeprazole, Voriconazole  \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 4 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      "Genotype  Phenotype  FDA Drugs  with  PGx Labelling  \n",
      "CYP2C9 *1/*1  \n",
      "VKORC1 G/A  \n",
      "CYP4F2 *1/*3  Use genotypes to guide dosing  Warfarin  \n",
      "SLCO1B1 *1A/*1A  SLCO1B1 Normal function  Elagolix, Rosuvastatin, Simvastatin  \n",
      "DPYD wild type at all tested positions  DPYD Normal metabolizer - Activity Score 2  Capecitabine, Fluorouracil  \n",
      "CYP3A5 *3/*3  CYP3A5 Poor metabolizer  Tacrolimus  \n",
      "UGT1A1 *1/*1  UGT1A1 Normal Metabolizer  Belinostat, Dolutegravir, Irinotecan, Nilotinib, \n",
      "Pazopanib, Raltegravir, Sacituzumab govitecan -\n",
      "hziy \n",
      " \n",
      "RECOMMENDATIONS  \n",
      "These results should be interpreted in the context of this individual's personal medical history and family history.  Genetic counseling is \n",
      "recommended for this individual and their relatives. Familial variant testing is available if desired.  \n",
      " \n",
      "COVERAGE SUMMARY  \n",
      "Sequenc ing of this individual’s genome covered 97.5%  of all positions at 15X coverage or higher. Please note that the presence of pathogenic \n",
      "variants in genes not analyzed, genes with incomplete coverage, or regions not captured by filtering strategies cannot be ful ly excluded.  \n",
      " \n",
      "  METHODOLOGY AND LIMITATIO NS \n",
      "Genome sequence is generated from genomic DNA that is fragmented and barcoded. Library fragments were sequenced (2x150 base paired \n",
      "end) using Sequencing -By-Synthesis (SBS) chemistry and the Illumina NovaSeq sequencer with a minimum  coverage of at least 20X for  90%. \n",
      "Sequence data were aligned to the GRCh38 assembly after discarding low quality sequences. Illumina's DRAGEN (Dynamic Read Ana lysis for \n",
      "GENomics) platform was used for demultiplexing, read mapping, genome alignment, read sorting, duplicate mark ing, and variant calling. \n",
      "Technical sensitivity of this assay is 99.10% (95% CI: 99.04 -99.16%) and positive predictive value is 99.39%  (95% CI: 99.37 -99.41%).  Variants in \n",
      "4,302 genes  with some level of published evidence for a gene -disease association are subsequently filtered to identify: (1) variants classified as \n",
      "pathogenic or likely pathogenic in our internal database; (2) variants classified as pathogenic or likely pathogenic by a ClinGen -approved expert \n",
      "panel; (3) variants classified as disease causing in public  databases that have a minor allele frequency <5.0% in the Genome Aggregation \n",
      "Database (gnomAD, https://gnomad.broadinstitute.org/ ); and (4) nonsense, frameshift, and +/ -1,2 splice -site variants in disease -associated \n",
      "genes with a minor allele frequency ≤0.1% in gnomAD. The evidence for phenotype -causality is then evaluated for each variant identified from  \n",
      "the filtering strategies  listed  above and variants are classified bas ed on ACMG/AMP criteria (Richards et al. 2015) with ClinGen rule \n",
      "specifications (https://www.clinicalgenome.org/working -groups/sequence -variant -interpretation/). Variants are reported according to HGVS \n",
      "nomenclature ( http://varnomen.hgvs.org/ ). Only those v ariants with evidence for causing or contributing to disease are reported. All disease -\n",
      "associated variants on this report are confirmed via Sanger sequencing or another orthogonal technology.  Please contact the laboratory for \n",
      "additional information.  \n",
      " \n",
      "Risk Alleles: Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. The \n",
      "following likely or established risk alleles are examined and reported if identified in the “reported genotype” listed below. Some  variants in \n",
      "these genes are associated with additional diseases and therefore other variants identified in these genes may be included on  different sections \n",
      "of this report. Additional risk variants, if identified, may also be i ncluded on this report at the discretion of the laboratory.  \n",
      " \n",
      "Gene  (Transcript)  Associated Risk  Reportable Variants  Reportable Genotypes  \n",
      "APC (NM_000038.4)  Colorectal Cancer  c.3920T>A (p.Ile1307Lys)  Heterozygous or homozygous  \n",
      "APOE  (NM_000041.2)  Alzheimer’s Disease  e4 Allele - \n",
      "c.388T>C (p.Cys130Arg)  Heterozygous with e2 or e3 or \n",
      "homozygous (e2/e4, e3/e4, e4/e4)  \n",
      "APOL1  (NM_003661.3)  Non -diabetic Nephropathy  G1 Allele  – \n",
      "c.1164_1169delTTATAA (p.Asn388_Tyr389del)  Homozygous (G1/G1 or G2/ G2) or \n",
      "compound heterozygous (G1/G2)  G2 Allele  - \n",
      "c.[1024A>G;1152T>G] (p.[Ser342Gly;Ile384Met])  \n",
      "CHEK2  (NM_001005735)  Breast, Colorectal, and Papillary Thyroid Cancers  c.599T>C (p.Ile200Thr)  Heterozygous or homozygous  \n",
      "CTRC  (NM_007272.2)  Pancreatitis  c.760C>T (p.Arg254Trp)  Heterozygous or homozygous  \n",
      "F2 (NM_000506.3)  Venous Thromboembolism  c.*97G>A  Heterozygous or homozygous  \n",
      "F5 (NM_000130.4)  Factor V Deficiency  c.1601G>A (p.Arg534Gln)  Heterozygous or homozygous  \n",
      "GBA  (NM_001005741.2)  Parkinson's Disease  c.1226A>G (p.Asn409Ser)  Heterozygous or homozygous  \n",
      "HFE (NM_000410.3)  Hemochromatosis  c.845G>A (p.Cys282Tyr)  Homozygous  \n",
      "LRRK2  (NM_198578.3)  Parkinson's Disease  c.6055G>A (p.Gly2019Ser)  Heterozygous or homozygous  \n",
      "MC1R  (NM_002386)  Melanoma  c.880G>C  (p.Asp294His)  Heterozygous or homozygous  \n",
      "MITF  (NM_000248.3)  Melanoma  c.952G>A (p.Glu318Lys)  Heterozygous or homozygous  \n",
      "MUC5B  (NM_002458.2)  Pulmonary Fibrosis  c.-3133G>T  Heterozygous or homozygous  \n",
      "PNPLA3  (NM_025225.2)  Non -alcoholic Fatty Liver Disease Type 1 c.444C>G (p.Ile148Met)  Homozygous  \n",
      "PRNP  (NM_000311.3)  Prion Disease  c.628G>A (p.Val210Ile)  Heterozygous or homozygous  \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 5 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      "SERPINA1  \n",
      "(NM_001127701.1)  Alpha -1 Antitrypsin Deficiency  S Allele - \n",
      "c.863A>T (p.Glu288Val)  Homozygous Z allele (Z/Z) or compound \n",
      "heterozygous Z and S allele (Z/S)  Z Allele - \n",
      "c.1096G>A (p.Glu366Lys)  \n",
      "SERPINC1  (NM_000488)  Venous Thromboembolism  c.1246G>T (p.Ala416Ser)  Heterozygous or homozygous  \n",
      "SPINK1  (NM_003122)  Pancreatitis  c.101A>G (p.Asn34Ser)  Heterozygous or homozygous  \n",
      " \n",
      "PGx: Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. Diplotypes and \n",
      "phenotypes are generated using the Clinical Pharmacogenetics Implementation Consortium (CPIC®) allele tables \n",
      "(https://cpicpgx.org/guidelines/ ) and custom scripts. The following pharmacogenomic variants are detected by this assay: CYP2C19  \n",
      "(NM_000769.1): *2 (rs4244285/c.681G>A, rs12769205/c. 332-23A>G), *3 (rs4986893/c.636G>A), *4A (rs28399504/c.1A>G), *4B \n",
      "(rs28399504/c.1A>G, rs12248560/c. -806C>T), *5 (rs56337013/c.1297C>T), *6 (rs72552267/c.395G>A), *7 (rs72558186/c.819+2T>A), *8 \n",
      "(rs41291556/c.358T>C), *17 (rs12248560/c. -806C>T), *35 (rs12769 205/c.332 -23A>G); CYP2C9 (NM_000771.3):  *2 (rs1799853/c.430C>T), *3 \n",
      "(rs1057910/c.1075A>C), *5 (rs28371686/c.1080C>G), *6 (rs9332131/c.818del), *8 (rs7900194/c.449G>A), *11 (rs28371685/c.1003C>T), *13 \n",
      "(rs72558187/c.269T>C); CYP3A5 (NM_000777.4) : *3 (rs776746/c.219 -237A>G), *6 (rs10264272/c.624G>A), *7 (rs41303343/c.1035dup); CYP4F2 \n",
      "(NM_001082.4) : *3 (rs2108622/c.1297G>A); DPYD (NM_000110.3):  rs3918290/c.1905+1G>A, rs55886062/c.1679T>G, rs67376798/c.2846A>T, \n",
      "rs72549309/c.299_302del/legacy name c.295_ 299TCATT>T, rs115232898/c.557A>G, rs1801266/c.703C>T, rs78060119/c.1156G>T, \n",
      "rs56038477/c.1236G>A, rs72549303/c.1898del, rs1801268/c.2983G>T, rs75017182/c.1129 -5923C>G;  IFNL3 (NM_172139.3):  rs12979860/c. -\n",
      "3180G>A; NUDT15 (NM_018283.3):  rs116855232/c.415C>T; SLCO1B1 (NM_006446.4):  rs4149056/c.521T>C; TPMT (NM_000367.4):  *2 \n",
      "(rs1800462/c.238G>C), *3A (rs1800460/c.460G>A, rs1142345/c.719A>G), *3B (rs1800460/c.460G>A), *3C (rs1142345/c.719A>G),  *4 \n",
      "(rs1800584/c.626 -1G>A);  VKORC1 (NM_024006.4):  rs9923231/c. -1639G>A;   UGT1A1  (NM_000463.3): *6 (rs4148323/c.211G>A), *27 \n",
      "(rs35350960/c.686C>A), *36 (rs3064744/c. -53TA[6]), *28 (rs3064744/ c. -53TA[8]), *37 (rs3064744/ c. -53TA[9]). Variant and star (*) allele \n",
      "definitions are available for download at https://personalizedmedicine.partners.org/Laboratory -For-Molecular -\n",
      "Medicine/Tests/Genomic%20Screening%20Spray.aspx  (see the link “details here” in the Methodology section of this webpage).  Additionally, \n",
      "variants in G6PD , RYR1 , and CACNA1S  associated with an altered metabolism status are reported, if identified. Detected variants include those \n",
      "that are clinically relevant based on allele function and frequency information found in tables created by PharmGKB and CPIC  \n",
      "(https://www.pharmgkb.org/page/pgxGeneRef ). This test does not report all pharmacogenomic variants that might alter protein function. A \n",
      "*1/*1 or wild type result is assigned if no variants that are included in the test are detected. Therefore, a *1/*1 or other result does not exclude \n",
      "the possibility  that an individual has a different phenotype that may alter drug response. This risk may vary among ancestries. This assay ca nnot \n",
      "determine if multiple variants in the same gene are present in cis or trans, leading to an inability to definitively assign a  diplotype and \n",
      "phenotype. This test does not detect copy number variants. Genetic information to guide prescribing decisions may be found in  a drug's FDA -\n",
      "approved label . These labels are found at https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm   and a table of current PGx biomarkers at \n",
      "https://www.fda.gov/medical -devices/precision -medicine/table -pharmacogenetic -associations  and https://www.fda.gov/drugs/science -and-\n",
      "research -drugs/table -pharmacogenomic -biomarkers -drug -labeling . \n",
      " \n",
      "Limitations : Specific types of genetic variation, such as short tandem repe ats (including triplet repeat expansions), structural variation, and large \n",
      "copy number events are currently not reliably detected by this assay. Additionally, while genome sequencing covers ~95% of th e genome; there \n",
      "are certain regions for which the assay may fail to adequately generate sequence information, such as regions of high homology or regions that \n",
      "are highly repetitive as well as regions with low coverage, mapping quality or base quality. Moreover, not all disease -associated genes have \n",
      "been identif ied and the clinical significance of variation in many genes is not well understood. The sequencing data is limited to the cell -type \n",
      "(material) source used for DNA extraction. Additionally, not all variants identified have been analyzed. Variant interpreta tion may change over \n",
      "time if more information becomes available.  \n",
      " \n",
      "The initial sequencing component of this test was performed by the Clinical Research Sequencing Platform of the Broad Institu te (320 Charles St, \n",
      "Cambridge, MA 02141; CLIA#22D2055652), and t he Sanger confirmation, interpretive algorithms and clinical reports were generated by the \n",
      "Laboratory for Molecular Medicine at Partners Healthcare Personalized Medicine (LMM, 65 Landsdowne St, Cambridge, MA 02139; 6 17-768-\n",
      "8500; CLIA#22D1005307). This test  has not been cleared or approved by the U.S. Food and Drug Administration (FDA). The FDA has determined \n",
      "that such clearance or approval is not necessary.  \n",
      " \n",
      "REFERENCES  \n",
      " \n",
      "Alfardan  J, Mohsen AW, Copeland S, Ellison J, Keppen -Davis L, Rohrbach M, Powell BR, Gillis J, Matern D, Kant J, Vockley J. 2010. Characterization of new ACADSB gene \n",
      "sequence mutations and clinical implications in patients with 2 - \n",
      "methylbutyrylglycinuria identifie d by newborn screening. Mol Genet Metab. 100(4):333 - 8. PMID: 20547083.  \n",
      " \n",
      "Amstutz U, Henricks LM, Offer SM, Barbarino J, Schellens JHM, Swen JJ, Klein TE,  McLeod HL, Caudle KE, Diasio RB, Schwab M. 2018. Clinical Pharmacogenetics  \n",
      "Implementation Consortium ( CPIC) Guideline for Dihydropyrimidine Dehydrogenase  \n",
      "Genotype and Fluoropyrimidine Dosing: 2017 Update. Clin. Pharmacol. Ther. 103(2):210 -216. PMID: 29152729.  \n",
      " \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 6 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      "Antony D, Becker -Heck A, Zariwala MA, Schmidts M, Onoufriadis A, Forouhan  M, Wilson R,  Taylor -Cox T, Dewar A, Jackson C, Goggin P, Loges NT, Olbrich H, Jaspers \n",
      "M, Jorissen  M, Leigh MW, Wolf WE, Daniels ML, Noone PG, Ferkol TW, Sagel SD, Rosenfeld M, Rutman  A, Dixit A, O'Callaghan C, Lucas JS, Hogg C, Scambler PJ, Emes \n",
      "RD, Uk10k , Chung EM,  \n",
      " \n",
      "Shoemark A, Knowles MR, Omran H, Mitchison HM. 2013. Mutations in CCDC39 and CCDC40  are the major cause of primary ciliary dyskinesia with axonemal \n",
      "disorganization and  absent inner dynein arms. Hum. Mutat. 34(3):462 -72. PMID: 23255504.  \n",
      " \n",
      "Becker -Heck A, Zohn IE, Okabe N, Pollock A, Lenhart KB, Sullivan -Brown J, McSheene J,  Loges NT, Olbrich H, Haeffner K,  \n",
      "Fliegauf M, Horvath J, Reinhardt R, Nielsen KG,  Marthin JK, Baktai G, Anderson KV, Geisler R, Niswander L, Omran H, Burdine RD. 2011.  The coil ed-coil domain \n",
      "containing protein CCDC40 is essential for motile cilia  function and left -right axis formation. Nat. Genet. 43(1):79 -84. PMID: 21131974.  \n",
      " \n",
      "Birdwell KA, Decker B, Barbarino JM, Peterson JF, Stein CM, Sadee W, Wang D, Vinks AA,  He Y, Swen JJ, L eeder JS, van Schaik R, Thummel KE, Klein TE, Caudle KE, \n",
      "MacPhee IA.  2015. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP3A5  Genotype and Tacrolimus Dosing. Clin. Pharmacol. Ther. \n",
      "98(1):19 -24. PMID: 25801146.  \n",
      " \n",
      "Caudle KE, Dunn enberger HM, Freimuth RR, Peterson JF, Burlison JD, Whirl -Carrillo M,  Scott SA, Rehm HL, Williams MS, Klein TE, Relling MV, Hoffman JM. 2017. \n",
      "Standardizingterms for clinical pharmacogenetic test results: consensus terms from the Clinical  Pharmacogenetics Implementation Consortium (CPIC). Genet. Med. \n",
      "19(2):215 -223. PMID:  27441996.  \n",
      " \n",
      "Caudle KE, Rettie AE, Whirl -Carrillo M, Smith LH, Mintzer S, Lee MT, Klein TE,  Callaghan JT, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical  \n",
      "pharmacogenetics implementation consortium guidelines for CYP2C9 and HLA -B genotypes  and phenytoin dosing. Clin. Pharmacol. Ther. 96(5):542 -8. PMID: \n",
      "25099164.  \n",
      " \n",
      "Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, Schwab M. 2013. Clinical  Pharmacogenetics Implement ation Consortium guidelines for dihydropyrimidine  \n",
      "dehydrogenase genotype and fluoropyrimidine dosing. Clin. Pharmacol. Ther. 94(6):640 -5. PMID: 23988873.  \n",
      " \n",
      "Crews KR, Gaedigk A, Dunnenberger HM, Leeder JS, Klein TE, Caudle KE, Haidar CE, Shen  DD, Callaghan J T, Sadhasivam S, Prows CA, Kharasch ED, Skaar TC, Clinical  \n",
      "Pharmacogenetics Implementation Consortium. 2014. Clinical Pharmacogenetics  \n",
      "Implementation Consortium guidelines for cytochrome P450 2D6 genotype and codeine  therapy: 2014 update. Clin. Pharmacol. Ther. 95(4):376 -82. PMID: 24458010.  \n",
      " \n",
      "Gammal RS, Court MH, Haidar CE, Iwuchukwu OF, Gaur AH, Alvarellos M, Guillemette C,  Lennox JL, Whirl -Carrillo M, Brummel SS, Ratain MJ, Klein TE, Schackman BR, \n",
      "Caudle KE,  Haas DW, Clinical Pharmacogenetics Implementatio n Consortium. 2016. Clinical  Pharmacogenetics Implementation Consortium (CPIC) Guideline for \n",
      "UGT1A1 and Atazanavir  \n",
      "Prescribing. Clin Pharmacol Ther. 99(4):363 -9. PMID: 26417955.  \n",
      " \n",
      "Gonsalves SG, Dirksen RT, Sangkuhl K, Pulk R, Alvarellos M, Vo T, Hikino K, R oden D,  Klein TE, Poler SM, Patel S, Caudle KE, Gordon R, Brandom B, Biesecker LG. 2019.  \n",
      "Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for the Use of  Potent Volatile Anesthetic Agents and Succinylcholine in the Context of RYR1 \n",
      "or CACNA1S Genotypes. Clin. Pharmacol. Ther. 105(6):1338 -1344. PMID: 30499100.  \n",
      " \n",
      "Hicks JK, Bishop JR, Sangkuhl K, Müller DJ, Ji Y, Leckband SG, Leeder JS, Graham RL,  Chiulli DL, LLerena A, Skaar TC, Scott SA, Stingl JC, Klein TE, Caudle KE, Gaedigk A,  \n",
      "Clinical Ph armacogenetics Implementation Consortium. 2015. Clinical Pharmacogenetics  Implementation Consortium (CPIC) Guideline for CYP2D6 and CYP2C19 \n",
      "Genotypes and Dosing  \n",
      "of Selective Serotonin Reuptake Inhibitors. Clin. Pharmacol. Ther. 98(2):127 -34. PMID:  25974703 . \n",
      " \n",
      "Hicks JK, Sangkuhl K, Swen JJ, Ellingrod VL, Müller DJ, Shimoda K, Bishop JR, Kharasch  ED, Skaar TC, Gaedigk A, Dunnenberger HM, Klein TE, Caudle KE, Stingl JC. 2017.  \n",
      "Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and  CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin.  \n",
      "Pharmacol. Ther. 102(1):37 -44. PMID: 27997040.  \n",
      " \n",
      "Johnson JA, Caudle KE, Gong L, Whirl -Carrillo M, Stein CM, Scott SA, Lee MT, Gage BF,  Kimmel SE, Perera  MA, Anderson JL, Pirmohamed M, Klein TE, Limdi NA, Cavallari \n",
      "LH, Wadelius M. 2017. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline  for Pharmacogenetics -Guided Warfarin Dosing: 2017 Update. Clin. \n",
      "Pharmacol. Ther. 102  \n",
      "(3):397 -404. PMID:  28198005.  \n",
      " \n",
      "Knowles MR, Leigh MW, Carson JL, Davis SD, Dell SD, Ferkol TW, Olivier KN, Sagel SD,  Rosenfeld M, Burns KA, Minnix SL, Armstrong MC, Lori A, Hazucha MJ, Loges NT, \n",
      "Olbrich  H, Becker -Heck A, Schmidts  M, Werner C, Omran H, Zariwala MA, Genetic Disorders of  Mucociliary Clearance Consortium. 2012. Mutations of DNAH11 in \n",
      "patients with primary  ciliary dyskinesia with normal ciliary ultrastructure. Thorax. 67(5):433 -41. PMID:  22184204.  \n",
      " \n",
      "Lima JJ, Thomas CD, Barbarino J, Desta Z, Van Driest SL, El Rouby N, Johnson JA,  Cavallari LH, Shakhnovich V, Thacker DL, Scott SA, Schwab M, Uppugunduri CRS, \n",
      "Formea  CM, Franciosi JP, Sangkuhl K, Gaedigk A, Klein TE, Gammal RS, Furuta T. 2021. Clinical  Pharmacogenetics Implem entation Consortium (CPIC) Guideline for \n",
      "CYP2C19 and Proton  Pump Inhibitor Dosing. Clin Pharmacol Ther. 109(6):1417 -1423. PMID: 32770672.  \n",
      " \n",
      "Monnier N, Procaccio V, Stieglitz P, Lunardi J. 1997. Malignant -hyperthermia  susceptibility is associated with a muta tion of the alpha 1 -subunit of the human  \n",
      "dihydropyridine -sensitive L -type voltage -dependent calcium -channel receptor in  \n",
      "skeletal muscle. Am. J. Hum. Genet. 60(6):1316 -25. PMID: 9199552.   \n",
      " \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 7 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      "Moriyama B, Obeng AO, Barbarino J, Penzak SR, Henning SA, Scott SA, Agúndez J, Wingard  JR, McLeod HL, Klein TE, Cross SJ, Caudle KE, Walsh TJ. 2017. Clinical  \n",
      "Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP2C19 and  \n",
      "Voriconazole Therapy. Clin. Pharmacol. Ther. 102(1):45 -51. PMID: 27981572.  \n",
      " \n",
      "Muir AJ, Gong  L, Johnson SG, Lee MT, Williams MS, Klein TE, Caudle KE, Nelson DR,  Clinical Pharmacogenetics Implementation Consortium (CPIC). 2014. Clinical  \n",
      "Pharmacogenetics Implementation Consortium (CPIC) guidelines for IFNL3 (IL28B)  genotype and PEG interferon --base d regimens. Clin. Pharmacol. Ther. 95(2):141 -6. \n",
      "PMID:  24096968.  \n",
      " \n",
      "Nakhleh N, Francis R, Giese RA, Tian X, Li Y, Zariwala MA, Yagi H, Khalifa O, Kureshi  S, Chatterjee B, Sabol SL, Swisher M, Connelly PS, Daniels MP, Srinivasan A, Kuehl \n",
      "K, Kravitz N, Burns K,  Sami I, Omran H, Barmada M, Olivier K, Chawla KK, Leigh M, Jonas  R, Knowles M, Leatherbury L, Lo CW. 2012. High prevalence of respiratory \n",
      "ciliary  dysfunction in congenital heart disease patients with heterotaxy. Circulation. 125(18):  2232 -42. PMID: 224999 50. \n",
      " \n",
      "Porta F, Chiesa N, Martinelli D, Spada M. 2019. Clinical, biochemical, and molecular  spectrum of short/branched -chain acyl -CoA dehydrogenase deficiency: two new \n",
      "cases and  review of literature. J Pediatr Endocrinol Metab. 32(2):101 -108. PMID: 30730842.  \n",
      " \n",
      "Ramsey LB, Johnson SG, Caudle KE, Haidar CE, Voora D, Wilke RA, Maxwell WD, McLeod HL,  Krauss RM, Roden DM, Feng Q, Cooper -DeHoff RM, Gong L, Klein TE, \n",
      "Wadelius M, Niemi M.  2014. The clinical pharmacogenetics implementation consortium guideline for SLCO1 B1 and simvastatin -induced myopathy: 2014 update. Clin. \n",
      "Pharmacol. Ther. 96(4):423 -8. \n",
      "PMID: 24918167.  \n",
      " \n",
      "Relling MV, Gardner EE, Sandborn WJ, Schmiegelow K, Pui CH, Yee SW, Stein CM, Carrillo  M, Evans WE, Hicks JK, Schwab M, Klein TE, Clinical Pharmacogeneti cs \n",
      "Implementation  Consortium. 2013. Clinical pharmacogenetics implementation consortium guidelines for  thiopurine methyltransferase genotype and thiopurine \n",
      "dosing: 2013 update. Clin.  Pharmacol. Ther. 93(4):324 -5. PMID: 23422873.  \n",
      " \n",
      "Relling  MV, McDonagh EM, Chang T, Caudle KE, McLeod HL, Haidar CE, Klein T, Luzzatto  L, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical  \n",
      "Pharmacogenetics Implementation Consortium (CPIC) guidelines for rasburicase therapy  in the context of G6PD  deficiency genotype. Clin. Pharmacol. Ther. 96(2):169 -\n",
      "74. PMID:  24787449.  \n",
      " \n",
      "Relling MV, Schwab M, Whirl -Carrillo M, Suarez -Kurtz G, Pui CH, Stein CM, Moyer AM,  Evans WE, Klein TE, Antillon -Klussmann FG, Caudle KE, Kato M, Yeoh AEJ, \n",
      "Schmiegelow K,  Yang JJ. 2019. Clinical Pharmacogenetics Implementation Consortium Guideline for  Thiopurine Dosing Based on TPMT and NUDT15 Genotypes: 2018 \n",
      "Update. Clin. Pharmacol.  \n",
      "Ther. 105(5):1095 -1105. PMID: 30447069.  \n",
      " \n",
      "Schwabe GC, Hoffmann K, Loges NT, Birker D, Rossier C, de Santi MM, Olbrich H,  Fliegauf M, Failly M, Liebers U, Collura M, Gaedicke G, Mundlos S, Wahn U, Blouin JL,  \n",
      "Niggemann B, Omran H, Antonarakis SE, Bartoloni L. 2008. Primary ciliary dyskinesia  associa ted with normal axoneme ultrastructure is caused by DNAH11 mutations. \n",
      "Hum  Mutat. 29(2):289 -98. PMID: 18022865.  \n",
      " \n",
      "Scott SA, Sangkuhl K, Stein CM, Hulot JS, Mega JL, Roden DM, Klein TE, Sabatine MS,  Johnson JA, Shuldiner AR, Clinical Pharmacogenetics Implemen tation Consortium. \n",
      "2013.  Clinical Pharmacogenetics Implementation Consortium guidelines for CYP2C19 genotype  and clopidogrel therapy: 2013 update. Clin. Pharmacol. Ther. 94(3):317 -\n",
      "23. PMID:  23698643.  \n",
      " \n",
      "Spedicati B, Cocca M, Palmisano R, Faletra F, Barbieri C, Francescatto M, Mezzavilla  M, Morgan A, Pelliccione G, Gasparini P, Girotto G. 2021. Natural human knockouts \n",
      "and Mendelian disorders: deep phenotyping in Italian isolates. Eur J Hum Genet. 29(8):1272 - 1281. PMID: 33727708.  \n",
      " \n",
      "Stewart SL, Hogan K, Rosenber g H, Fletcher JE. 2001. Identification of the Arg1086His  mutation in the alpha subunit of the voltage -dependent calcium channel \n",
      "(CACNA1S) in a  North American family with malignant hyperthermia. Clin. Genet. 59(3):178 -84. PMID:  11260227.  \n",
      " \n",
      "Swen JJ, Nijenhuis  M, de Boer A, Grandia L, Maitland -van der Zee AH, Mulder H, Rongen  GA, van Schaik RH, Schalekamp T, Touw DJ, van der Weide J, Wilffert B, Deneer \n",
      "VH, Guchelaar HJ. 2011. Pharmacogenetics: from bench to byte --an update of guidelines.  Clin. Pharmacol. Ther. 89(5):662 -73. PMID: 21412232.  \n",
      " \n",
      "Theken KN, Lee CR, Gong L, Caudle KE, Formea CM, Gaedigk A, Klein TE, Agúndez JAG,  Grosser T. 2020. Clinical Pharmacogenetics Implementation Consortium \n",
      "Guideline (CPIC)  for CYP2C9 and Nonsteroidal Anti -Inflammatory Drugs. Cli n Pharmacol Ther. 108(2):191 - 200. PMID: 32189324.  \n",
      " \n",
      "Wilke RA, Ramsey LB, Johnson SG, Maxwell WD, McLeod HL, Voora D, Krauss RM, Roden DM,  Feng Q, Cooper -Dehoff RM, Gong L, Klein TE, Wadelius M, Niemi M, \n",
      "Clinical  Pharmacogenomics Implementation Consortium (CPIC). 2012. The clinical pharmacogenomics  implementation consortium: CPIC guideline for SLCO1B1 and \n",
      "simvastatin -induced  myopathy. Clin. Pharmacol. Ther. 92(1):112 -7. PMID: 22617227.  \n",
      " \n",
      "Witherspoon JW, Meilleur KG. 2016. Review of RyR1 pathway and associated  pathomechanisms. Acta Neuropathol Commun. 4(1):121. PMID: 27855725.  \n",
      " \n",
      "Yang SK, Hong M, Baek J, Choi H, Zhao W, Jung Y, Haritunians T, Ye BD, Kim KJ, Park  SH, Park SK, Yang DH, Dubinsky M, Lee I, McGovern DP, Liu J, Song K. 2014. A \n",
      "common  missense variant in NUDT15 confers susceptibility to thiopurine -induced leukopenia.  Nat. Genet. 46(9):1017 -20. PMID: 25108385.  \n",
      " \n",
      "Zariwala MA, Gee HY, Kurkowiak M, Al -Mutairi DA, Leigh MW, Hurd TW, Hjeij R, Dell SD,  Chaki M, Dougherty GW, Adan M, Sp ear PC, Esteve -Rudd J, Loges NT, Rosenfeld \n",
      "M, Diaz  KA, Olbrich H, Wolf WE, Sheridan E, Batten TF, Halbritter J, Porath JD, Kohl S, Lovric  S, Hwang DY, Pittman JE, Burns KA, Ferkol TW, Sagel SD, Olivier KN, Morgan \n",
      "LC, Werner  C, Raidt J, Pennekamp P, Sun Z, Zhou W, Airik R, Natarajan S, Allen SJ, Amirav I,  Wieczorek D, Landwehr K, Nielsen K, Schwerk N, Sertic J, Köhler G, Washburn \n",
      "Accession ID:  PM-23-NXXXX  \n",
      "Name : LAST , FIRST  \n",
      "Page: 8 of 8 \n",
      " \n",
      "_____________________________________________________________________________________________________________________________ _________________________________________________________  \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                 Matthew S. Lebo, Ph.D., FACMG, Director  \n",
      "https://www.massgeneralbrigham.org/en/research -and-innovation/centers -and-programs \\personalized -medicine \\molecular -medicine   CLIA#: 22D1005307  \n",
      " Personalized Medicin e \n",
      "LABORATORY FOR MOLECULAR MEDICINE  \n",
      "65 Landsdowne St, Cambridge, MA  02139  \n",
      "Phone: (617) 768 -8500 • Fax: (617) 768 -8513  \n",
      "J, Levy S,  Fan S, Koerner -Rettberg C, Amselem S, Williams DS, Mitchell BJ, Drummond IA, Otto EA,  Omran H, Knowles MR, Hildebrandt F. 2013. ZMYND10 is mutated \n",
      "in primary ciliary  dyskinesia and interacts with LRRC6. Am. J. Hum. Genet. 93(2):336 -45. PMID: 23891469.  \n",
      " \n",
      " \n",
      "Report Preparation by: First  Last \n",
      "Final Report by:  First  Last, PhD, FACMG  \n",
      "\n"
     ]
    }
   ],
   "execution_count": 21
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:21.402199Z",
     "start_time": "2025-06-05T13:12:21.396969Z"
    }
   },
   "cell_type": "code",
   "source": [
    "\n",
    "information = pdf_object.metadata\n",
    "number_of_pages = len(pdf_object.pages)\n",
    "txt = f\"\"\"\n",
    "Author: {information.author}\n",
    "Creator: {information.creator}\n",
    "Producer: {information.producer}\n",
    "Subject: {information.subject}\n",
    "Title: {information.title}\n",
    "Number of pages: {number_of_pages}\n",
    "\"\"\"\n",
    "# Here the metadata of your pdf\n",
    "print(txt)\n"
   ],
   "id": "719097cdf9b555c0",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Author: None\n",
      "Creator: Microsoft® Word for Microsoft 365\n",
      "Producer: Microsoft® Word for Microsoft 365\n",
      "Subject: None\n",
      "Title: Genome Report Example\n",
      "Number of pages: 8\n",
      "\n"
     ]
    }
   ],
   "execution_count": 22
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "# pymupdf\n",
   "id": "4829ac201fc52ebc"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:21.494881Z",
     "start_time": "2025-06-05T13:12:21.458881Z"
    }
   },
   "cell_type": "code",
   "source": [
    "import pymupdf # imports the pymupdf library\n",
    "\n",
    "pdf_file = \"PDF/genome-report-example.pdf\"\n",
    "doc = pymupdf.open(pdf_file)"
   ],
   "id": "cdb23220e0e00d2e",
   "outputs": [],
   "execution_count": 23
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## text",
   "id": "b8620d327a5465be"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:21.628602Z",
     "start_time": "2025-06-05T13:12:21.519130Z"
    }
   },
   "cell_type": "code",
   "source": [
    "for page in doc:  # iterate the document pages\n",
    "    page.read_contents()\n",
    "    text = page.get_text()\n",
    "    print(text)"
   ],
   "id": "b5b8fb624d262821",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      "CLIA#: 22D1005307\n",
      "Name: LAST, FIRST \n",
      "MRN: XXXXXXXX \n",
      "LMM Accession ID: PM-23-NXXXX \n",
      "DOB: MM/DD/YYYY\n",
      "Referring Facility: HS\n",
      "Specimen: Blood, Peripheral\n",
      "Sex at birth: Female\n",
      "Referring Physician: LAST, FIRST\n",
      "Received:  MM/DD/2023\n",
      "Family #: FXXXXXX\n",
      "Copies To: HOSPITAL SYSTEM\n",
      "Page: 1 of 7\n",
      "Test Performed: Expanded Genome Screening\n",
      "Test Codes: lmWGS-pnlFv4_L\n",
      "VARIANT SUMMARY \n",
      "REPORT SECTION \n",
      "Disease, \n",
      "Inheritance \n",
      "Gene \n",
      "Transcript \n",
      "Variant \n",
      "Allele State \n",
      "Classification \n",
      "A. MONOGENIC DISEASE FINDINGS\n",
      "None identified. \n",
      "B. RISK ALLELES\n",
      "None identified. \n",
      "C. CARRIER STATUS VARIANTS\n",
      "Primary ciliary \n",
      "dyskinesia, \n",
      "Autosomal Recessive \n",
      "CCDC40 \n",
      "NM_017950.4 \n",
      "c.248delC\n",
      "p.Ala83ValfsX84\n",
      "Heterozygous \n",
      "Pathogenic \n",
      "Primary ciliary \n",
      "dyskinesia, \n",
      "Autosomal Recessive \n",
      "DNAH11 \n",
      "NM_001277115.2 \n",
      "c.12363C>G \n",
      "p.Tyr4121X\n",
      "Heterozygous \n",
      "Pathogenic \n",
      "Short/branched chain \n",
      "acyl-CoA dehydrogenase \n",
      "deficiency, \n",
      "Autosomal Recessive \n",
      "ACADSB \n",
      "NM_001609.4 \n",
      "c.303+1G>A\n",
      "Heterozygous \n",
      "Pathogenic \n",
      "D. PHARMACOGENOMIC\n",
      "ASSOCIATIONS\n",
      "See below. \n",
      "DETAILED VARIANT INFORMATION \n",
      "A. MONOGENIC DISEASE FINDINGS\n",
      "This test did not identify any variants with the potential to cause monogenic disease in this individual. \n",
      "B. RISK ALLELES\n",
      " This test did not identify any reportable risk alleles (see methodology for the list of risk alleles included in this analysis). \n",
      "C. CARRIER STATUS VARIANTS\n",
      "This test identified 3 carrier status variants for autosomal recessive conditions. Autosomal recessive disorders are caused by the presence of \n",
      "pathogenic variants in both copies of the same gene. Being a carrier of these variants does not put this individual at risk for disease but may \n",
      "impact disease risk in this individual’s children. PLEASE NOTE: The presence of a second pathogenic variant in one of these genes cannot be \n",
      "definitively ruled out due to the technical and analytical limitations of this assay.  \n",
      "FAMILIAL AND REPRODUCTIVE RISK: The risk to this individual’s child or future child of developing an autosomal recessive condition is \n",
      "dependent on the carrier status of this individual's reproductive partner(s). Two carriers have a 25% risk for having a child with the associated \n",
      "disease. First degree relatives of this individual have a 50% chance of also being carriers of each of these variants. Other biologically related \n",
      "family members may also be carriers. \n",
      "GENOME SCREENING RESULTS SUMMARY \n",
      "NEGATIVE- NO MONOGENIC DISEASE FINDINGS. Sequencing of this individual’s genome identified 3 carrier status \n",
      "variants. Pharmacogenomic associations are also included in this report. Result details are listed below. \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 2 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      " \n",
      "Disease, \n",
      "Inheritance \n",
      "Gene  \n",
      "Transcript \n",
      "Variant \n",
      "Allele State \n",
      " \n",
      "Classification  \n",
      "Primary ciliary dyskinesia, \n",
      "Autosomal Recessive \n",
      "CCDC40 \n",
      "NM_017950.4 \n",
      "c.248delC \n",
      "p.Ala83ValfsX84 \n",
      "Heterozygous \n",
      "Pathogenic \n",
      "Genomic Coordinate \n",
      "Location \n",
      "Penetrance \n",
      "Carrier Phenotype \n",
      "Gene Coverage  \n",
      "g.80039966delC (chr17, GRCh38) \n",
      "Exon 3 \n",
      "High \n",
      "None reported \n",
      "100% at 15X \n",
      "VARIANT INTERPRETATION: The p.Ala83ValfsX84 variant in CCDC40 has been previously reported in 19 homozygous and 7 compound \n",
      "heterozygous individuals with primary ciliary dyskinesia (PCD) and segregated with disease in 1 homozygous affected relative (Becker-Heck 2011 \n",
      "PMID: 21131974, Nakhleh 2012 PMID: 22499950, Antony 2013 PMID: 23255504, Zariwala 2013 PMID: 23891469). This variant has been identified \n",
      "in 0.074% (860/1167354) of non-Finnish European chromosomes by gnomAD (http://gnomad.broadinstitute.org). However, this frequency is low \n",
      "enough to be consistent with a recessive allele frequency. This variant has also been reported in ClinVar (Variation ID 31069). This variant is \n",
      "predicted to cause a frameshift, which alters the protein’s amino acid sequence beginning at position 83 and leads to a premature termination \n",
      "codon 84 amino acids downstream. This alteration is then predicted to lead to a truncated or absent protein. Functional studies indicate that loss \n",
      "of CCDC40 function results in abnormal cilia structure and motility (Becker-Heck 2011 PMID: 21131974). In summary, this variant meets criteria to \n",
      "be classified as pathogenic for autosomal recessive primary ciliary dyskinesia. ACMG/AMP Criteria applied: PVS1, PM3_VeryStrong, \n",
      "PM2_Supporting, PP1. \n",
      "DISEASE INFORMATION: Primary ciliary dyskinesia is a rare genetic condition that is genetically heterogeneous. It is associated with recurrent \n",
      "respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities in the motility of the cilia \n",
      "and flagella that are found in the linings of organs and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and \n",
      "chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror-image reversal of all visceral organs, is \n",
      "found in 40-50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females \n",
      "with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Other symptoms can include recurrent ear \n",
      "infections and hydrocephalus in the brain. Pathogenic variants in CCDC40 contribute to 3-4% of primary ciliary dyskinesia (Medline Plus: \n",
      "https://medlineplus.gov/genetics/condition/primary-ciliary-dyskinesia, GeneReviews: https://www.ncbi.nlm.nih.gov/books/NBK1122). \n",
      "FAMILIAL AND REPRODUCTIVE RISK \n",
      "Disease Prevalence (Estimated) \n",
      "Carrier Frequency (Estimated) \n",
      "Reproductive Risk (Estimated) \n",
      "1/16000 (https://medlineplus.gov/genetics/condition/primary-\n",
      "ciliary-dyskinesia) \n",
      "1/317 to 1/366 (for CCDC40) \n",
      "1/1268 to 1/1464 (for CCDC40) \n",
      " \n",
      " \n",
      "Disease, \n",
      "Inheritance \n",
      "Gene  \n",
      "Transcript \n",
      "Variant \n",
      "Allele State \n",
      " \n",
      "Classification  \n",
      "Primary ciliary dyskinesia, \n",
      "Autosomal Recessive \n",
      "DNAH11 \n",
      "NM_001277115.2 \n",
      "c.12363C>G \n",
      "p.Tyr4121X \n",
      "Heterozygous \n",
      "Pathogenic \n",
      "Genomic Coordinate \n",
      "Location \n",
      "Penetrance \n",
      "Carrier Phenotype \n",
      "Gene Coverage  \n",
      "g.21880869C>G (chr7, GRCh38) \n",
      "Exon 75 \n",
      "High \n",
      "None reported \n",
      "100% at 15X \n",
      "VARIANT INTERPRETATION: The p.Tyr4121X variant in DNAH11 has been reported in the compound heterozygous state in 1 individual with \n",
      "primary ciliary dyskinesia and segregated with disease in 5 affected siblings (Schwabe 2008 PMID: 18022865). It has been identified in 0.045% \n",
      "(29/63996) of European (Finnish) chromosomes by gnomAD (http://gnomad.broadinstitute.org, v4.0.0). However, this frequency is low enough to \n",
      "be consistent with the carrier frequency. This variant has also been reported in ClinVar (Variation ID 6475). This nonsense variant leads to a \n",
      "premature termination codon at position 4121, which is predicted to lead to a truncated or absent protein. Biallelic loss-of-function of the \n",
      "DNAH11 gene is an established disease mechanism in autosomal recessive primary ciliary dyskinesia 7. In summary, this variant meets criteria to \n",
      "be classified as pathogenic for autosomal recessive primary ciliary dyskinesia 7. ACMG/AMP Criteria applied: PVS1, PP1_Strong, PM3, \n",
      "PM2_Supporting. \n",
      "DISEASE INFORMATION: Primary ciliary dyskinesia is a rare genetic condition that is genetically heterogeneous. It is associated with recurrent \n",
      "respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities in the motility of the cilia \n",
      "and flagella that are found in the linings of organs and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and \n",
      "chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror-image reversal of all visceral organs, is \n",
      "found in 40-50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females \n",
      "with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Other symptoms can include recurrent ear \n",
      "infections and hydrocephalus in the brain. Pathogenic variants in DNAH11 contribute to 6-9% of primary ciliary dyskinesia (Medline Plus: \n",
      "https://medlineplus.gov/genetics/condition/primary-ciliary-dyskinesia; GeneReviews: https://www.ncbi.nlm.nih.gov/books/NBK1122). \n",
      "FAMILIAL AND REPRODUCTIVE RISK \n",
      "Disease Prevalence (Estimated) \n",
      "Carrier Frequency (Estimated) \n",
      "Reproductive Risk (Estimated) \n",
      "1/16000 (https://medlineplus.gov/genetics/condition/primary-\n",
      "ciliary-dyskinesia) \n",
      "1/211 to 1/259 (for DNAH11) \n",
      "1/844 to 1/1036 (for DNAH11) \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 3 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      " \n",
      "Disease, \n",
      "Inheritance \n",
      "Gene  \n",
      "Transcript \n",
      "Variant \n",
      "Allele State \n",
      " \n",
      "Classification  \n",
      "Short/branched chain acyl-CoA \n",
      "dehydrogenase deficiency, \n",
      "Autosomal Recessive \n",
      "ACADSB \n",
      "NM_001609.4 \n",
      "c.303+1G>A \n",
      "Heterozygous \n",
      "Pathogenic \n",
      "Genomic Coordinate \n",
      "Location \n",
      "Penetrance \n",
      "Carrier Phenotype \n",
      "Gene Coverage  \n",
      "g.123037848G>A (chr10, GRCh38) \n",
      "Intron 3 \n",
      "Reduced \n",
      "None reported \n",
      "100% at 15X \n",
      "VARIANT INTERPRETATION: The c.303+1G>A variant in ACADSB has been reported in the homozygous state in 2 individual with short/branched \n",
      "chain acyl-CoA dehydrogenase (SBCAD) deficiency; both showed the biochemical phenotype but only one showed clinical symptoms (Alfardan \n",
      "2010 PMID: 20547083). It has also been reported in the homozygous state in 2 siblings without clinical symptoms (Spedicati 2021 PMID: \n",
      "33727708). It has been identified in 0.056% (628/1127804) of European (non-Finnish) chromosomes, including 1 homozygote, by gnomAD \n",
      "(http://gnomad.broadinstitute.org, v4.0.0). This variant occurs within the canonical splice site (+/- 1,2) and is predicted to cause altered splicing \n",
      "leading to an abnormal or absent protein. Computational prediction tools and conservation analyses are consistent with pathogenicity. Biallelic \n",
      "loss-of-function of the ACADSB gene is an established disease mechanism in autosomal recessive SBCAD deficiency. SBCAD deficiency results in a \n",
      "biochemical phenotype that be detected from birth; however, the clinical significance of this is unclear. 90% of individuals with SBCAD deficiency \n",
      "have no clinical symptoms, and the remaining 10% may display developmental delay and/or neurological disorders. These 10% of individuals may \n",
      "represent extreme end of the clinical spectrum or coincidental findings (Porta 2019 PMID: 30730842, Alfardan 2010 PMID: 20547083). In \n",
      "summary, this variant meets criteria to be classified as pathogenic for autosomal recessive SBCAD deficiency. ACMG/AMP Criteria applied: PVS1, \n",
      "PM3_Supporting. \n",
      "DISEASE INFORMATION: Short/branched chain acyl-CoA dehydrogenase deficiency (SBCAD deficiency) is a rare inborn error of metabolism that is \n",
      "more common among Hmong population in Southeast Asia. Affected individuals can be detected during newborn screening (NBS) but most have \n",
      "no clinical symptoms. 10% of SBCAD deficiency patients have developmental delay and neurological disorders; however, it is unclear if the SBCAD \n",
      "deficiency is causative (Porta 2019 PMID: 30730842, Medline Plus: https://medlineplus.gov/genetics/condition/short-branched-chain-acyl-coa-\n",
      "dehydrogenase-deficiency). \n",
      "FAMILIAL AND REPRODUCTIVE RISK \n",
      "Disease Prevalence (Estimated) \n",
      "Carrier Frequency (Estimated) \n",
      "Reproductive Risk (Estimated) \n",
      "Unknown \n",
      "Unknown \n",
      "Unknown \n",
      " \n",
      "D. PHARMACOGENOMIC ASSOCIATIONS \n",
      "Detailed dosing instructions are not provided in the brief interpretation notes below. Extrinsic factors (e.g. diet, smoking status, co-administered \n",
      "medications, drug-drug interactions) and intrinsic factors (e.g. sex, age, weight, renal or hepatic function) may affect drug response. A *1/*1 or \n",
      "wild type result is assigned if no variants that are included in the test are detected and does not exclude the possibility that an individual has a \n",
      "different phenotype that may alter drug response. Patients should not use the test results to stop or change any medication unless directed by a \n",
      "qualified clinician. Genetic information to guide prescribing decisions may be found in a drug's FDA-approved label. These labels are found at \n",
      "https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm and a table of current PGx biomarkers at https://www.fda.gov/medical-\n",
      "devices/precision-medicine/table-pharmacogenetic-associations and https://www.fda.gov/drugs/science-and-research-drugs/table-\n",
      "pharmacogenomic-biomarkers-drug-labeling. The Clinical Pharmacogenetics Implementation Consortium (CPIC®) website may be consulted for \n",
      "their most current recommendations, at https://cpicpgx.org/guidelines/; however, these have NOT received FDA-approval. Always consult a \n",
      "clinician or clinical pharmacologist before changing drug dosage or for additional information. The Pharmacogenomics Clinic at BWH is a \n",
      "resource for patients looking for a more detailed interpretation of the clinical relevance of the results \n",
      "(https://www.brighamandwomens.org/medicine/genetics/genetics-and-genomic-medicine/pharmacogenomics).  \n",
      " \n",
      "Genotype \n",
      "Phenotype \n",
      "FDA Drugs with PGx Labelling \n",
      "TPMT *1/*1 \n",
      "NUDT15 c.415C/c.415C \n",
      "TPMT Normal metabolizer \n",
      "NUDT15 Normal metabolizer \n",
      "Azathioprine, Mercaptopurine, Thioguanine \n",
      "IFNL3 c.-3180G/c.-3180G>A \n",
      "IFNL3 Unfavorable response \n",
      "Peginterferon Alfa-2b \n",
      "CYP2C9 *1/*1 \n",
      "CYP2C9 Normal Metabolizer  - Activity score 2 \n",
      "Celecoxib, Dronabinol, Erdafitinib, Flibanserin, \n",
      "Flurbiprofen, Lesinurad, Meloxicam, Phenytoin, \n",
      "Piroxicam, Siponimod \n",
      "CYP2C19 *1/*17 \n",
      "CYP2C19 Rapid metabolizer \n",
      "Brivaracetam, Carisoprodol, Citalopram, \n",
      "Clobazam, Clopidogrel, Dexlansoprazole, \n",
      "Diazepam, Doxepin, Escitalopram, \n",
      "Esomeprazole, Flibanserin, Omeprazole, \n",
      "Pantoprazole, Rabeprazole, Voriconazole \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 4 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      "Genotype \n",
      "Phenotype \n",
      "FDA Drugs with PGx Labelling \n",
      "CYP2C9 *1/*1 \n",
      "VKORC1 G/A \n",
      "CYP4F2 *1/*3 \n",
      "Use genotypes to guide dosing \n",
      "Warfarin \n",
      "SLCO1B1 *1A/*1A \n",
      "SLCO1B1 Normal function \n",
      "Elagolix, Rosuvastatin, Simvastatin \n",
      "DPYD wild type at all tested positions \n",
      "DPYD Normal metabolizer - Activity Score 2 \n",
      "Capecitabine, Fluorouracil \n",
      "CYP3A5 *3/*3 \n",
      "CYP3A5 Poor metabolizer \n",
      "Tacrolimus \n",
      "UGT1A1 *1/*1 \n",
      "UGT1A1 Normal Metabolizer \n",
      "Belinostat, Dolutegravir, Irinotecan, Nilotinib, \n",
      "Pazopanib, Raltegravir, Sacituzumab govitecan-\n",
      "hziy \n",
      " \n",
      "RECOMMENDATIONS \n",
      "These results should be interpreted in the context of this individual's personal medical history and family history. Genetic counseling is \n",
      "recommended for this individual and their relatives. Familial variant testing is available if desired.  \n",
      " \n",
      "COVERAGE SUMMARY \n",
      "Sequencing of this individual’s genome covered 97.5% of all positions at 15X coverage or higher. Please note that the presence of pathogenic \n",
      "variants in genes not analyzed, genes with incomplete coverage, or regions not captured by filtering strategies cannot be fully excluded. \n",
      " \n",
      "  METHODOLOGY AND LIMITATIONS \n",
      "Genome sequence is generated from genomic DNA that is fragmented and barcoded. Library fragments were sequenced (2x150 base paired \n",
      "end) using Sequencing-By-Synthesis (SBS) chemistry and the Illumina NovaSeq sequencer with a minimum coverage of at least 20X for 90%. \n",
      "Sequence data were aligned to the GRCh38 assembly after discarding low quality sequences. Illumina's DRAGEN (Dynamic Read Analysis for \n",
      "GENomics) platform was used for demultiplexing, read mapping, genome alignment, read sorting, duplicate marking, and variant calling. \n",
      "Technical sensitivity of this assay is 99.10% (95% CI: 99.04-99.16%) and positive predictive value is 99.39% (95% CI: 99.37-99.41%). Variants in \n",
      "4,302 genes with some level of published evidence for a gene-disease association are subsequently filtered to identify: (1) variants classified as \n",
      "pathogenic or likely pathogenic in our internal database; (2) variants classified as pathogenic or likely pathogenic by a ClinGen-approved expert \n",
      "panel; (3) variants classified as disease causing in public databases that have a minor allele frequency <5.0% in the Genome Aggregation \n",
      "Database (gnomAD, https://gnomad.broadinstitute.org/); and (4) nonsense, frameshift, and +/-1,2 splice-site variants in disease-associated \n",
      "genes with a minor allele frequency ≤0.1% in gnomAD. The evidence for phenotype-causality is then evaluated for each variant identified from \n",
      "the filtering strategies listed above and variants are classified based on ACMG/AMP criteria (Richards et al. 2015) with ClinGen rule \n",
      "specifications (https://www.clinicalgenome.org/working-groups/sequence-variant-interpretation/). Variants are reported according to HGVS \n",
      "nomenclature (http://varnomen.hgvs.org/). Only those variants with evidence for causing or contributing to disease are reported. All disease-\n",
      "associated variants on this report are confirmed via Sanger sequencing or another orthogonal technology. Please contact the laboratory for \n",
      "additional information. \n",
      " \n",
      "Risk Alleles: Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. The \n",
      "following likely or established risk alleles are examined and reported if identified in the “reported genotype” listed below. Some variants in \n",
      "these genes are associated with additional diseases and therefore other variants identified in these genes may be included on different sections \n",
      "of this report. Additional risk variants, if identified, may also be included on this report at the discretion of the laboratory. \n",
      " \n",
      "Gene (Transcript) \n",
      "Associated Risk \n",
      "Reportable Variants \n",
      "Reportable Genotypes \n",
      "APC (NM_000038.4) \n",
      "Colorectal Cancer \n",
      "c.3920T>A (p.Ile1307Lys) \n",
      "Heterozygous or homozygous \n",
      "APOE (NM_000041.2) \n",
      "Alzheimer’s Disease \n",
      "e4 Allele- \n",
      "c.388T>C (p.Cys130Arg) \n",
      "Heterozygous with e2 or e3 or \n",
      "homozygous (e2/e4, e3/e4, e4/e4) \n",
      "APOL1 (NM_003661.3) \n",
      "Non-diabetic Nephropathy \n",
      "G1 Allele – \n",
      "c.1164_1169delTTATAA (p.Asn388_Tyr389del) \n",
      "Homozygous (G1/G1 or G2/G2) or \n",
      "compound heterozygous (G1/G2) \n",
      "G2 Allele - \n",
      "c.[1024A>G;1152T>G] (p.[Ser342Gly;Ile384Met]) \n",
      "CHEK2 (NM_001005735) \n",
      "Breast, Colorectal, and Papillary Thyroid Cancers \n",
      "c.599T>C (p.Ile200Thr) \n",
      "Heterozygous or homozygous \n",
      "CTRC (NM_007272.2) \n",
      "Pancreatitis \n",
      "c.760C>T (p.Arg254Trp) \n",
      "Heterozygous or homozygous \n",
      "F2 (NM_000506.3) \n",
      "Venous Thromboembolism \n",
      "c.*97G>A \n",
      "Heterozygous or homozygous \n",
      "F5 (NM_000130.4) \n",
      "Factor V Deficiency \n",
      "c.1601G>A (p.Arg534Gln) \n",
      "Heterozygous or homozygous \n",
      "GBA (NM_001005741.2) \n",
      "Parkinson's Disease \n",
      "c.1226A>G (p.Asn409Ser) \n",
      "Heterozygous or homozygous \n",
      "HFE (NM_000410.3) \n",
      "Hemochromatosis \n",
      "c.845G>A (p.Cys282Tyr) \n",
      "Homozygous \n",
      "LRRK2 (NM_198578.3) \n",
      "Parkinson's Disease \n",
      "c.6055G>A (p.Gly2019Ser) \n",
      "Heterozygous or homozygous \n",
      "MC1R (NM_002386) \n",
      "Melanoma \n",
      "c.880G>C (p.Asp294His) \n",
      "Heterozygous or homozygous \n",
      "MITF (NM_000248.3) \n",
      "Melanoma \n",
      "c.952G>A (p.Glu318Lys) \n",
      "Heterozygous or homozygous \n",
      "MUC5B (NM_002458.2) \n",
      "Pulmonary Fibrosis \n",
      "c.-3133G>T \n",
      "Heterozygous or homozygous \n",
      "PNPLA3 (NM_025225.2) \n",
      "Non-alcoholic Fatty Liver Disease Type 1 \n",
      "c.444C>G (p.Ile148Met) \n",
      "Homozygous \n",
      "PRNP (NM_000311.3) \n",
      "Prion Disease \n",
      "c.628G>A (p.Val210Ile) \n",
      "Heterozygous or homozygous \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 5 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      "SERPINA1 \n",
      "(NM_001127701.1) \n",
      "Alpha-1 Antitrypsin Deficiency \n",
      "S Allele- \n",
      "c.863A>T (p.Glu288Val) \n",
      "Homozygous Z allele (Z/Z) or compound \n",
      "heterozygous Z and S allele (Z/S) \n",
      "Z Allele- \n",
      "c.1096G>A (p.Glu366Lys) \n",
      "SERPINC1 (NM_000488) \n",
      "Venous Thromboembolism \n",
      "c.1246G>T (p.Ala416Ser) \n",
      "Heterozygous or homozygous \n",
      "SPINK1 (NM_003122) \n",
      "Pancreatitis \n",
      "c.101A>G (p.Asn34Ser) \n",
      "Heterozygous or homozygous \n",
      " \n",
      "PGx: Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. Diplotypes and \n",
      "phenotypes are generated using the Clinical Pharmacogenetics Implementation Consortium (CPIC®) allele tables \n",
      "(https://cpicpgx.org/guidelines/) and custom scripts. The following pharmacogenomic variants are detected by this assay: CYP2C19 \n",
      "(NM_000769.1): *2 (rs4244285/c.681G>A, rs12769205/c.332-23A>G), *3 (rs4986893/c.636G>A), *4A (rs28399504/c.1A>G), *4B \n",
      "(rs28399504/c.1A>G, rs12248560/c.-806C>T), *5 (rs56337013/c.1297C>T), *6 (rs72552267/c.395G>A), *7 (rs72558186/c.819+2T>A), *8 \n",
      "(rs41291556/c.358T>C), *17 (rs12248560/c.-806C>T), *35 (rs12769205/c.332-23A>G); CYP2C9 (NM_000771.3): *2 (rs1799853/c.430C>T), *3 \n",
      "(rs1057910/c.1075A>C), *5 (rs28371686/c.1080C>G), *6 (rs9332131/c.818del), *8 (rs7900194/c.449G>A), *11 (rs28371685/c.1003C>T), *13 \n",
      "(rs72558187/c.269T>C); CYP3A5 (NM_000777.4): *3 (rs776746/c.219-237A>G), *6 (rs10264272/c.624G>A), *7 (rs41303343/c.1035dup); CYP4F2 \n",
      "(NM_001082.4): *3 (rs2108622/c.1297G>A); DPYD (NM_000110.3): rs3918290/c.1905+1G>A, rs55886062/c.1679T>G, rs67376798/c.2846A>T, \n",
      "rs72549309/c.299_302del/legacy name c.295_299TCATT>T, rs115232898/c.557A>G, rs1801266/c.703C>T, rs78060119/c.1156G>T, \n",
      "rs56038477/c.1236G>A, rs72549303/c.1898del, rs1801268/c.2983G>T, rs75017182/c.1129-5923C>G; IFNL3 (NM_172139.3): rs12979860/c.-\n",
      "3180G>A; NUDT15 (NM_018283.3): rs116855232/c.415C>T; SLCO1B1 (NM_006446.4): rs4149056/c.521T>C; TPMT (NM_000367.4): *2 \n",
      "(rs1800462/c.238G>C), *3A (rs1800460/c.460G>A, rs1142345/c.719A>G), *3B (rs1800460/c.460G>A), *3C (rs1142345/c.719A>G),  *4 \n",
      "(rs1800584/c.626-1G>A); VKORC1 (NM_024006.4): rs9923231/c.-1639G>A;  UGT1A1 (NM_000463.3): *6 (rs4148323/c.211G>A), *27 \n",
      "(rs35350960/c.686C>A), *36 (rs3064744/c.-53TA[6]), *28 (rs3064744/ c.-53TA[8]), *37 (rs3064744/ c.-53TA[9]). Variant and star (*) allele \n",
      "definitions are available for download at https://personalizedmedicine.partners.org/Laboratory-For-Molecular-\n",
      "Medicine/Tests/Genomic%20Screening%20Spray.aspx (see the link “details here” in the Methodology section of this webpage).  Additionally, \n",
      "variants in G6PD, RYR1, and CACNA1S associated with an altered metabolism status are reported, if identified. Detected variants include those \n",
      "that are clinically relevant based on allele function and frequency information found in tables created by PharmGKB and CPIC \n",
      "(https://www.pharmgkb.org/page/pgxGeneRef). This test does not report all pharmacogenomic variants that might alter protein function. A \n",
      "*1/*1 or wild type result is assigned if no variants that are included in the test are detected. Therefore, a *1/*1 or other result does not exclude \n",
      "the possibility that an individual has a different phenotype that may alter drug response. This risk may vary among ancestries. This assay cannot \n",
      "determine if multiple variants in the same gene are present in cis or trans, leading to an inability to definitively assign a diplotype and \n",
      "phenotype. This test does not detect copy number variants. Genetic information to guide prescribing decisions may be found in a drug's FDA-\n",
      "approved label. These labels are found at https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm  and a table of current PGx biomarkers at \n",
      "https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations and https://www.fda.gov/drugs/science-and-\n",
      "research-drugs/table-pharmacogenomic-biomarkers-drug-labeling. \n",
      " \n",
      "Limitations: Specific types of genetic variation, such as short tandem repeats (including triplet repeat expansions), structural variation, and large \n",
      "copy number events are currently not reliably detected by this assay. Additionally, while genome sequencing covers ~95% of the genome; there \n",
      "are certain regions for which the assay may fail to adequately generate sequence information, such as regions of high homology or regions that \n",
      "are highly repetitive as well as regions with low coverage, mapping quality or base quality. Moreover, not all disease-associated genes have \n",
      "been identified and the clinical significance of variation in many genes is not well understood. The sequencing data is limited to the cell-type \n",
      "(material) source used for DNA extraction. Additionally, not all variants identified have been analyzed. Variant interpretation may change over \n",
      "time if more information becomes available. \n",
      " \n",
      "The initial sequencing component of this test was performed by the Clinical Research Sequencing Platform of the Broad Institute (320 Charles St, \n",
      "Cambridge, MA 02141; CLIA#22D2055652), and the Sanger confirmation, interpretive algorithms and clinical reports were generated by the \n",
      "Laboratory for Molecular Medicine at Partners Healthcare Personalized Medicine (LMM, 65 Landsdowne St, Cambridge, MA 02139; 617-768-\n",
      "8500; CLIA#22D1005307). This test has not been cleared or approved by the U.S. Food and Drug Administration (FDA). The FDA has determined \n",
      "that such clearance or approval is not necessary. \n",
      " \n",
      "REFERENCES \n",
      " \n",
      "Alfardan J, Mohsen AW, Copeland S, Ellison J, Keppen-Davis L, Rohrbach M, Powell BR, Gillis J, Matern D, Kant J, Vockley J. 2010. Characterization of new ACADSB gene \n",
      "sequence mutations and clinical implications in patients with 2- \n",
      "methylbutyrylglycinuria identified by newborn screening. Mol Genet Metab. 100(4):333- 8. PMID: 20547083. \n",
      " \n",
      "Amstutz U, Henricks LM, Offer SM, Barbarino J, Schellens JHM, Swen JJ, Klein TE, McLeod HL, Caudle KE, Diasio RB, Schwab M. 2018. Clinical Pharmacogenetics \n",
      "Implementation Consortium (CPIC) Guideline for Dihydropyrimidine Dehydrogenase \n",
      "Genotype and Fluoropyrimidine Dosing: 2017 Update. Clin. Pharmacol. Ther. 103(2):210-216. PMID: 29152729. \n",
      " \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 6 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      "Antony D, Becker-Heck A, Zariwala MA, Schmidts M, Onoufriadis A, Forouhan M, Wilson R, Taylor-Cox T, Dewar A, Jackson C, Goggin P, Loges NT, Olbrich H, Jaspers \n",
      "M, Jorissen M, Leigh MW, Wolf WE, Daniels ML, Noone PG, Ferkol TW, Sagel SD, Rosenfeld M, Rutman A, Dixit A, O'Callaghan C, Lucas JS, Hogg C, Scambler PJ, Emes \n",
      "RD, Uk10k, Chung EM, \n",
      " \n",
      "Shoemark A, Knowles MR, Omran H, Mitchison HM. 2013. Mutations in CCDC39 and CCDC40 are the major cause of primary ciliary dyskinesia with axonemal \n",
      "disorganization and absent inner dynein arms. Hum. Mutat. 34(3):462-72. PMID: 23255504. \n",
      " \n",
      "Becker-Heck A, Zohn IE, Okabe N, Pollock A, Lenhart KB, Sullivan-Brown J, McSheene J, Loges NT, Olbrich H, Haeffner K,  \n",
      "Fliegauf M, Horvath J, Reinhardt R, Nielsen KG, Marthin JK, Baktai G, Anderson KV, Geisler R, Niswander L, Omran H, Burdine RD. 2011. The coiled-coil domain \n",
      "containing protein CCDC40 is essential for motile cilia function and left-right axis formation. Nat. Genet. 43(1):79-84. PMID: 21131974. \n",
      " \n",
      "Birdwell KA, Decker B, Barbarino JM, Peterson JF, Stein CM, Sadee W, Wang D, Vinks AA, He Y, Swen JJ, Leeder JS, van Schaik R, Thummel KE, Klein TE, Caudle KE, \n",
      "MacPhee IA. 2015. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP3A5 Genotype and Tacrolimus Dosing. Clin. Pharmacol. Ther. \n",
      "98(1):19-24. PMID: 25801146. \n",
      " \n",
      "Caudle KE, Dunnenberger HM, Freimuth RR, Peterson JF, Burlison JD, Whirl-Carrillo M, Scott SA, Rehm HL, Williams MS, Klein TE, Relling MV, Hoffman JM. 2017. \n",
      "Standardizingterms for clinical pharmacogenetic test results: consensus terms from the Clinical Pharmacogenetics Implementation Consortium (CPIC). Genet. Med. \n",
      "19(2):215-223. PMID: 27441996. \n",
      " \n",
      "Caudle KE, Rettie AE, Whirl-Carrillo M, Smith LH, Mintzer S, Lee MT, Klein TE, Callaghan JT, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical \n",
      "pharmacogenetics implementation consortium guidelines for CYP2C9 and HLA-B genotypes and phenytoin dosing. Clin. Pharmacol. Ther. 96(5):542-8. PMID: \n",
      "25099164. \n",
      " \n",
      "Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, Schwab M. 2013. Clinical Pharmacogenetics Implementation Consortium guidelines for dihydropyrimidine \n",
      "dehydrogenase genotype and fluoropyrimidine dosing. Clin. Pharmacol. Ther. 94(6):640-5. PMID: 23988873. \n",
      " \n",
      "Crews KR, Gaedigk A, Dunnenberger HM, Leeder JS, Klein TE, Caudle KE, Haidar CE, Shen DD, Callaghan JT, Sadhasivam S, Prows CA, Kharasch ED, Skaar TC, Clinical \n",
      "Pharmacogenetics Implementation Consortium. 2014. Clinical Pharmacogenetics \n",
      "Implementation Consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin. Pharmacol. Ther. 95(4):376-82. PMID: 24458010. \n",
      " \n",
      "Gammal RS, Court MH, Haidar CE, Iwuchukwu OF, Gaur AH, Alvarellos M, Guillemette C, Lennox JL, Whirl-Carrillo M, Brummel SS, Ratain MJ, Klein TE, Schackman BR, \n",
      "Caudle KE, Haas DW, Clinical Pharmacogenetics Implementation Consortium. 2016. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for \n",
      "UGT1A1 and Atazanavir \n",
      "Prescribing. Clin Pharmacol Ther. 99(4):363-9. PMID: 26417955. \n",
      " \n",
      "Gonsalves SG, Dirksen RT, Sangkuhl K, Pulk R, Alvarellos M, Vo T, Hikino K, Roden D, Klein TE, Poler SM, Patel S, Caudle KE, Gordon R, Brandom B, Biesecker LG. 2019. \n",
      "Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for the Use of Potent Volatile Anesthetic Agents and Succinylcholine in the Context of RYR1 \n",
      "or CACNA1S Genotypes. Clin. Pharmacol. Ther. 105(6):1338-1344. PMID: 30499100. \n",
      " \n",
      "Hicks JK, Bishop JR, Sangkuhl K, Müller DJ, Ji Y, Leckband SG, Leeder JS, Graham RL, Chiulli DL, LLerena A, Skaar TC, Scott SA, Stingl JC, Klein TE, Caudle KE, Gaedigk A, \n",
      "Clinical Pharmacogenetics Implementation Consortium. 2015. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6 and CYP2C19 \n",
      "Genotypes and Dosing \n",
      "of Selective Serotonin Reuptake Inhibitors. Clin. Pharmacol. Ther. 98(2):127-34. PMID: 25974703. \n",
      " \n",
      "Hicks JK, Sangkuhl K, Swen JJ, Ellingrod VL, Müller DJ, Shimoda K, Bishop JR, Kharasch ED, Skaar TC, Gaedigk A, Dunnenberger HM, Klein TE, Caudle KE, Stingl JC. 2017. \n",
      "Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. \n",
      "Pharmacol. Ther. 102(1):37-44. PMID: 27997040. \n",
      " \n",
      "Johnson JA, Caudle KE, Gong L, Whirl-Carrillo M, Stein CM, Scott SA, Lee MT, Gage BF, Kimmel SE, Perera MA, Anderson JL, Pirmohamed M, Klein TE, Limdi NA, Cavallari \n",
      "LH, Wadelius M. 2017. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for Pharmacogenetics-Guided Warfarin Dosing: 2017 Update. Clin. \n",
      "Pharmacol. Ther. 102 \n",
      "(3):397-404. PMID: 28198005. \n",
      " \n",
      "Knowles MR, Leigh MW, Carson JL, Davis SD, Dell SD, Ferkol TW, Olivier KN, Sagel SD, Rosenfeld M, Burns KA, Minnix SL, Armstrong MC, Lori A, Hazucha MJ, Loges NT, \n",
      "Olbrich H, Becker-Heck A, Schmidts M, Werner C, Omran H, Zariwala MA, Genetic Disorders of Mucociliary Clearance Consortium. 2012. Mutations of DNAH11 in \n",
      "patients with primary ciliary dyskinesia with normal ciliary ultrastructure. Thorax. 67(5):433-41. PMID: 22184204. \n",
      " \n",
      "Lima JJ, Thomas CD, Barbarino J, Desta Z, Van Driest SL, El Rouby N, Johnson JA, Cavallari LH, Shakhnovich V, Thacker DL, Scott SA, Schwab M, Uppugunduri CRS, \n",
      "Formea CM, Franciosi JP, Sangkuhl K, Gaedigk A, Klein TE, Gammal RS, Furuta T. 2021. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for \n",
      "CYP2C19 and Proton Pump Inhibitor Dosing. Clin Pharmacol Ther. 109(6):1417-1423. PMID: 32770672. \n",
      " \n",
      "Monnier N, Procaccio V, Stieglitz P, Lunardi J. 1997. Malignant-hyperthermia susceptibility is associated with a mutation of the alpha 1-subunit of the human \n",
      "dihydropyridine-sensitive L-type voltage-dependent calcium-channel receptor in \n",
      "skeletal muscle. Am. J. Hum. Genet. 60(6):1316-25. PMID: 9199552.  \n",
      " \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 7 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      "Moriyama B, Obeng AO, Barbarino J, Penzak SR, Henning SA, Scott SA, Agúndez J, Wingard JR, McLeod HL, Klein TE, Cross SJ, Caudle KE, Walsh TJ. 2017. Clinical \n",
      "Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP2C19 and \n",
      "Voriconazole Therapy. Clin. Pharmacol. Ther. 102(1):45-51. PMID: 27981572. \n",
      " \n",
      "Muir AJ, Gong L, Johnson SG, Lee MT, Williams MS, Klein TE, Caudle KE, Nelson DR, Clinical Pharmacogenetics Implementation Consortium (CPIC). 2014. Clinical \n",
      "Pharmacogenetics Implementation Consortium (CPIC) guidelines for IFNL3 (IL28B) genotype and PEG interferon--based regimens. Clin. Pharmacol. Ther. 95(2):141-6. \n",
      "PMID: 24096968. \n",
      " \n",
      "Nakhleh N, Francis R, Giese RA, Tian X, Li Y, Zariwala MA, Yagi H, Khalifa O, Kureshi S, Chatterjee B, Sabol SL, Swisher M, Connelly PS, Daniels MP, Srinivasan A, Kuehl \n",
      "K, Kravitz N, Burns K, Sami I, Omran H, Barmada M, Olivier K, Chawla KK, Leigh M, Jonas R, Knowles M, Leatherbury L, Lo CW. 2012. High prevalence of respiratory \n",
      "ciliary dysfunction in congenital heart disease patients with heterotaxy. Circulation. 125(18): 2232-42. PMID: 22499950. \n",
      " \n",
      "Porta F, Chiesa N, Martinelli D, Spada M. 2019. Clinical, biochemical, and molecular spectrum of short/branched-chain acyl-CoA dehydrogenase deficiency: two new \n",
      "cases and review of literature. J Pediatr Endocrinol Metab. 32(2):101-108. PMID: 30730842. \n",
      " \n",
      "Ramsey LB, Johnson SG, Caudle KE, Haidar CE, Voora D, Wilke RA, Maxwell WD, McLeod HL, Krauss RM, Roden DM, Feng Q, Cooper-DeHoff RM, Gong L, Klein TE, \n",
      "Wadelius M, Niemi M. 2014. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced myopathy: 2014 update. Clin. \n",
      "Pharmacol. Ther. 96(4):423-8. \n",
      "PMID: 24918167. \n",
      " \n",
      "Relling MV, Gardner EE, Sandborn WJ, Schmiegelow K, Pui CH, Yee SW, Stein CM, Carrillo M, Evans WE, Hicks JK, Schwab M, Klein TE, Clinical Pharmacogenetics \n",
      "Implementation Consortium. 2013. Clinical pharmacogenetics implementation consortium guidelines for thiopurine methyltransferase genotype and thiopurine \n",
      "dosing: 2013 update. Clin. Pharmacol. Ther. 93(4):324-5. PMID: 23422873. \n",
      " \n",
      "Relling MV, McDonagh EM, Chang T, Caudle KE, McLeod HL, Haidar CE, Klein T, Luzzatto L, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical \n",
      "Pharmacogenetics Implementation Consortium (CPIC) guidelines for rasburicase therapy in the context of G6PD deficiency genotype. Clin. Pharmacol. Ther. 96(2):169-\n",
      "74. PMID: 24787449. \n",
      " \n",
      "Relling MV, Schwab M, Whirl-Carrillo M, Suarez-Kurtz G, Pui CH, Stein CM, Moyer AM, Evans WE, Klein TE, Antillon-Klussmann FG, Caudle KE, Kato M, Yeoh AEJ, \n",
      "Schmiegelow K, Yang JJ. 2019. Clinical Pharmacogenetics Implementation Consortium Guideline for Thiopurine Dosing Based on TPMT and NUDT15 Genotypes: 2018 \n",
      "Update. Clin. Pharmacol. \n",
      "Ther. 105(5):1095-1105. PMID: 30447069. \n",
      " \n",
      "Schwabe GC, Hoffmann K, Loges NT, Birker D, Rossier C, de Santi MM, Olbrich H, Fliegauf M, Failly M, Liebers U, Collura M, Gaedicke G, Mundlos S, Wahn U, Blouin JL, \n",
      "Niggemann B, Omran H, Antonarakis SE, Bartoloni L. 2008. Primary ciliary dyskinesia associated with normal axoneme ultrastructure is caused by DNAH11 mutations. \n",
      "Hum Mutat. 29(2):289-98. PMID: 18022865. \n",
      " \n",
      "Scott SA, Sangkuhl K, Stein CM, Hulot JS, Mega JL, Roden DM, Klein TE, Sabatine MS, Johnson JA, Shuldiner AR, Clinical Pharmacogenetics Implementation Consortium. \n",
      "2013. Clinical Pharmacogenetics Implementation Consortium guidelines for CYP2C19 genotype and clopidogrel therapy: 2013 update. Clin. Pharmacol. Ther. 94(3):317-\n",
      "23. PMID: 23698643. \n",
      " \n",
      "Spedicati B, Cocca M, Palmisano R, Faletra F, Barbieri C, Francescatto M, Mezzavilla M, Morgan A, Pelliccione G, Gasparini P, Girotto G. 2021. Natural human knockouts \n",
      "and Mendelian disorders: deep phenotyping in Italian isolates. Eur J Hum Genet. 29(8):1272- 1281. PMID: 33727708. \n",
      " \n",
      "Stewart SL, Hogan K, Rosenberg H, Fletcher JE. 2001. Identification of the Arg1086His mutation in the alpha subunit of the voltage-dependent calcium channel \n",
      "(CACNA1S) in a North American family with malignant hyperthermia. Clin. Genet. 59(3):178-84. PMID: 11260227. \n",
      " \n",
      "Swen JJ, Nijenhuis M, de Boer A, Grandia L, Maitland-van der Zee AH, Mulder H, Rongen GA, van Schaik RH, Schalekamp T, Touw DJ, van der Weide J, Wilffert B, Deneer \n",
      "VH, Guchelaar HJ. 2011. Pharmacogenetics: from bench to byte--an update of guidelines. Clin. Pharmacol. Ther. 89(5):662-73. PMID: 21412232. \n",
      " \n",
      "Theken KN, Lee CR, Gong L, Caudle KE, Formea CM, Gaedigk A, Klein TE, Agúndez JAG, Grosser T. 2020. Clinical Pharmacogenetics Implementation Consortium \n",
      "Guideline (CPIC) for CYP2C9 and Nonsteroidal Anti-Inflammatory Drugs. Clin Pharmacol Ther. 108(2):191- 200. PMID: 32189324. \n",
      " \n",
      "Wilke RA, Ramsey LB, Johnson SG, Maxwell WD, McLeod HL, Voora D, Krauss RM, Roden DM, Feng Q, Cooper-Dehoff RM, Gong L, Klein TE, Wadelius M, Niemi M, \n",
      "Clinical Pharmacogenomics Implementation Consortium (CPIC). 2012. The clinical pharmacogenomics implementation consortium: CPIC guideline for SLCO1B1 and \n",
      "simvastatin-induced myopathy. Clin. Pharmacol. Ther. 92(1):112-7. PMID: 22617227. \n",
      " \n",
      "Witherspoon JW, Meilleur KG. 2016. Review of RyR1 pathway and associated pathomechanisms. Acta Neuropathol Commun. 4(1):121. PMID: 27855725. \n",
      " \n",
      "Yang SK, Hong M, Baek J, Choi H, Zhao W, Jung Y, Haritunians T, Ye BD, Kim KJ, Park SH, Park SK, Yang DH, Dubinsky M, Lee I, McGovern DP, Liu J, Song K. 2014. A \n",
      "common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat. Genet. 46(9):1017-20. PMID: 25108385. \n",
      " \n",
      "Zariwala MA, Gee HY, Kurkowiak M, Al-Mutairi DA, Leigh MW, Hurd TW, Hjeij R, Dell SD, Chaki M, Dougherty GW, Adan M, Spear PC, Esteve-Rudd J, Loges NT, Rosenfeld \n",
      "M, Diaz KA, Olbrich H, Wolf WE, Sheridan E, Batten TF, Halbritter J, Porath JD, Kohl S, Lovric S, Hwang DY, Pittman JE, Burns KA, Ferkol TW, Sagel SD, Olivier KN, Morgan \n",
      "LC, Werner C, Raidt J, Pennekamp P, Sun Z, Zhou W, Airik R, Natarajan S, Allen SJ, Amirav I, Wieczorek D, Landwehr K, Nielsen K, Schwerk N, Sertic J, Köhler G, Washburn \n",
      "\n",
      "Accession ID: PM-23-NXXXX \n",
      "Name: LAST, FIRST \n",
      "Page: 8 of 8 \n",
      " \n",
      "______________________________________________________________________________________________________________________________________________________________________________________ \n",
      "Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director \n",
      "https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\personalized-medicine\\molecular-medicine \n",
      " \n",
      "CLIA#: 22D1005307 \n",
      " \n",
      "Personalized Medicine \n",
      "LABORATORY FOR MOLECULAR MEDICINE \n",
      "65 Landsdowne St, Cambridge, MA  02139 \n",
      "Phone: (617) 768-8500 • Fax: (617) 768-8513 \n",
      "J, Levy S, Fan S, Koerner-Rettberg C, Amselem S, Williams DS, Mitchell BJ, Drummond IA, Otto EA, Omran H, Knowles MR, Hildebrandt F. 2013. ZMYND10 is mutated \n",
      "in primary ciliary dyskinesia and interacts with LRRC6. Am. J. Hum. Genet. 93(2):336-45. PMID: 23891469. \n",
      " \n",
      " \n",
      "Report Preparation by: First Last \n",
      "Final Report by:  First Last, PhD, FACMG \n",
      "\n"
     ]
    }
   ],
   "execution_count": 24
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## Html",
   "id": "504e1a5db0353e09"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-05T13:12:21.808467Z",
     "start_time": "2025-06-05T13:12:21.658539Z"
    }
   },
   "cell_type": "code",
   "source": [
    "\n",
    "for page in doc:  # iterate the document pages\n",
    "    page.read_contents()\n",
    "    text = page.get_text('html', clip = None).encode(\"utf8\")\n",
    "    print(text)  # write text of page\n",
    "with open(\"demofile.html\", \"wb\", ) as f:\n",
    "    for page in doc:  # iterate the document pages\n",
    "        page.read_contents()\n",
    "        text = page.get_text('html', clip = None).encode(\"utf8\")\n",
    "        f.write(text)\n"
   ],
   "id": "dc4a72fac266fa84",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:743.5pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:752.9pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:763.9pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:762.3pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span></p>\\n<p style=\"top:99.4pt;left:41.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#808080\">Name:</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> LAST, FIRST </span></b></p>\\n<p style=\"top:99.4pt;left:248.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#808080\">MRN: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">XXXXXXXX</span></b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> </span></p>\\n<p style=\"top:99.4pt;left:417.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#808080\">LMM Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">PM-23-NXXXX</span></b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> </span></p>\\n<p style=\"top:113.7pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">DOB: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">MM/DD/YYYY</span></b></p>\\n<p style=\"top:113.7pt;left:149.2pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Referring Facility: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">HS</span></b></p>\\n<p style=\"top:113.7pt;left:465.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Specimen: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Blood, Peripheral</span></b></p>\\n<p style=\"top:128.1pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Sex at birth: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Female</span></b></p>\\n<p style=\"top:128.1pt;left:149.2pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Referring Physician: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">LAST, FIRST</span></b></p>\\n<p style=\"top:128.1pt;left:476.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Received:</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:12.0pt;color:#000000\">  </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">MM/DD/2023</span></b></p>\\n<p style=\"top:142.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Family #: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FXXXXXX</span></b></p>\\n<p style=\"top:142.5pt;left:149.2pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Copies To: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">HOSPITAL SYSTEM</span></b></p>\\n<p style=\"top:142.5pt;left:527.3pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1 of 7</span></b></p>\\n<p style=\"top:156.9pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Test Performed:</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#aeaaaa\"> </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Expanded Genome Screening</span></b></p>\\n<p style=\"top:156.9pt;left:461.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">Test Codes:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:12.0pt;color:#000000\"> </span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">lmWGS-pnlFv4_L</span></b></p>\\n<p style=\"top:251.1pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">VARIANT SUMMARY </span></b></p>\\n<p style=\"top:275.6pt;left:75.7pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">REPORT SECTION</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:270.0pt;left:185.1pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease, </span></b></p>\\n<p style=\"top:281.0pt;left:185.1pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Inheritance</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:270.0pt;left:288.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene </span></b></p>\\n<p style=\"top:281.0pt;left:288.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Transcript</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:275.6pt;left:369.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Variant</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:275.6pt;left:441.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Allele State</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:275.6pt;left:504.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Classification</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:302.8pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#00637e\">A. MONOGENIC DISEASE FINDINGS</span></b></p>\\n<p style=\"top:302.8pt;left:348.3pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">None identified. </span></p>\\n<p style=\"top:328.1pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#00637e\">B. RISK ALLELES</span></b></p>\\n<p style=\"top:328.1pt;left:348.3pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">None identified.</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:413.7pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#00637e\">C. CARRIER STATUS VARIANTS</span></b></p>\\n<p style=\"top:353.4pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Primary ciliary </span></p>\\n<p style=\"top:364.5pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">dyskinesia, </span></p>\\n<p style=\"top:375.4pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:359.0pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40 </span></p>\\n<p style=\"top:369.9pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_017950.4</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:359.0pt;left:369.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.248delC</span></p>\\n<p style=\"top:369.9pt;left:369.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">p.Ala83ValfsX84</span></p>\\n<p style=\"top:364.5pt;left:441.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:364.5pt;left:504.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:397.1pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Primary ciliary </span></p>\\n<p style=\"top:408.2pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">dyskinesia, </span></p>\\n<p style=\"top:419.1pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:402.6pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11 </span></p>\\n<p style=\"top:413.6pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_001277115.2</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:402.6pt;left:369.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.12363C&gt;G </span></p>\\n<p style=\"top:413.6pt;left:369.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">p.Tyr4121X</span></p>\\n<p style=\"top:408.2pt;left:441.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:408.2pt;left:504.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:440.9pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Short/branched chain </span></p>\\n<p style=\"top:451.9pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">acyl-CoA dehydrogenase </span></p>\\n<p style=\"top:462.9pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">deficiency, </span></p>\\n<p style=\"top:473.8pt;left:185.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive </span></p>\\n<p style=\"top:451.9pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ACADSB </span></p>\\n<p style=\"top:462.8pt;left:288.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_001609.4 </span></p>\\n<p style=\"top:457.4pt;left:369.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.303+1G&gt;A</span></p>\\n<p style=\"top:457.4pt;left:441.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous </span></p>\\n<p style=\"top:457.4pt;left:504.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic </span></p>\\n<p style=\"top:499.1pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#00637e\">D. PHARMACOGENOMIC</span></b></p>\\n<p style=\"top:510.2pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#00637e\">ASSOCIATIONS</span></b></p>\\n<p style=\"top:504.7pt;left:358.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">See below.</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:540.6pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">DETAILED VARIANT INFORMATION </span></b></p>\\n<p style=\"top:554.0pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">A. MONOGENIC DISEASE FINDINGS</span></b></p>\\n<p style=\"top:567.2pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">This test did not identify any variants with the potential to cause monogenic disease in this individual. </span></p>\\n<p style=\"top:590.6pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">B. RISK ALLELES</span></b></p>\\n<p style=\"top:603.9pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> This test did not identify any reportable risk alleles (see methodology for the list of risk alleles included in this analysis).</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:626.1pt;left:36.8pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">C. CARRIER STATUS VARIANTS</span></b></p>\\n<p style=\"top:639.3pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">This test identified 3 carrier status variants for autosomal recessive conditions. Autosomal recessive disorders are caused by the presence of </span></p>\\n<p style=\"top:650.3pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">pathogenic variants in both copies of the same gene. Being a carrier of these variants does not put this individual at risk for disease but may </span></p>\\n<p style=\"top:661.3pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">impact disease risk in this individual&#x2019;s children. </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PLEASE NOTE: The presence of a second pathogenic variant in one of these genes cannot be </span></p>\\n<p style=\"top:672.2pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">definitively ruled out due to the technical and analytical limitations of this assay. </span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008000\"> </span></b></p>\\n<p style=\"top:694.2pt;left:36.8pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FAMILIAL AND REPRODUCTIVE RISK:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">The risk to this individual&#x2019;s child or future child of developing </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">an autosomal recessive condition is </span></p>\\n<p style=\"top:705.2pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">dependent on the carrier status of this individual&apos;s reproductive partner(s). Two carriers have a 25% risk for having a child with the associated </span></p>\\n<p style=\"top:716.1pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">disease. First degree relatives of this individual have a 50% chance of also being carriers of each of these variants. Other biologically related </span></p>\\n<p style=\"top:727.2pt;left:36.8pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">family members may also be carriers. </span></p>\\n<img style=\"position:absolute;transform:matrix(.8674699,0,-0,.867284,-7,29.299973)\" src=\"data:image/png;base64,\\niVBORw0KGgoAAAANSUhEUgAAAz4AAABsCAIAAAAkB/DuAAAACXBIWXMAAA7EAAAO\\nxAGVKw4bAAAzs0lEQVR4nO2dB3gVVdqAE7oiKpa1/K7dXQtICQRCEQQUUVFEFBtY\\n17WXtWFBwYaLlPTeeyOFJLRAIJQkpBfSe++91/t/M+fm5GRa7g1BuOz3PvPkucyd\\nOW2uz7x+p+mpEARBEARBEB1B72IXAEEQBEEQBNEUVDcEQRAEQRCdAdUNQRAEQRBE\\nZ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE\\n0RlQ3RAEQRAEQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAE\\nQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAE\\nQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAEQRCdAdUNQRAE\\nQRBEZ0B1QxAEQZDLja+iTj/g6SY4LnahhLwVcRRKtTMxAT4/eyAEPrtnZ41Vgiy0\\nBWj68GEUzUKvP1NZAR8WB/iNojDnD6obgiAIglxuSKqbsmr89Yy53IyobtAs5Azb\\nPpqnf4kY8CjVrbOvL7Wuwfxc1obwkzsS085W17X19I5tyRAEQRAEGR1ETaipAOBt\\nNOzExpyo6JBbSPQLjjOVFazfwHk2fUkdJFlAguKvaLJwgGCRk5JRN7ZsAt8Sp8DW\\nBbIbUd1oLeBiUlqqYrTYguAfSZMUQy7qRv45Yu3YLMTl19xitVO3/oGBtPrGncnp\\nj4cdu9nNX9/WXc/aTc/W/ToXv6X7j/wUlxJTXdfT369VmgiCIAiCjC0K6sZ6Bmsq\\nrKjBxWKFIlIivp2KCJUhgXWJQ4DEUbRSN0Hi5KRkYeTUjWRB76JOphruT9RcJUsu\\nVjdxGUjZxLXTpDE16TLWSN06+vpS6xttMnLWHT5xnYuvnpWrng1nbHp2Huq/5IOV\\n61WO3qvCju5JzYirqWvt6dEkcQRBEARBxhaFDlNWKYg6kFgUuYVGkkiIiI1vEcjt\\n9DyREtauWF0T3846pfJYN1atBIUhGYlzUVY3kgj8pR9o+mzuRONImmyokr2eVTe2\\nDOx5ce3INSQd0uZyz0IZJXXrGxjIbGzalZK++kDETW77OF2zduMszd6TEzUbtwl2\\nHre4B0xx8Bo0OU/uW2tO7Ka7+C4LOfJTfGpCbT3G4RAEQRDkr0SsbtTJxBEgVkHY\\nQJ1kKIvcToJSKinRIV+R86yIsN2FI6obvVgh+gVfkRToXcodpjTYRi6jES+VVLyQ\\nlPyB4d2+kuomKANFrnbsvXLPQhlpdctqbHLMzns+/OS1zr6cltm4DxmbtdtkB6+H\\n/ELfjowOLCqp6ug8Ulrx8ZnYuQFhVzp6c24HF9vzDmfD6R2cfDzsqFlaVlp9Y//A\\nwIgFQhAEQRDkPBF7GEWsC2wES3AL25NIbGYU6samrEnUjWYqGIcn7lvUSt1UzBA3\\nYkgK6kYv+IvV7QENpkFIq5tR0GESVxvqDOWczO1+35DPouLDissq2jv6h2tYTUfn\\nsbLKb2OT5uwLG8dd7zrYl+rJh+tcFwUfru7oHLFACIIgCIKcJwrqJqcaCreoGGsZ\\nscNUrG7sefZ6SbmhwTC2JCQIJy6bVh2mquFzDtjzAh+laNthqmIifJqom9yzUEZa\\n3e7zDuZiZoMxtgd8978VGb2voLi2s2vEFJu7ew6UlH94Onb2vrArnbzJPAawt4f8\\nQsrbO7QqHIIgCIIgo0DBw+TmHwhuYfs32aibeGQ9jUspqxt7KKibOMBG+y7FmWo1\\nTYGtFLEl8XlBCUc3TYG0pybqJvcslJFWt3u8gkC27vUJ/vRMXGhxWXlb+4BK677O\\nms6u8LLKb88mzfQPBRF80Gc/qhuCIAiC/AUoh9BYY6CuIL5FMOeUTUHyvJy6UbmB\\nC9jB+Fqpm1ymmi8OwpZEcF4lNRSPoO3iILS0mqib3LNQRlrdtsWn+OcXt3SPzRTR\\nzt6+sOLyXcnp9Z3dY5IggiAIgiDI/ybS6pbW0Jhc15hQWx9bMwZHfG19cl1DekNT\\ndx9ONUUQBEEQBBk90uo22z/sCgfvqxx9xuqYYu+1MOBgJXaYIgiCIAiCnAcy0xR8\\ngtULgozVYeM2wy+kAtUNQRAEQRDkPJBWt3u9g9Sre4zVYe36kC/OMEUQBEEQBDkv\\npNXNJTvfODXTJC1rrI69qZnuOQW4RT2CIAiCIMj5oN328wgix8DAQHVDe3lNW2Vd\\nW3dP31+TaVVde0lVS1//aHbpaG7rLq9tgwI3toy8WiGCIAiCXCJIq9ux8sqQorKQ\\n4rKDJeXNzBIhec0t+4tKQ4vL9heVFbS00vPVHZ1hJeUh3PnSxNp6+iLt6O2LgKT4\\n8ycqKjt7ld7o7V29B6OL958shCPsdGFd8whbLxRVtgSfKAg5VRgcWRCZVD5wSe6y\\n1dTWfTa9yu1A5m6PpB0uCXu8kj0OZcekVdY2doxiqbxLmZb2npUfBd37vLvBZp+k\\n7BrJa+AZxaZXw/OCp0YPeNbhsaXJuXXgUlrl6B+R9+BLHnevdzP3SxlFgXc4x9+7\\nwe3u51y/MTszitsRBEEQ5KKgONbNxm2qg1dKXQM9/9/kc3pWLmTsmum5oX0bDpSW\\nTyab0Fu5vHLsVN+gRRW3tt3rFcRtqGDtOmOksW4FFc3TH7XRMzDVm2cGf429k5WL\\n/uqPR/Qe3qs330xvlvGcTd69l9jKI9nFjdvtY2e96jV+kSVXo7mmenNMuAM+L7SY\\nu9mnqr79YpdxLGlq7f77Wme92SZTH7E+kyLcS4QAz2jtf0LheXFPDY55/AEfDM0n\\nLrGa85q35b7Uji6NetXbOnsNX/fh2nOu6QMvuDeMJPpiPt4Tyd3+sPGmHw5rey+C\\nIAiCXCyk1e2fvvv1bLmZoVOdfFLqh9Ttz5R0zs/4fejN07Pp+YOlFVOcfMhM0tci\\nTtNt5kva2u7z4ZOydZ/pH6o8w7SwouW6x+1Ba/TAdeabrfggsLVDdk3gjMKGm9Y4\\n6i0w1+PFyOBN30tK3TwPZd+z3o3TUDjmgq6ZTlpqffUK20mPWBPXvGa5dWFF88Uu\\n5lgC6nbHOleo3VWP2kSlVkpeA8/oma8PcLq2yFJ/8OAeIrSSoTmc1zc0/4/JKU2i\\nkZ09fSveCwDxAlmc+5p3k5YRO+Az41Pcs5hruvnHcG3vRRAEQZCLxaWqbgstrlxu\\nfTSuVO7i3Z6J3CvfyPISVDfHkIwrl1iRwNLNTzq+/XuEX0ReSm5dbkljal7dwaji\\nXxziXtp6uLy27WKXdCzRTt0MzD7ZeyopuyY2o+pkcrmJd/K9G9y5R29oftVSKzij\\nSY7RqZVPfBK87N19gSfyR1FgVDcEQRBEF7kU1Y2LxBhZ6BmYvr5dejOvjq7eJe8G\\ncEGaRZecuoGO3PKkE4khzd3sE5NeJXlZd29//6gG11+yaKduc0z3eA3rEA8+WXjl\\nchtib7vcEzXMtK9/oKdXi+de39yVmFVdzXdVo7ohCIIgusilp27zze7Z4Hbr05z9\\nXLPSNimnTnzlgaiiiYstJy+1nr3ZZ9xiK4G6NbV0xaZX+UfkWe9L2+uZZOqd4nEo\\nOy6juqNTYhAV3JVR0OBzNNfMJ8XYK9kpJONoXGlt09DAKahJcVVL6OlCK//UPZ5J\\ntoHn4HORfF/n278c43pIF5jftMYhLrNaob5i8suaXMIyv7eO+dz41Da72IDj+TUN\\nw1oMChZ2pmj/yYKCcq4A2cWNUJ4fbGLgYvcDWSVVLeI04VlkFjbYBaV/axkNyf7q\\nGBd6qlAwp7Kyrj3sdBG0am0jl11JdWvwyQLbgLTc0iYu08aO6LRK36O5Vv5pezyS\\noKG8j+Qk59R0D590op26zTX90yNJUIY74XZQXkOz7XZnycnO7t5jcaXBkQUZhQ18\\nFl0R8aV2gedOJ3Nj6TKKGvafKoTHkZxTy05S6erpO3uuyswn9WvzM58bn4YqB57I\\nb2juPJteZbDZR3+hObSGilG317eFd3b1hp8t2eES/511zC63xIi4MsGQu/au3pTc\\nOsjOLujcXq8k8lM5mVTe2CIcYwfuHhRZcCKxHOy8raNn/8lCKAA8UxPv5ISsavKf\\nBiR+KKr4N6d4OG/qk5KQWX2ZeTyCIAhy4bj01M3AdOVHQe/tOE7GsG8bfItTIPEX\\nvzuk9/Deh1/12mIZzcXnhqvbH84JUx+x1l9koZ4cMM9M38hi+mP2G7YchCzYpMA2\\nPttz6tannMYZWfKD0kzBG8YZWYDwkQv6+gbgJf3PF9wnLrGiqektsHhje7jkmxZc\\n55Yn+RF488z+Y3JaobICoMGsA87ds95NH+6dZcxNv5htMmGx5YI3/U4xQ/4jEsqu\\n5EfLfWseBaZ113MuevP4sXQGpvoLLR562etwTAmbbF//wJ9uidzsAXAUSBOOOSaT\\nl1o9+n5gUtbQJFBwI0gWsvOPyD2VXDHjZc9xi7kG8QnPhW+3mJ+5AjJdONSe0EQ3\\nrrZ/d0dEHeO456lu0HS3PunENd0Cc5uAc+RkVX37Pze4Qx3f/T2ivKbt6c9DuMjc\\nbOPP956Eb7+ziuHqtcDsnd8jqLpV1Xe8/eux6Y/ZsVUG0V/0jj/8YPixccYkfbW6\\nGZi+8v3hL01OT4ZHPJf/Dcw3u3KZNSTCzngFQbx5jcM4+FHxtsf9WhaaQ01XfBAY\\nmzFM0N/6+Rh8e+/zbvGZ1Zu2hY+H3+cc7gHBjTestrf0T+3r7//oz8hJ8L8cc0zJ\\n+esft//DNeHSnCKNIJRZGyzhGPGCV7b4syd/sztJzrMHnGSvsduXwH4L/6Rffbbz\\noPh29gIgJbuKnIeL5dIUZK2Qo0KBSWEkc5EsD3uwzSKoFFxPv1r1LxfxvewF4izo\\nt7QwkIjmbStoTLnyC65ky8k2CDnPFhgqDmdCIrMFnyUht7PlB+B65fak0GfHfksy\\npQ9RsoVJFcQ/NvFTk/txsmUWV5ltIjYdWrYRqybg0lO3uZy6nUmtmL7SDl5s8173\\nEawSkpxTe+NqBzCbHS7xe71TSNckq26f7Dyh99CeaStsb3vW5e7n3abzkTzOCWYb\\nv/ZTOF0DDN6U8AblvMfQbPwiy/97xvmO51yvgUxn7NlmFU2ucdifPgFunM8Nn79x\\njeNd692uf4LL+qlPgyWjJL7hufwIPItJS6xCThUqNfxwnMMyp4A6GJiCJC19d98z\\nX4Td94I7FAxa474N7jklTeSyY3FlYCGgMjNf9Xxwgxt8uOEJh6uhzNAIoFZzTWe+\\n7FlUOaSnvzvH63PtYzptuc2j7wes/SLkjmddePkwMXzDhw62CzqRPwmSnWvyzo7j\\ni9/2gxYgUy+9j+TAt5u2HobPV6+yu32dy13r+SYi7TnHZItFFM1LW3Xb7TnUYdrS\\n1v2tZTQYNpy/+1kXEu1T8aG4u59zhYwM3/Lb+P0hvZmch+k9sOezXZHwLSfus8GK\\nTN749RjxHkj/X79HcNcYmoN+LX8vcO0XoXdBCvxDhJOQ+6+OcbF8L7Za3RZa3PaM\\ny+TFllOXWd/4hMOkJZbqxpxj8oN1NC2hW1gm/H6mPGJ961pn+FH97UlHfd4yodEW\\nvOXHRjE3bzuqN8d46grblR8HT5xnNm2lLfwCuYsXctoHGv3hrkg4P3W5zfWrHcYv\\nJOfNr1puc+TsMO1GkEsNZXVjFYF9c0uaEPsKlJQz+vaS/Fbw+mezkCyPQMKUc1Qu\\nsCB34gFyKimZvqQ6UJeV/Fbc7GwW4pakGqFJ22qrbprUbnTqxioae42kugl8SOGX\\nIGhkrdRN3Ji0uSTTkasyW6PLVN0MTJe9H9DY2s2F1gxMxxlZBEUWsJf95hSvZ2D2\\ntzWOeWVNuzyTSIiLVTe/Y3kWvqlgDzkljeAxCZk1r4B8wGWGXCcmnCSXpeXX3bCK\\nk54JS6xAcQrKm/PLm2POVZn6pISeLlRxvXV9qz4MIpMfX9sWnlFYX1jRnJRd434w\\nyzkkQ7KH6w/XBLL2BxhVWp5EV68k2SWNtz3lBDdOWmq11zu5o6sX2g/KufAdP944\\nTT7mTQU4Fl8GUsgFGg3NwUjcD2Wl5tVCTTdtCyfKqL/Q3O2AetGWuMzqm0A053Ex\\nJOfQzM7uXnguKbl1M1/15uzEwHTv4PIr0MKTlnJxNfCJK5ZaPfWfkJ/szm4xi4rn\\nO3y9juQ4BKfHpFVBg8Mzik6tBB8i7XnLk47lNWr/007dFpgbvOkHyrX556Mv/3Rk\\n4dt+UC9Qt9mvebNzU0Dd7lnP9aJOfMR64kLzOZt9vrGI+s4yel9Enoqo2xxTENM3\\nf1OrW3pB/XT+mY4zsrTcl9rV0wePCU7ev9Gd74o1/8EqhiauVjd+iusrW4+cTqmA\\nR+YTnvsPkGa+dvdvcK8YtNuz56r2eCZFxJdlFjUUV7ZAmlttYiYutSJCFhQ5NE/i\\n9e1Hud/MIk4BP9h5Ii6jOjGrBso8YTE/gtOIOw+/7ajUipSc2j/dEtUj/AxMP949\\nLA6BIJcakg5BodGFWcODauRtKj5DXpb0rUxfbNQMyLtNrBfiBMlLVE5EBNePmKM4\\nfWKBpMDkM3mdkyvFL1qSoCB0xBZmlpRiEt1RVh9BFgKDIZnSrEfRtiOWXyUKI9Fc\\nSDrno240qCnQMpIF285yJWTVjeRCn5dA3cSBTMkGYW9kvyUZsQVgn4Wkukkq4Ihh\\nSDn+OnWb4R+qvK4bVbel/w7o7un3PJw9ng8FPfl5CE2wqa3b8E0/vVnGG78/BP/8\\nr1uiWN3EFJS33AJuNN9sylLr4/Fl5OTR2BKuG5SbymoTHisR8Khv7pzxiheZKPqn\\nh0YD57+xiCZecvs6l8LKofFwYGNgk1ttzm6ziyUHOITP4RxSqx0uCSCjkIvR234t\\n7UOddKa+qfyyZ+ZzN3k38HEdtbottACjdQnLZIv60MueZKW07Xax5OQ2u7Nkhbzn\\ntxxgC7kTGo3v9Vv96X5yRq1uRpyFfGU29PjkOJVcPm2lHVRzyhKr2MF5GNouDsKV\\nYbYxf5hw1V9gcf1qh52uiXWNQ0FWtbrxj3jZewG1jcPir2J1OxRTTKJrd65zKa0e\\nWjL6Qy68ynX1rvs6jDq3Wt1A0TZ6sBfbBJ7TW2gO5bnhcfvELOm1hQF4IoZv+pIu\\n1x0u8fS8Wt3gab7j39apXt2mr29g9SfBXMUNze9e71rBTC5+5/cI8r8H89/wVW52\\nBLm4KKgbfZWK36ly6sa+DiX7T8mrekR1o+918Qte8voRc1RWN9Xg61agXJKtIW4o\\nogKCVzVbJK3UDS4mB20HkhQ5M4q2HbH8kl9BOoKo4ejUjf7ABL80bdWN5CLo4x6d\\nurFZs9+KE4HztISS6iapgJe4unFL+M4LCKvpVNpxiFW3jq6+irq2+190h7fdtOU2\\ndInXkFOFZDiUxyEu9x2uCcrq1tXT19rRU1rT+uBGzmwmLbaio8ESs2uvXmHLBTwW\\nWkBGxl7JVXXD1sht7+wFXSBRtOmP2X9pciazqEGcBctXZmeIuv19nUsBM5WhvqXr\\nQdAa0hFJ1hyeuXfdf0IH+OFoG7YcIC71wc4TbGqHootJ+Ofvg32IanUzNL9xtYNg\\nxwISpASP+Y7v7e3p7QczI2dMfIdtNuAfkccNZZtn9o8XPchwLrW6LTAHiz2XXy9b\\nvQEuEtne2ROfWX3zk45QjMmLrU4lqy1NO3UzsrjxCcf7X/T45wse921wv2G1A7Go\\nCYstwVMjE9WLg9CoG3zrdSRXkJpY3U4mVXAPdIH59Y/bZxc3DjXO94f4AWpmL/9w\\nmGopnabwwpaDrK2eSa2cuswGMp2+yi4qTaIiPX398KOqbep44uNgom5bbYdGZHLq\\nxg+G+2zvKfaubyyiyPkXvjvIhmzN/VJJv+0NTzjItjyCXAIoqBurO4K3kWT/o+RL\\njsK+mOU6TOlbU/mFqhKp2Ig5ynWYimNXYjESXyAosGQbsmoo2Q0ndhRaYFJaKBv5\\nwAYFNWxbzTtMVVIWJWDU6sY2gqBgch2m4sanz5oaLfnA/gYkW5gUSdwgcs2l8N+C\\nZJVZ0R+xw1T8RMRcWHUram270yNA39Z9QeDBAyUjLNbFqhtZjPdL09PcWLT5Zl8Y\\nq9+Cz34Vpvew8YMbPWr4uZCS6tbb238ysfx7q2hwlwdf9rzzOdc7nnOdzA+0B3U7\\nMqhuIDdcPyOkz7/s9QzN7nne7ReHuLKaoQCMiU/yOF4p+EH6pjestn//v8dTcmvl\\nqrDdLpZ2mKYyHaZgSE9+tv/O59zue9HjmlV2evysiOe/5oJhre09S97dpzefywKK\\nuuKDoOXvB8IBH+Zs9uG78yxuWOOYlMNlStXt1qecBI712o9H+MCSWt1qGzvvf8GD\\nK7mRxQMvedJkH/0gaOarXvp8d97ta51JtEmtbobmNz/lVN0gjIyCwh6OKf7S9MzK\\nD4MgqTvXu4JKTuD7bUHdTo9O3eaa/uwYB3Wva+qEo6SqxTk0gzN13qXmbPKu5Nfv\\noOo2zsjiVLJwhwaxulXUthPdh/Z87uuwk0llCVk1uz0ToUjQjJC4mc+QxQ4tDvLT\\nsMVBzqZXXf2oWt2iGXUDV/7DJeHZL8Me3uR95/Pcj2oqXMY/yh9shvphqbp9wwwE\\nBH60PUvObxq+5I11wDny9K9bjeqGXNIovK7YF7YgRiVnQuKuJYqyugm8gS2VOGAm\\nPjkKdZOL7sh1KY6tuinYIVxMtAauIfViK6Jb6sZ+JchFUt0kw2b0WZOq0UjkRVc3\\n6tbwzwuobvd4BepZuepZu02190xmNsLakXROz9JFz8Zdz9LVhN0Iq6R8MvictRt8\\n+9LRk3QjrNrOrj+T04MKSxu6uOiOckccq24t7Zy6peTVkTP3b/QApUjO4f85x+R3\\nZ3X/lFjd2jp7P9l9ctpyG87JZu6FvxMWgbFZkrXiWHUDqus7/v3H8cn8bEr1SPa5\\npgve8s0qVle5q6dvp1vCjavtych3ft1/k/972ik4UnoNWPugdC4RbpqCZQg/YI7Q\\n3z8AFlJS3Vpe27Zpe7i6H5NXt4aWrnlv+nGJL+KLMWPP0MGV31Rvlsm1y63P8v2S\\nw9Qtb5i6vbp1mLpBRretdVbvNgGFZ5N9mCRrfMtK27wyLphH1e2Wp5yq6oepW2Nr\\n1+vbwicvteK6NeFeA5OJRpZce3LDts5L3dhpCoSTSeXTH7MnD8IplOsOVqvbfPPx\\niyxPizbXEqubihuZl309WdjZwHTKMuurV9np8wP7oBE2bDnQ0DwU95Vb1w2amvv9\\nMOoGSZt4J9+0xoFrN3gos43HL4S6W44jK0LLqZv5MHXbOqhurw1XN6tBdbseo27I\\npY3c60ryzUq1RqxT1AxUWnbqUY0TXKnsdufTYUo1TiAKAj0VMFYdpuSzpCexWcwa\\nHMsFV4pV4y/oMCWqxFZQW3VTkEXVcJMTDK0TQJ8dO8QNPovVbcw7TOFK2ixyVWat\\n8YJ0mPrmF9tm5tpm5Tll59d3Db3tUuoabPjz8O25hiZ6vrS1zTE7H87Dt8fLq8SG\\nVt/V7ZVXWNOhtNGkWN1UpKuL9yrXA5m73JPg5Xfzk47pBWprGa5uXLaW/mn687iR\\nQ1OWWr/+81H/iLyotMrw2NK717txHaZLhqmbipeqwzHFG78/9LcnHNTbM8wyfu7L\\nsO6eoUXLknNqP9kVqe6549cKnrHRQ3IH0pj0qqlk1Pl8s69MpTc158ZdGQypW3Nb\\nt9E7/qRPcOWHgTtdE393imePXx3j9ngmkdmgmqsbmO69UGU+Xgi1k0jWIc7EM5kM\\noVNQN66b72FjvQUW16y0+3hXZPCpAjAbr/Cc61dzzXU+6iZYHASANueagvfaz/ml\\nVUahbjUNHZtBnhZyy5dcs8ruykdt4ckabPb5zSkONJS9XTN14/7LPBBVNNmIG5w3\\nbpHl2i9D3Q5mnUquOJlcsfhf/mTCB6ob8r+AnLopTwJVVjfxm1hwRvA2Ja86QX+r\\nckhGbpqCXI6SqieQlVGrm9gFBWcETjBLJvAmjq6R8rPntW1bTcpPs5ObpiD4lj5r\\nUiM5U1Ge1SsIwimMMqTPjnVB1fBnqpW6iYfNjW6aAhtNvIDqNlb09g9kNTYbp2Ua\\nBh6c5R9apvE0BapuHodzyPbtBm/4zHjVU2+OyctbD/cN9o2y6tbXP8CZwZeh3FJn\\n8IL86QhdZx8E5Z7npdVNXc6+/riM6ic+3c8PpOPGpwu6Iwe4uQ5NH++O5Fb24ofz\\nB58sEKcD8vHo+4HENf/vaWGfJuH9/55go25QSG7CJu8Q/9oRodyemqtbZ3efeqCe\\ngekukSQJkFO3uqbOh9UTNcw/YeY/JuXUcupmOMbq1t7ZO29w4P8XvPhqq24lVS2c\\nTi0wN3zLLzG7Bo6Yc1Ug+gJpI2iibnA7nPlwZySZSTBnk3d1g1rZO7p6V34QSEqL\\n6ob8LyAnSXK2RF5jyh2mKi0XsGDnSLIKSNFkiqsmi4Ow14v9adTqptJscRDxUm0C\\n1ZDs4YWTgqxHsfCKwHvEj0zyW5qjchxUrn9QHIxko4PiXlqBwUs+azZmOWKHqcLA\\nSq0WBxEbqkDOaBYKHaazFFdPJCipW09ff21nV25zS0JtXWR51aHS8v1FpfsKSvwL\\niiUP3/wi77wil5x8i/ScnxJSXj8RZRh06DoXP30bbprCQ34hms8wpepWUdv2jw3q\\n6BG8kscbWXodGXq6rLr19w/A23TxO/5k3bIfmbV8Q04XcsazwGLiYstD0cXkJFwv\\nmEp5LK70Cn60/jWD8iG4oLaxg5vICYWZa+rETPBk8Y/InUSWgZhn+ugHgXmlTYIL\\nPto1FHUjyf9oE0PM4L4Nbrmi64eVUGN1A742P0OmOs5/01ewNp4AOXUrqmzhllXj\\ndzhg+zdtA89xfYULSYep2qjOcyMsIOBE/uBeWGbOYaPpMN3rlcxNWZ1ntuz9wJKq\\n1tb2nua27qa2blC3dtFeGpqr20s/HCILRD/2cXBfv/r/B9ILGm59mu+SBnWzRnVD\\nLn8k3zG/2ESKX6KsVClPUyBotWwsvZgVRAp5zStPcVXOUXw9DZYIjGp06qYSKYJ4\\nSV72DLlYsoXZ6Br5LM5a2+WOR1Q3tpzi8ypRB7rk6rjs8cWuw5LNRRtHcoCduDoq\\n+cnLo1a3sVqSl7XSC6huyXUNW+OS1xyMmO0fdrtX0HWu/tOcfK5w8Jrk4DnB3nO8\\nnYfkMc7OU9/Og5+swLkaN/QNPvDTHfSsXA0DDtRqPMOUqhvwjdkZbrSWERd7e2Cj\\nRz1jIYKom3q2Jq9BcGVkUnlNY8eBM0UPbHTnTvJ9XlttzpLtnhIyq9/9PcL9YHZG\\nQUNlfXthRfOXpqe5PRjmm9/2lFNxVUtrR8+XpmdMvZPjM6vLa9sq6tod9meQZcMm\\nLbI4kVAmWQvw3Q93nuAKvMCCL4bnTrfEpJxasBBIIb2wYe2XoSRUA+pG1DAtv/4W\\nMsVygcWy9wIORhWDP4FtlFa3nkqu2G4XG5mozksrdQN/umGVPRmi98Sn+4/Hl9U0\\ndDS0dEHVQFJ/c4yjK8/JqVtTa9fcTd5kAV6jd/YlZtdU13f4Hs29a52LeoGPheY7\\n3RLqmjqhIiBJWm0//4XZmeziRqg7HCeTy7dax9zCz1qFb2e+4kWWz9BW3Uy8k7m9\\nKBZZjltsee8GN4PNPnP5Y97rPo/8O+CtX44djS0RzzBV6jDl1e3zvae4tuU6YR18\\nj+XCjwqueeKz/fRHtfbLMDBFYoeobgiCIMgFRVrdwNj0rPjpCLb8QYRM08OTm4IK\\nH2y5CacgcFMcvJYEHwkvE756BVB1W/LvgFZG3dIL6q9bZUfCG78zC2ipGHWbOzjW\\nzSUsc4J6xqj51OU2dz/nOnGBxbQlVgve9tNfyI1UG7fIYsZLHmAGEfFl47nwj8nV\\nK2zvWe920xrHwYkIplv4l25dcye3C9OMPZOXWt21zvX2Z53Hk5TnmDz9RYg4ikMB\\niflgZyS3Y5WBGekcnLrM+p7nXOGY9igfVTKyAMNY93kIXSTCOTRj2jIbspbHhAXm\\n4CuzX/W+/RnnCXDlP3c7BKaTy7RSNxW3fHHcJLIh2DyzyYss/7HBbdarXnCvvqH5\\neANTGoBUGOv2i1McLyichl630u6OdS76Bqa3cqPHfPk1PrgpC4abfUqqOXHRal23\\niUutrlxmA8cVy2z0F1ioNxmba3rb006hgxtRKKvbN4y6qQZnmD7yXgDZn4CbV0GP\\nOSbE0q5cauUyuGTxiOp27eA0hROJZVfzZ7jhfUus4Ed15RKrSYZmi9/dp8+Pjxxn\\nZHHDanvfY9xCwVTdvkZ1QxAEQS4A0up2n0+wWtpsyOHGHIM+J3Po27qPt/eY4uR9\\nu2fgY2FHv41NOlBc1tTdLZkRS0FFy/QVtnoPGy9+x5+NuvX09h88U+gcmul+IIuO\\nNCL87hzPvQ5nm8zZ7ENmmHZ1922zPQuqREa8gSFdu8LWxCclMbvmvufduIvnmExb\\nbFlU2RKXUX092SNrLv92h7/zzKY/ZrfVOpqsi9vU2s0t22HAp0MMwMB0ylLrl344\\nTCZmKtDd0xdyqmDtF6HcrAWy6+XsoVxARMBITLyHegz7BwYCT+QveNNvHL8KCXfl\\nLGNOOOBiIwvfoznksvC40on8hle3rB62+Ajw8vfcdlVw1xbGGKDp3A9mcXt3LlDP\\nslR7jIHp9FV2p5LVy7UEHs+fxG9C9bfVDoLpF60dPR/vipzEzaM0I/NS4Rr3Q1kn\\nk8pvAdklM1VX2MKzA3X7+9POkPhVS63OiDSLAM+IG9g324RMyyBOya/lwe2L+rc1\\nDm/9coxdrw7U7a5nXMDP9A3NxIuDfGMepTeTq/LrPx9Vj3Wrbn31xyP6RhYPvuT5\\n2EfBKz8MIsdDL3mqNzOYZ/bQix61/Narn+yO5JpipvGm7w+zycacq5q2lNvP9Nrl\\nttRB4efH1XfwxzDRyOKT3ScLypshcbKHrN5Dux35WbEgguRBCOapfG8dQ86/tvUI\\ne97SP40XUDMwRclGQxAEQRAWaXW72ytoqqP3DP/Q1WHHXos482l0ws8JaXvTMm0y\\ncpyy8zzyCj1lDu/8ouCi0hMV1ekNTbWdnb2DUaXG7m7f/CLlJXnbOnr8juW5Hcg6\\nElPSI781Akt6Qb3HwWy3A9kHo4rpuLT+/oHY9Oo/3BK/Mo/a5ZGYkqNWnPyyJjPf\\n1G32sbZB58BI4PqMwgbIbrtDLFy51fasU2hGbkkjO7yttLp1X0TeDpeEr82jvrWK\\nNvdNBeHr7u0Tl0QSuBIEy3F/xvc2MR/sivxwV+S3ltFW+9JOJldIDj5rbus+Gluy\\nyyNpi2U05PibczwULymnpmtwumtlfYfX4Rz3g9kBx/MEQ+/BltzCsqApyApwLA0t\\nXQeiina6J35tEf2NRdQfrgmeh7PP5df1Dk7jgGrSZDu6hdFE8K3IhLLfnOKgSGa+\\nKXRD1bT8ur1eydvs41xCM8Hb+voGgiIL3A9k+4TnklX3xECbRyaWQyE9D+V4Hh46\\nfI/mRsSVQjEEW7B3dvUG82lCgWtEC84lZde6hsHTzyKxsbKa1vlv+IAeGbzuW1zV\\nArUDjydHa3vPr45xZLjkDYPWG59ZDSlDowlEs66p0zc8F1rD72huLVOR7KIGM5+U\\nr82ifnWKA3Ml/6tQ3dDhFJKx3T52j2cS+KuK76fmH0RW4vA1k5Nz6+Ak1F0Qkswp\\naQS99jiY439MuOYwgiAIgoiRVjfTc1mnKqtrOrp6JHdZ15ju/v6sxmbL9OxFQYe4\\nGaZtStMUEOR8AIXignmzTd7+VWKi7uGY4on8QsQ3r3HILmkUX4AgCIIgOoHSDNPe\\ngYH6zq685pakuobTVTXHyivDSsqDCmVnmMLhnV/knJNvnp69LT7l7cjopSFHrnPx\\n5aYsWLnOGGmGKYKcDw7B6WSP+TvXue47nldS1Vrb2FnT0FFU0XwsrnTFR0FknOKa\\nT/azi/YhCIIgiG4hrW7nGpq2xqWsPXR89r6wO7wCb3T1n+bsM9nBa4KD5zh7bhqp\\n3KGeqWDLD48DY4O/th7crAVuD9MRZpgiyPlQWNH84EYPMkBw8hKre553m7XZ5+FN\\n3nevd+PWfOFHpD3worugHxNBEARBdAtpdTMIOMBthGUzfHqpvacWByNwE+09FwYe\\nCiuWXk0DQcaK1Ny6V7Yevnu969TlNvpkZzN+AsS1q+xmvOz1wX+PZxY1jJwKgiCX\\nF7GxsdsGoSe3McAF7PXh4eF//vnnWOU74mUBAQFWVlbnn52GsBWnJ6G+UGsNbzyf\\nxoFclNvElUdwUpPijQgkInjW0PJwBv7SM+SRjZgX/UXRohYUFND2YRNUiX4GJFOA\\nfeiQjuS9csjPMOUCZoNzS/keT/VhrXxwq4Ho23pMdfS+yyvo0dCjX0Qn7C8qq8N4\\nG/KX0NPbX1jefDq5IuxMUcCJ/KDIgsMxJXEZ1ZX17f3nN3ATQRAdhb6w4b1I3rXK\\nUnUZqxvUi8oBcQh6XsFXiJfQC86nwCOqmyRjpW5s9QGoheAM/DyseJSTor8oWjD4\\nQM6QtqKCSCWP/pP+tGjWpE3gRs1rKrP9vHfQVY7eM/xCVoUdfS3izOdR8dvjU3en\\nZFimZ9tl5Tlk50sejtn5brkFfgXFR8oqEmrry9s7+pQ3nEcQBEGQCwm8CMVRHDip\\n8HqWUzcaViH3kpc0jZfQNy4bkqHvbHovK0D0MloYQfSFDUFRISCFHzF3cRXILUQR\\naJokNcnoI0UcCaOmQrSDLQMRO5ImuYz9llxPi03TJ9e7DkJzIa1NhUaybRWqLCgz\\nlI1eA+0AnyFxVt1ImoJWEsD+PCQtlpaWPGI5W6U1Zaus/MukSKvbn8npkRXVlR0d\\nXX04oBtBEATRVeClSCWJfePKdW+pZNSNvuNpWIXKE0mQ3kI1heoFKQO9VzVoGzTQ\\nQt7W9B1Ps6BBGvKB+hxcKZk7G/KBNMXOSgxJHNdRDvbIfcu6IC0DaVtSBtrm9FtS\\nR1oRUnHWw1ihId+ynZi0JKQimlRZUAv4SwtMsqA/AGr59HlJwuqa5E9FIMFy6kbr\\nIlA3TSK+F3b7eQRBEAS5iBAtIG9r+mqnHwRdgYQRX5+supGXNO0bZTtJ6Wdx1Ip9\\n/dPPrEbQW0gKcJ6KBZEPydzZmI1cd604TqYarboJkhVYmorpE2RLKNk+bNyRVlNQ\\nPMm21aTKbCJUy+AuuJ5tc/hMY3sKPwBldRP3t4rVjT4C8QWkV1cuawqqG4IgCHLZ\\nMmJIQ+xVkpexcx0U1I19DdMYm2owSsQG5NjXP/nMjruipSKGAZ8hNdJPKg6w0dzZ\\naCIrB5KQuwRjtiRR+JZ2fYoDbKqR1I0G7STVjTVLIlWS6qZ5lan/0e5mlSiYKnjE\\nkukoqBuUXPzLkYu6sefZjnJUNwRBEOR/mhG7tzRUN1ZftI26UagxaB51C+ChHYvk\\nnyoZdRtxDJ/gW3aUvYK6SY51E7jUpRB1U4YmSAyJmjEpoWDUmuREV0FNBXfJzW+Q\\nUzfx0EMVM5NGGVQ3BEEQ5LKFfUFSk6BvccF8QILcACZ27LmcurG50CFctCeOqht7\\nr9xYN3oL7d4lViSeyUhzZ09Kxm/YUJbmExvFM0zZgWu01pqrmyD0KKluNB3JsW60\\nbUesMkUwTo5+pj8J8SohckmJo5XKqicWXBUToqP+J/lrlATVDUEQBLmcoX2dbFCE\\n9ouJlYUdDUb74GjHHHnRwl85daNdbwK9EHTD0Vxc5WeYEsQCpJJRN9Xwjl3J1mDr\\nRUM+JF8oklzUh60Cq0dkEigpCUlwRHUjqsqmI6luqkHHItdrMsOUNSSFiZ+ss5JH\\nyc4aYRtKbrKCYF03cWcre6Mg6sb28NKTgv70EUF1QxAEQRBEjSYddhedEdeW04la\\njBpUNwRBEARBOMJ5LnYppBEEtxSujI2N1XBbAh0F1Q1BEARBEERnQHVDEARBEATR\\nGVDdEARBEARBdAZUNwRBEETHYFeCZUc+KW+FNIaQlV0FhVFYH06ywGRLK0HK4jVm\\nycxHyamjI85MZGeA0lKJ54rSYWTs2H96pebjxsTVVNgMVEPoKsQqpnEEcxTYObaS\\nDUjXw2MRTyVmF14RlF9y8i+L+FkIlnFmLxZMs6D3atjUqG4IgiCIjiFYRotd1fav\\nUTcqE6yf0UXXVINaILnMGLv0Lt1SUxK2mtuY1cskVz4Tp0MX3RAs3C9uInbNM3bJ\\nX5LmNs0WG5NL/Dwhmx+ohvsZXS5ExRiScnnIvhRyubAuxS4jIre9rPh28bOQmwZL\\n5JJdk5ldZATXdUMQBEEuQwSKQDedJOomDswIgk/k/UqvFEekxK9VFrqdqEpGCMhq\\nq5IrxArW+2WTEiDYq0B8maTYsUh6GCs9khkJ1olVaSNkcley0Ti6ASspD12mWHIx\\nNkF0Uy5IKbljgWBZXTYpAWx5JC+TFDu5itNn4Sq1jT3ZdIFdyo5Fk+1iVahuCIIg\\niM4hUAT6T9LxpBoewBAHn8hLne55QL5l4yWCtfUFsLpG0pHs7ZJUN8G7me0NFOfC\\n7otFcxEH/OSiblQ4BNseCHoDWZtkN6eiLSyOHskFuiTVjU2fhkWpISkHw1hdIw+F\\n5CvumxYXhm1AgmRkTlxZYt5sBSX3q5CsI/ssaDeo+BZJdZPcGksSVDcEQRBExxCM\\nqRKbgWr4npX0xcnuhikQFzbAJrkXFoV9v9IQnfilLlY3Gh1kkewzZbeup/WlQsCW\\nU66vkJZhm2hMG+143cbscEBLqIm60fYXhIgEz0Wsv+JkleN5bB803WtLNXxHL5WU\\nugkakCCniewDpRudqYYLFrVVObUSPwtaSHb/BtoOkk2q4X6sqG4IgiCIjiHXMSdW\\nN4FRKaibnA6KEagbTVwQ5hGrm2QPGmsn7L2SYUW2wGxliSVQeyAX0NihXKyInLzQ\\nUTfV8FAfvUCTIJNA3aiuCTISq5tkT7e4s1gl6qhlK8u2BtthSvfOoo0gfhbiirAn\\n5TpM5c4LQHVDEARBdAzN1U01qqibMmycjH0lj6hukpYmGXUTSAbrE7TACipDUNBK\\n9oILPdZNLilSR2VTYeNkgh5kZXWTtDS5gYmCLUfFIstmLTn5YMRnoaG6KYd7Kahu\\nCIIgiI6hlboJxrqBJ0mqG3tSOfjBGgB9kYsHnAnUTbILT26smyApNkBFY2maRHoE\\nUTfBIhrs3NgLN8OUNiZJk1xAJ15AdgoD89mxbvRJiYOIYnWTjOdJ9iyLfZomTltY\\nHHUTJCJ+FmwDiovHJsg2GiSuiSWjuiEIgiA6hlbqppKZYSoepC/ocFRpMMNUJT/w\\nS6BukgPdJKeOSkoe7aGTnDkr974XDztjFzMT5LhtLNZ1k6sOaVXS2oK5n+TMiDNM\\nVcyabYIiCdxIrgEldVwseWyB6Uk650DO6cXPgu1UFWQh+H+DERMXgOqGIAiCINqh\\nSa+Whumc/6K1lzd0XbfzT0fD2OGlD6obgiAIgmiHwiJhmiM5awERoLB+iuZoPpBR\\nJ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE\\n0RlQ3RAEQRAEQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAE\\nQXQGVDcEQRAEQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAE\\nQRCdAdUNQRAEQRBEZ0B1QxAEQRAE0RlQ3RAEQRAEQXQGVDcEQRAEQRCd4f8BrPZE\\nAGuIn50AAAAASUVORK5CYII=\">\\n<p style=\"top:191.9pt;left:45.0pt;line-height:13.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:13.0pt;color:#00637e\">GENOME SCREENING RESULTS SUMMARY </span></b></p>\\n<p style=\"top:207.3pt;left:45.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">NEGATIVE- NO MONOGENIC DISEASE FINDINGS</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">. Sequencing of this individual&#x2019;s genome identified</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> 3 carrier status </span></b></p>\\n<p style=\"top:220.7pt;left:45.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">variants. Pharmacogenomic associations are also included in this report. Result details are listed below. </span></b></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">2 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:79.0pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:90.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease, </span></b></p>\\n<p style=\"top:101.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Inheritance </span></b></p>\\n<p style=\"top:90.5pt;left:212.4pt;line-height:9.0pt\"><b><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene  </span></i></b></p>\\n<p style=\"top:101.5pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Transcript </span></b></p>\\n<p style=\"top:90.5pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Variant </span></b></p>\\n<p style=\"top:90.5pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Allele State </span></b></p>\\n<p style=\"top:101.5pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:90.5pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Classification  </span></b></p>\\n<p style=\"top:114.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Primary ciliary dyskinesia, </span></p>\\n<p style=\"top:125.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive </span></p>\\n<p style=\"top:114.3pt;left:212.4pt;line-height:9.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:125.3pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_017950.4 </span></p>\\n<p style=\"top:114.3pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.248delC </span></p>\\n<p style=\"top:125.3pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">p.Ala83ValfsX84 </span></p>\\n<p style=\"top:119.8pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous </span></p>\\n<p style=\"top:119.8pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic </span></p>\\n<p style=\"top:136.7pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genomic Coordinate </span></b></p>\\n<p style=\"top:136.7pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Location </span></b></p>\\n<p style=\"top:136.7pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Penetrance </span></b></p>\\n<p style=\"top:136.7pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Phenotype </span></b></p>\\n<p style=\"top:136.7pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene Coverage  </span></b></p>\\n<p style=\"top:149.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">g.80039966delC (chr17, GRCh38) </span></p>\\n<p style=\"top:149.8pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Exon 3 </span></p>\\n<p style=\"top:149.8pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">High </span></p>\\n<p style=\"top:149.8pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">None reported </span></p>\\n<p style=\"top:149.8pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">100% at 15X </span></p>\\n<p style=\"top:161.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">VARIANT INTERPRETATION: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">The p.Ala83ValfsX84 variant in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> has been previously reported in 19 homozygous and 7 compound </span></p>\\n<p style=\"top:172.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">heterozygous individuals with primary ciliary dyskinesia (PCD) and segregated with disease in 1 homozygous affected relative (Becker-Heck 2011 </span></p>\\n<p style=\"top:183.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PMID: 21131974, Nakhleh 2012 PMID: 22499950, Antony 2013 PMID: 23255504, Zariwala 2013 PMID: 23891469). This variant has been identified </span></p>\\n<p style=\"top:194.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">in 0.074% (860/1167354) of non-Finnish European chromosomes by gnomAD (http://gnomad.broadinstitute.org). However, this frequency is low </span></p>\\n<p style=\"top:205.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">enough to be consistent with a recessive allele frequency. This variant has also been reported in ClinVar (Variation ID 31069). This variant is </span></p>\\n<p style=\"top:216.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">predicted to cause a frameshift, which alters the protein&#x2019;s amino acid sequence beginning at position 83 and leads to a prema</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ture termination </span></p>\\n<p style=\"top:227.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">codon 84 amino acids downstream. This alteration is then predicted to lead to a truncated or absent protein. Functional studies indicate that loss </span></p>\\n<p style=\"top:238.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">of CCDC40 function results in abnormal cilia structure and motility (Becker-Heck 2011 PMID: 21131974). In summary, this variant meets criteria to </span></p>\\n<p style=\"top:249.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">be classified as pathogenic for autosomal recessive primary ciliary dyskinesia. ACMG/AMP Criteria applied: PVS1, PM3_VeryStrong, </span></p>\\n<p style=\"top:260.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PM2_Supporting, PP1. </span></p>\\n<p style=\"top:271.8pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DISEASE INFORMATION:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> Primary ciliary dyskinesia is a rare genetic condition that is genetically heterogeneous. It is associated with recurrent </span></p>\\n<p style=\"top:282.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities in the motility of the cilia </span></p>\\n<p style=\"top:293.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">and flagella that are found in the linings of organs and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and </span></p>\\n<p style=\"top:304.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror-image reversal of all visceral organs, is </span></p>\\n<p style=\"top:315.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">found in 40-50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females </span></p>\\n<p style=\"top:326.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Other symptoms can include recurrent ear </span></p>\\n<p style=\"top:337.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">infections and hydrocephalus in the brain. Pathogenic variants in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> contribute to 3-4% of primary ciliary dyskinesia (Medline Plus: </span></p>\\n<p style=\"top:348.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">https://medlineplus.gov/genetics/condition/primary-ciliary-dyskinesia, GeneReviews: https://www.ncbi.nlm.nih.gov/books/NBK1122). </span></p>\\n<p style=\"top:360.3pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FAMILIAL AND REPRODUCTIVE RISK </span></b></p>\\n<p style=\"top:371.9pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease Prevalence (Estimated) </span></b></p>\\n<p style=\"top:371.9pt;left:288.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Frequency (Estimated) </span></b></p>\\n<p style=\"top:371.9pt;left:419.1pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Reproductive Risk (Estimated) </span></b></p>\\n<p style=\"top:383.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/16000 (https://medlineplus.gov/genetics/condition/primary-</span></p>\\n<p style=\"top:394.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ciliary-dyskinesia) </span></p>\\n<p style=\"top:383.7pt;left:288.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/317 to 1/366 (for </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">) </span></p>\\n<p style=\"top:383.7pt;left:419.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/1268 to 1/1464 (for </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CCDC40</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">) </span></p>\\n<p style=\"top:406.1pt;left:36.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008000\"> </span></b></p>\\n<p style=\"top:417.2pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:428.6pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease, </span></b></p>\\n<p style=\"top:439.6pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Inheritance </span></b></p>\\n<p style=\"top:428.6pt;left:212.4pt;line-height:9.0pt\"><b><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene  </span></i></b></p>\\n<p style=\"top:439.6pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Transcript </span></b></p>\\n<p style=\"top:428.6pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Variant </span></b></p>\\n<p style=\"top:428.6pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Allele State </span></b></p>\\n<p style=\"top:439.6pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:428.6pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Classification  </span></b></p>\\n<p style=\"top:452.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Primary ciliary dyskinesia, </span></p>\\n<p style=\"top:463.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive </span></p>\\n<p style=\"top:452.3pt;left:212.4pt;line-height:9.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:463.4pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_001277115.2 </span></p>\\n<p style=\"top:452.3pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.12363C&gt;G </span></p>\\n<p style=\"top:463.4pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">p.Tyr4121X </span></p>\\n<p style=\"top:457.9pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous </span></p>\\n<p style=\"top:457.9pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic </span></p>\\n<p style=\"top:474.9pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genomic Coordinate </span></b></p>\\n<p style=\"top:474.9pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Location </span></b></p>\\n<p style=\"top:474.9pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Penetrance </span></b></p>\\n<p style=\"top:474.9pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Phenotype </span></b></p>\\n<p style=\"top:474.9pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene Coverage  </span></b></p>\\n<p style=\"top:487.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">g.21880869C&gt;G (chr7, GRCh38) </span></p>\\n<p style=\"top:487.9pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Exon 75 </span></p>\\n<p style=\"top:487.9pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">High </span></p>\\n<p style=\"top:487.9pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">None reported </span></p>\\n<p style=\"top:487.9pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">100% at 15X </span></p>\\n<p style=\"top:499.6pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">VARIANT INTERPRETATION: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">The p.Tyr4121X variant in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> has been reported in the compound heterozygous state in 1 individual with </span></p>\\n<p style=\"top:510.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">primary ciliary dyskinesia and segregated with disease in 5 affected siblings (Schwabe 2008 PMID: 18022865). It has been identified in 0.045% </span></p>\\n<p style=\"top:521.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(29/63996) of European (Finnish) chromosomes by gnomAD (http://gnomad.broadinstitute.org, v4.0.0). However, this frequency is low enough to </span></p>\\n<p style=\"top:532.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">be consistent with the carrier frequency. This variant has also been reported in ClinVar (Variation ID 6475). This nonsense variant leads to a </span></p>\\n<p style=\"top:543.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">premature termination codon at position 4121, which is predicted to lead to a truncated or absent protein. Biallelic loss-of-function of the </span></p>\\n<p style=\"top:554.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11 gene is an established disease mechanism in autosomal recessive primary ciliary dyskinesia 7. In summary, this variant meets criteria to </span></p>\\n<p style=\"top:565.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">be classified as pathogenic for autosomal recessive primary ciliary dyskinesia 7. ACMG/AMP Criteria applied: PVS1, PP1_Strong, PM3, </span></p>\\n<p style=\"top:576.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PM2_Supporting. </span></p>\\n<p style=\"top:588.0pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DISEASE INFORMATION:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> Primary ciliary dyskinesia is a rare genetic condition that is genetically heterogeneous. It is associated with recurrent </span></p>\\n<p style=\"top:599.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">respiratory tract infections, abnormal positioning of the internal organs, and infertility. This is caused by abnormalities in the motility of the cilia </span></p>\\n<p style=\"top:610.1pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">and flagella that are found in the linings of organs and tissues. Respiratory tract infections, reduced clearance of mucus, nasal congestion and </span></p>\\n<p style=\"top:621.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">chronic cough begin in early childhood and could result in bronchiectasis. Situs inversus totalis, a mirror-image reversal of all visceral organs, is </span></p>\\n<p style=\"top:632.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">found in 40-50% of individuals. Males with primary ciliary dyskinesia are frequently infertile because of abnormal sperm motility, while females </span></p>\\n<p style=\"top:642.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">with this condition are occasionally infertile likely due to abnormal cilia in the fallopian tubes. Other symptoms can include recurrent ear </span></p>\\n<p style=\"top:654.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">infections and hydrocephalus in the brain. Pathogenic variants in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> contribute to 6-9% of primary ciliary dyskinesia (Medline Plus: </span></p>\\n<p style=\"top:664.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">https://medlineplus.gov/genetics/condition/primary-ciliary-dyskinesia; GeneReviews: https://www.ncbi.nlm.nih.gov/books/NBK1122). </span></p>\\n<p style=\"top:676.4pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FAMILIAL AND REPRODUCTIVE RISK </span></b></p>\\n<p style=\"top:688.2pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease Prevalence (Estimated) </span></b></p>\\n<p style=\"top:688.2pt;left:288.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Frequency (Estimated) </span></b></p>\\n<p style=\"top:688.2pt;left:419.1pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Reproductive Risk (Estimated) </span></b></p>\\n<p style=\"top:699.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/16000 (https://medlineplus.gov/genetics/condition/primary-</span></p>\\n<p style=\"top:710.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ciliary-dyskinesia) </span></p>\\n<p style=\"top:699.8pt;left:288.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/211 to 1/259 (for </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">) </span></p>\\n<p style=\"top:699.8pt;left:419.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">1/844 to 1/1036 (for </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DNAH11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">) </span></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">3 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:79.0pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:90.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease, </span></b></p>\\n<p style=\"top:101.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Inheritance </span></b></p>\\n<p style=\"top:90.5pt;left:212.4pt;line-height:9.0pt\"><b><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene  </span></i></b></p>\\n<p style=\"top:101.5pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Transcript </span></b></p>\\n<p style=\"top:90.5pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Variant </span></b></p>\\n<p style=\"top:90.5pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Allele State </span></b></p>\\n<p style=\"top:101.5pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:90.5pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Classification  </span></b></p>\\n<p style=\"top:114.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Short/branched chain acyl-CoA </span></p>\\n<p style=\"top:125.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">dehydrogenase deficiency, </span></p>\\n<p style=\"top:136.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Autosomal Recessive </span></p>\\n<p style=\"top:119.8pt;left:212.4pt;line-height:9.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ACADSB</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:130.7pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NM_001609.4 </span></p>\\n<p style=\"top:125.3pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">c.303+1G&gt;A </span></p>\\n<p style=\"top:125.3pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Heterozygous </span></p>\\n<p style=\"top:125.3pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pathogenic </span></p>\\n<p style=\"top:147.7pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genomic Coordinate </span></b></p>\\n<p style=\"top:147.7pt;left:212.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Location </span></b></p>\\n<p style=\"top:147.7pt;left:320.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Penetrance </span></b></p>\\n<p style=\"top:147.7pt;left:410.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Phenotype </span></b></p>\\n<p style=\"top:147.7pt;left:491.5pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Gene Coverage  </span></b></p>\\n<p style=\"top:160.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">g.123037848G&gt;A (chr10, GRCh38) </span></p>\\n<p style=\"top:160.8pt;left:212.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Intron 3 </span></p>\\n<p style=\"top:160.8pt;left:320.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Reduced </span></p>\\n<p style=\"top:160.8pt;left:410.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">None reported </span></p>\\n<p style=\"top:160.8pt;left:491.5pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">100% at 15X </span></p>\\n<p style=\"top:172.5pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">VARIANT INTERPRETATION: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">The c.303+1G&gt;A variant in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ACADSB</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> has been reported in the homozygous state in 2 individual with short/branched </span></p>\\n<p style=\"top:183.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">chain acyl-CoA dehydrogenase (SBCAD) deficiency; both showed the biochemical phenotype but only one showed clinical symptoms (Alfardan </span></p>\\n<p style=\"top:194.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">2010 PMID: 20547083). It has also been reported in the homozygous state in 2 siblings without clinical symptoms (Spedicati 2021 PMID: </span></p>\\n<p style=\"top:205.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">33727708). It has been identified in 0.056% (628/1127804) of European (non-Finnish) chromosomes, including 1 homozygote, by gnomAD </span></p>\\n<p style=\"top:216.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(http://gnomad.broadinstitute.org, v4.0.0). This variant occurs within the canonical splice site (+/- 1,2) and is predicted to cause altered splicing </span></p>\\n<p style=\"top:227.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">leading to an abnormal or absent protein. Computational prediction tools and conservation analyses are consistent with pathogenicity. Biallelic </span></p>\\n<p style=\"top:238.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">loss-of-function of the ACADSB gene is an established disease mechanism in autosomal recessive SBCAD deficiency. SBCAD deficiency results in a </span></p>\\n<p style=\"top:249.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">biochemical phenotype that be detected from birth; however, the clinical significance of this is unclear. 90% of individuals with SBCAD deficiency </span></p>\\n<p style=\"top:260.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">have no clinical symptoms, and the remaining 10% may display developmental delay and/or neurological disorders. These 10% of individuals may </span></p>\\n<p style=\"top:271.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">represent extreme end of the clinical spectrum or coincidental findings (Porta 2019 PMID: 30730842, Alfardan 2010 PMID: 20547083). In </span></p>\\n<p style=\"top:282.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">summary, this variant meets criteria to be classified as pathogenic for autosomal recessive SBCAD deficiency. ACMG/AMP Criteria applied: PVS1, </span></p>\\n<p style=\"top:293.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PM3_Supporting. </span></p>\\n<p style=\"top:304.8pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DISEASE INFORMATION:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> Short/branched chain acyl-CoA dehydrogenase deficiency (SBCAD deficiency) is a rare inborn error of metabolism that is </span></p>\\n<p style=\"top:315.9pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">more common among Hmong population in Southeast Asia. Affected individuals can be detected during newborn screening (NBS) but most have </span></p>\\n<p style=\"top:326.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">no clinical symptoms. 10% of SBCAD deficiency patients have developmental delay and neurological disorders; however, it is unclear if the SBCAD </span></p>\\n<p style=\"top:337.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">deficiency is causative (Porta 2019 PMID: 30730842, Medline Plus: https://medlineplus.gov/genetics/condition/short-branched-chain-acyl-coa-</span></p>\\n<p style=\"top:348.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">dehydrogenase-deficiency). </span></p>\\n<p style=\"top:360.3pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FAMILIAL AND REPRODUCTIVE RISK </span></b></p>\\n<p style=\"top:371.9pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Disease Prevalence (Estimated) </span></b></p>\\n<p style=\"top:371.9pt;left:288.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Carrier Frequency (Estimated) </span></b></p>\\n<p style=\"top:371.9pt;left:419.1pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Reproductive Risk (Estimated) </span></b></p>\\n<p style=\"top:383.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Unknown </span></p>\\n<p style=\"top:383.7pt;left:288.6pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Unknown </span></p>\\n<p style=\"top:383.7pt;left:419.1pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Unknown </span></p>\\n<p style=\"top:395.3pt;left:36.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008000\"> </span></b></p>\\n<p style=\"top:406.5pt;left:41.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">D. PHARMACOGENOMIC ASSOCIATIONS</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008000\"> </span></b></p>\\n<p style=\"top:419.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Detailed dosing instructions are not provided in the brief interpretation notes below. Extrinsic factors (e.g. diet, smoking status, co-administered </span></p>\\n<p style=\"top:430.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">medications, drug-drug interactions) and intrinsic factors (e.g. sex, age, weight, renal or hepatic function) may affect drug response. A *1/*1 or </span></p>\\n<p style=\"top:441.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">wild type result is assigned if no variants that are included in the test are detected and does not exclude the possibility that an individual has a </span></p>\\n<p style=\"top:452.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">different phenotype that may alter drug response. Patients should not use the test results to stop or change any medication unless directed by a </span></p>\\n<p style=\"top:463.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">qualified clinician. Genetic information to guide prescribing decisions may be found in a drug&apos;s FDA-approved label. These labels are found at </span></p>\\n<p style=\"top:474.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> and a table of current PGx biomarkers at </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.fda.gov/medical-</span></p>\\n<p style=\"top:485.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">devices/precision-medicine/table-pharmacogenetic-associations</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> and </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.fda.gov/drugs/science-and-research-drugs/table-</span></p>\\n<p style=\"top:496.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">pharmacogenomic-biomarkers-drug-labeling</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">. The Clinical Pharmacogenetics Implementation Consortium (CPIC&#xae;) website may be consulted for </span></p>\\n<p style=\"top:507.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">their most current recommendations, at </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://cpicpgx.org/guidelines/</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">; however, these have NOT received FDA-approval. Always consult a </span></p>\\n<p style=\"top:518.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">clinician or clinical pharmacologist before changing drug dosage or for additional information. The Pharmacogenomics Clinic at BWH is a </span></p>\\n<p style=\"top:529.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">resource for patients looking for a more detailed interpretation of the clinical relevance of the results </span></p>\\n<p style=\"top:540.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.brighamandwomens.org/medicine/genetics/genetics-and-genomic-medicine/pharmacogenomics</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">). </span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008000\"> </span></b></p>\\n<p style=\"top:551.6pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:563.3pt;left:36.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genotype</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:563.3pt;left:216.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Phenotype</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:563.3pt;left:396.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FDA Drugs with PGx Labelling</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:575.1pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">TPMT *1/*1 </span></p>\\n<p style=\"top:586.1pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NUDT15 c.415C/c.415C</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:575.1pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">TPMT Normal metabolizer </span></p>\\n<p style=\"top:586.1pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NUDT15 Normal metabolizer</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:580.6pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Azathioprine, Mercaptopurine, Thioguanine</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:597.8pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">IFNL3 c.-3180G/c.-3180G&gt;A</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:597.8pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">IFNL3 Unfavorable response</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:597.8pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Peginterferon Alfa-2b</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:620.6pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C9 *1/*1</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:620.6pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C9 Normal Metabolizer  - Activity score 2</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:609.6pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Celecoxib, Dronabinol, Erdafitinib, Flibanserin, </span></p>\\n<p style=\"top:620.6pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Flurbiprofen, Lesinurad, Meloxicam, Phenytoin, </span></p>\\n<p style=\"top:631.5pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Piroxicam, Siponimod</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:665.3pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C19 *1/*17</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:665.3pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C19 Rapid metabolizer</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:643.3pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Brivaracetam, Carisoprodol, Citalopram, </span></p>\\n<p style=\"top:654.3pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Clobazam, Clopidogrel, Dexlansoprazole, </span></p>\\n<p style=\"top:665.3pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Diazepam, Doxepin, Escitalopram, </span></p>\\n<p style=\"top:676.3pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Esomeprazole, Flibanserin, Omeprazole, </span></p>\\n<p style=\"top:687.2pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pantoprazole, Rabeprazole, Voriconazole</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">4 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:79.8pt;left:36.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genotype</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:79.8pt;left:216.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Phenotype</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:79.8pt;left:396.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">FDA Drugs with PGx Labelling</span></b><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:91.6pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C9 *1/*1 </span></p>\\n<p style=\"top:102.5pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">VKORC1 G/A </span></p>\\n<p style=\"top:113.5pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP4F2 *1/*3</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:102.5pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Use genotypes to guide dosing</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:102.5pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Warfarin</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:125.2pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">SLCO1B1 *1A/*1A</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:125.2pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">SLCO1B1 Normal function</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:125.2pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Elagolix, Rosuvastatin, Simvastatin</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:136.9pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DPYD wild type at all tested positions</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:136.9pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DPYD Normal metabolizer - Activity Score 2</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:136.9pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Capecitabine, Fluorouracil</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:148.7pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP3A5 *3/*3</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:148.7pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP3A5 Poor metabolizer</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:148.7pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Tacrolimus</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:171.4pt;left:36.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">UGT1A1 *1/*1</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:171.4pt;left:216.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">UGT1A1 Normal Metabolizer</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:160.5pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Belinostat, Dolutegravir, Irinotecan, Nilotinib, </span></p>\\n<p style=\"top:171.4pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Pazopanib, Raltegravir, Sacituzumab govitecan-</span></p>\\n<p style=\"top:182.4pt;left:396.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">hziy</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:194.3pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\"> </span></b></p>\\n<p style=\"top:207.8pt;left:41.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">RECOMMENDATIONS</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008080\"> </span></b></p>\\n<p style=\"top:221.1pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">These results should be interpreted in the context of this individual&apos;s personal medical history and family history.</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008080\"> </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genetic counseling is </span></p>\\n<p style=\"top:232.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">recommended for this individual and their relatives. Familial variant testing is available if desired. </span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008080\"> </span></b></p>\\n<p style=\"top:243.0pt;left:36.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#008080\"> </span></b></p>\\n<p style=\"top:254.3pt;left:41.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\">COVERAGE SUMMARY </span></b></p>\\n<p style=\"top:267.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Sequenc</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">ing of this individual&#x2019;s genome </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">covered 97.5% of all positions at 15X coverage or higher. Please note that the presence of pathogenic </span></p>\\n<p style=\"top:278.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">variants in genes not analyzed, genes with incomplete coverage, or regions not captured by filtering strategies cannot be fully excluded.</span><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#00637e\"> </span></b></p>\\n<p style=\"top:289.4pt;left:36.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:300.6pt;left:36.0pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\">  METHODOLOGY AND LIMITATIONS </span></b></p>\\n<p style=\"top:313.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#212121\">Genome sequence is generated from genomic DNA that is fragmented and barcoded. </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Library fragments were sequenced (2x150 base paired </span></p>\\n<p style=\"top:324.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">end) using Sequencing-By-Synthesis (SBS) chemistry and the Illumina NovaSeq sequencer with a minimum coverage of at least 20X for 90%. </span></p>\\n<p style=\"top:335.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Sequence data were aligned to the GRCh38 assembly after discarding low quality sequences. Illumina&apos;s DRAGEN (Dynamic Read Analysis for </span></p>\\n<p style=\"top:346.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">GENomics) platform was used for demultiplexing, read mapping, genome alignment, read sorting, duplicate marking, and variant calling. </span></p>\\n<p style=\"top:357.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Technical sensitivity of this assay is 99.10% (95% CI: 99.04-99.16%) and positive predictive value is 99.39% (95% CI: 99.37-99.41%). Variants in </span></p>\\n<p style=\"top:368.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">4,302 genes with some level of published evidence for a gene-disease association are subsequently filtered to identify: (1) variants classified as </span></p>\\n<p style=\"top:379.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">pathogenic or likely pathogenic in our internal database; (2) variants classified as pathogenic or likely pathogenic by a ClinGen-approved expert </span></p>\\n<p style=\"top:390.8pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">panel; (3) variants classified as disease causing in public databases that have a minor allele frequency &lt;5.0% in the Genome Aggregation </span></p>\\n<p style=\"top:401.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Database (gnomAD, https://gnomad.broadinstitute.org/); and (4) nonsense, frameshift, and +/-1,2 splice-site variants in disease-associated </span></p>\\n<p style=\"top:412.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">genes with a minor allele frequency </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">&#x2264;</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">0.1% in gnomAD. The evidence for phenotype-causality is then evaluated for each variant identified from </span></p>\\n<p style=\"top:423.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">the filtering strategies listed above and variants are classified based on ACMG/AMP criteria (Richards et al. 2015) with ClinGen rule </span></p>\\n<p style=\"top:434.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">specifications (https://www.clinicalgenome.org/working-groups/sequence-variant-interpretation/). Variants are reported according to HGVS </span></p>\\n<p style=\"top:445.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">nomenclature (http://varnomen.hgvs.org/). Only those variants with evidence for causing or contributing to disease are reported. All disease-</span></p>\\n<p style=\"top:456.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">associated variants on this report are confirmed via Sanger sequencing or another orthogonal technology. Please contact the laboratory for </span></p>\\n<p style=\"top:467.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">additional information. </span></p>\\n<p style=\"top:478.8pt;left:41.4pt;line-height:11.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:492.1pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Risk Alleles: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. The </span></p>\\n<p style=\"top:503.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">following likely or established risk alleles are examined and reported if identified in the &#x201c;reported genotype&#x201d; listed below. Some</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> variants in </span></p>\\n<p style=\"top:514.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">these genes are associated with additional diseases and therefore other variants identified in these genes may be included on different sections </span></p>\\n<p style=\"top:525.0pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">of this report. Additional risk variants, if identified, may also be included on this report at the discretion of the laboratory.</span><b><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:536.0pt;left:36.0pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:547.2pt;left:41.6pt;line-height:8.0pt\"><b><i><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Gene </span></i></b><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">(Transcript) </span></b></p>\\n<p style=\"top:547.2pt;left:126.9pt;line-height:8.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Associated Risk </span></b></p>\\n<p style=\"top:547.2pt;left:293.5pt;line-height:8.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Reportable Variants </span></b></p>\\n<p style=\"top:547.2pt;left:446.5pt;line-height:8.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Reportable Genotypes </span></b></p>\\n<p style=\"top:557.4pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">APC</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000038.4) </span></p>\\n<p style=\"top:557.4pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Colorectal Cancer </span></p>\\n<p style=\"top:557.4pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.3920T&gt;A (p.Ile1307Lys) </span></p>\\n<p style=\"top:557.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous </span></p>\\n<p style=\"top:570.7pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">APOE</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000041.2) </span></p>\\n<p style=\"top:570.7pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Alzheimer&#x2019;s </span><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Disease </span></p>\\n<p style=\"top:566.4pt;left:293.5pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">e4 Allele</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">- </span></p>\\n<p style=\"top:574.9pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.388T&gt;C (p.Cys130Arg) </span></p>\\n<p style=\"top:566.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous with e2 or e3 or </span></p>\\n<p style=\"top:574.9pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">homozygous (e2/e4, e3/e4, e4/e4) </span></p>\\n<p style=\"top:597.1pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">APOL1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_003661.3) </span></p>\\n<p style=\"top:597.1pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Non-diabetic Nephropathy </span></p>\\n<p style=\"top:584.1pt;left:293.5pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">G1 Allele</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">&#x2013;</span><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></p>\\n<p style=\"top:592.6pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.1164_1169delTTATAA (p.Asn388_Tyr389del) </span></p>\\n<p style=\"top:592.8pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Homozygous (G1/G1 or G2/G2) or </span></p>\\n<p style=\"top:601.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">compound heterozygous (G1/G2) </span></p>\\n<p style=\"top:601.6pt;left:293.5pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">G2 Allele</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> - </span></p>\\n<p style=\"top:610.1pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.[1024A&gt;G;1152T&gt;G] (p.[Ser342Gly;Ile384Met]) </span></p>\\n<p style=\"top:619.3pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">CHEK2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_001005735) </span></p>\\n<p style=\"top:619.3pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Breast, Colorectal, and Papillary Thyroid Cancers </span></p>\\n<p style=\"top:619.3pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.599T&gt;C (p.Ile200Thr) </span></p>\\n<p style=\"top:619.3pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:628.3pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">CTRC</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_007272.2) </span></p>\\n<p style=\"top:628.3pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Pancreatitis </span></p>\\n<p style=\"top:628.3pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.760C&gt;T (p.Arg254Trp) </span></p>\\n<p style=\"top:628.3pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:637.4pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">F2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000506.3) </span></p>\\n<p style=\"top:637.4pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Venous Thromboembolism </span></p>\\n<p style=\"top:637.4pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.*97G&gt;A </span></p>\\n<p style=\"top:637.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:646.4pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">F5</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000130.4) </span></p>\\n<p style=\"top:646.4pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Factor V Deficiency </span></p>\\n<p style=\"top:646.4pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.1601G&gt;A (p.Arg534Gln) </span></p>\\n<p style=\"top:646.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:655.4pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">GBA</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_001005741.2) </span></p>\\n<p style=\"top:655.4pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Parkinson&apos;s Disease </span></p>\\n<p style=\"top:655.4pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.1226A&gt;G (p.Asn409Ser) </span></p>\\n<p style=\"top:655.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:664.5pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">HFE</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000410.3) </span></p>\\n<p style=\"top:664.5pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Hemochromatosis </span></p>\\n<p style=\"top:664.5pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.845G&gt;A (p.Cys282Tyr) </span></p>\\n<p style=\"top:664.5pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Homozygous </span></p>\\n<p style=\"top:673.5pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">LRRK2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_198578.3) </span></p>\\n<p style=\"top:673.5pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Parkinson&apos;s Disease </span></p>\\n<p style=\"top:673.5pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.6055G&gt;A (p.Gly2019Ser) </span></p>\\n<p style=\"top:673.5pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:682.5pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">MC1R</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_002386) </span></p>\\n<p style=\"top:682.5pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Melanoma </span></p>\\n<p style=\"top:682.5pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.880G&gt;C (p.Asp294His) </span></p>\\n<p style=\"top:682.5pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:691.6pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">MITF</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000248.3) </span></p>\\n<p style=\"top:691.6pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Melanoma </span></p>\\n<p style=\"top:691.6pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.952G&gt;A (p.Glu318Lys) </span></p>\\n<p style=\"top:691.6pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:700.6pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">MUC5B</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_002458.2) </span></p>\\n<p style=\"top:700.6pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Pulmonary Fibrosis </span></p>\\n<p style=\"top:700.6pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.-3133G&gt;T </span></p>\\n<p style=\"top:700.6pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:709.7pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">PNPLA3</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_025225.2) </span></p>\\n<p style=\"top:709.7pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Non-alcoholic Fatty Liver Disease Type 1 </span></p>\\n<p style=\"top:709.7pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.444C&gt;G (p.Ile148Met) </span></p>\\n<p style=\"top:709.7pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Homozygous </span></p>\\n<p style=\"top:718.7pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">PRNP</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000311.3) </span></p>\\n<p style=\"top:718.7pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Prion Disease </span></p>\\n<p style=\"top:718.7pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.628G&gt;A (p.Val210Ile) </span></p>\\n<p style=\"top:718.7pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">5 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:91.9pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">SERPINA1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></p>\\n<p style=\"top:100.4pt;left:41.6pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">(NM_001127701.1) </span></p>\\n<p style=\"top:96.2pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Alpha-1 Antitrypsin Deficiency </span></p>\\n<p style=\"top:81.2pt;left:293.5pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">S Allele</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">- </span></p>\\n<p style=\"top:89.7pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.863A&gt;T (p.Glu288Val) </span></p>\\n<p style=\"top:91.9pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Homozygous Z allele (Z/Z) or compound </span></p>\\n<p style=\"top:100.4pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">heterozygous Z and S allele (Z/S) </span></p>\\n<p style=\"top:102.7pt;left:293.5pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Z Allele</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">- </span></p>\\n<p style=\"top:111.2pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.1096G&gt;A (p.Glu366Lys) </span></p>\\n<p style=\"top:122.3pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">SERPINC1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_000488) </span></p>\\n<p style=\"top:122.3pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Venous Thromboembolism </span></p>\\n<p style=\"top:122.3pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.1246G&gt;T (p.Ala416Ser) </span></p>\\n<p style=\"top:122.3pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:131.3pt;left:41.6pt;line-height:7.0pt\"><i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">SPINK1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> (NM_003122) </span></p>\\n<p style=\"top:131.3pt;left:126.9pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Pancreatitis </span></p>\\n<p style=\"top:131.3pt;left:293.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">c.101A&gt;G (p.Asn34Ser) </span></p>\\n<p style=\"top:131.3pt;left:446.5pt;line-height:7.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\">Heterozygous or homozygous</span><b><span style=\"font-family:Calibri,sans-serif;font-size:7.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:140.7pt;left:36.0pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:151.7pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">PGx: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Genotype calls for specific genomic positions are identified using the Genomic Analysis Tool Kit (GATK) and a custom script. Diplotypes and </span></p>\\n<p style=\"top:162.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">phenotypes are generated using the Clinical Pharmacogenetics Implementation Consortium (CPIC&#xae;) allele tables </span></p>\\n<p style=\"top:173.7pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://cpicpgx.org/guidelines/</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">) and custom scripts. The following pharmacogenomic variants are detected by this assay: </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C19</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:184.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_000769.1): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs4244285/c.681G&gt;A, rs12769205/c.332-23A&gt;G), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs4986893/c.636G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*4A</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs28399504/c.1A&gt;G), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*4B</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:195.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs28399504/c.1A&gt;G, rs12248560/c.-806C&gt;T), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*5</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs56337013/c.1297C&gt;T), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*6</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs72552267/c.395G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*7</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs72558186/c.819+2T&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*8</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:206.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs41291556/c.358T&gt;C), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*17</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs12248560/c.-806C&gt;T), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*35</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs12769205/c.332-23A&gt;G); </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP2C9 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_000771.3): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs1799853/c.430C&gt;T), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:217.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs1057910/c.1075A&gt;C), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*5</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs28371686/c.1080C&gt;G), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*6</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs9332131/c.818del), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*8</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs7900194/c.449G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*11</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs28371685/c.1003C&gt;T), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*13</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:228.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs72558187/c.269T&gt;C); </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP3A5 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_000777.4): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs776746/c.219-237A&gt;G), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*6</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs10264272/c.624G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*7</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs41303343/c.1035dup); </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CYP4F2 </span></i></p>\\n<p style=\"top:239.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_001082.4): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs2108622/c.1297G&gt;A); </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">DPYD </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_000110.3): rs3918290/c.1905+1G&gt;A, rs55886062/c.1679T&gt;G, rs67376798/c.2846A&gt;T, </span></p>\\n<p style=\"top:250.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">rs72549309/c.299_302del/legacy name c.295_299TCATT&gt;T, rs115232898/c.557A&gt;G, rs1801266/c.703C&gt;T, rs78060119/c.1156G&gt;T, </span></p>\\n<p style=\"top:261.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">rs56038477/c.1236G&gt;A, rs72549303/c.1898del, rs1801268/c.2983G&gt;T, rs75017182/c.1129-5923C&gt;G;</span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\"> </span></i><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">IFNL3 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_172139.3): rs12979860/c.-</span></p>\\n<p style=\"top:272.6pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">3180G&gt;A; </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">NUDT15 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_018283.3): rs116855232/c.415C&gt;T; </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">SLCO1B1 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_006446.4): rs4149056/c.521T&gt;C; </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">TPMT </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_000367.4): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*2</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:283.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs1800462/c.238G&gt;C), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3A</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs1800460/c.460G&gt;A, rs1142345/c.719A&gt;G), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3B</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs1800460/c.460G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*3C</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs1142345/c.719A&gt;G),  </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*4</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:294.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs1800584/c.626-1G&gt;A);</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\"> </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">VKORC1 </span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(NM_024006.4): rs9923231/c.-1639G&gt;A;</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#808080\">  </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">UGT1A1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (NM_000463.3): </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*6</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs4148323/c.211G&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*27</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:305.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(rs35350960/c.686C&gt;A), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*36</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs3064744/c.-53TA[6]), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*28</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs3064744/ c.-53TA[8]), </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*37</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> (rs3064744/ c.-53TA[9]). Variant and star (*) allele </span></p>\\n<p style=\"top:316.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">definitions are available for download at </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://personalizedmedicine.partners.org/Laboratory-For-Molecular-</span></p>\\n<p style=\"top:327.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">Medicine/Tests/Genomic%20Screening%20Spray.aspx</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(see the link &#x201c;details here&#x201d; in the Methodology section of this webpage).  Additionally, </span></p>\\n<p style=\"top:338.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">variants in </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">G6PD</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">, </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">RYR1</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">, and </span><i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">CACNA1S</span></i><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> associated with an altered metabolism status are reported, if identified. Detected variants include those </span></p>\\n<p style=\"top:349.5pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">that are clinically relevant based on allele function and frequency information found in </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#222222\">tables created by PharmGKB and CPIC</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:360.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.pharmgkb.org/page/pgxGeneRef</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">). This test does not report all pharmacogenomic variants that might alter protein function. A </span></p>\\n<p style=\"top:371.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">*1/*1 or wild type result is assigned if no variants that are included in the test are detected. Therefore, a *1/*1 or other result does not exclude </span></p>\\n<p style=\"top:382.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">the possibility that an individual has a different phenotype that may alter drug response. This risk may vary among ancestries. This assay cannot </span></p>\\n<p style=\"top:393.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">determine if multiple variants in the same gene are present in cis or trans, leading to an inability to definitively assign a diplotype and </span></p>\\n<p style=\"top:404.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">phenotype. This test does not detect copy number variants. Genetic information to guide prescribing decisions may be found in a drug&apos;s FDA-</span></p>\\n<p style=\"top:415.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">approved label. These labels are found at </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">  and a table of current PGx biomarkers at </span></p>\\n<p style=\"top:426.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> and </span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">https://www.fda.gov/drugs/science-and-</span></p>\\n<p style=\"top:437.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#0000ff\">research-drugs/table-pharmacogenomic-biomarkers-drug-labeling</span><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">. </span></p>\\n<p style=\"top:448.4pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:459.3pt;left:41.4pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Limitations:</span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> Specific types of genetic variation, such as short tandem repeats (including triplet repeat expansions), structural variation, and large </span></p>\\n<p style=\"top:470.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">copy number events are currently not reliably detected by this assay. Additionally, while genome sequencing covers ~95% of the genome; there </span></p>\\n<p style=\"top:481.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">are certain regions for which the assay may fail to adequately generate sequence information, such as regions of high homology or regions that </span></p>\\n<p style=\"top:492.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">are highly repetitive as well as regions with low coverage, mapping quality or base quality. Moreover, not all disease-associated genes have </span></p>\\n<p style=\"top:503.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">been identified and the clinical significance of variation in many genes is not well understood. The sequencing data is limited to the cell-type </span></p>\\n<p style=\"top:514.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">(material) source used for DNA extraction. Additionally, not all variants identified have been analyzed. Variant interpretation may change over </span></p>\\n<p style=\"top:525.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">time if more information becomes available. </span></p>\\n<p style=\"top:536.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:547.3pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">The initial sequencing component of this test was performed by the Clinical Research Sequencing Platform of the Broad Institute (320 Charles St, </span></p>\\n<p style=\"top:558.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Cambridge, MA 02141; CLIA#22D2055652), and the Sanger confirmation, interpretive algorithms and clinical reports were generated by the </span></p>\\n<p style=\"top:569.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Laboratory for Molecular Medicine at Partners Healthcare Personalized Medicine (LMM, 65 Landsdowne St, Cambridge, MA 02139; 617-768-</span></p>\\n<p style=\"top:580.1pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">8500; CLIA#22D1005307). This test has not been cleared or approved by the U.S. Food and Drug Administration (FDA). The FDA has determined </span></p>\\n<p style=\"top:591.2pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">that such clearance or approval is not necessary. </span></p>\\n<p style=\"top:602.1pt;left:41.4pt;line-height:9.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></p>\\n<p style=\"top:612.9pt;left:40.6pt;line-height:8.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">REFERENCES </span></b></p>\\n<p style=\"top:622.7pt;left:40.6pt;line-height:8.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:632.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Alfardan J, Mohsen AW, Copeland S, Ellison J, Keppen-Davis L, Rohrbach M, Powell BR, Gillis J, Matern D, Kant J, Vockley J. 2010. Characterization of new ACADSB gene </span></p>\\n<p style=\"top:642.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">sequence mutations and clinical implications in patients with 2- </span></p>\\n<p style=\"top:652.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">methylbutyrylglycinuria identified by newborn screening. Mol Genet Metab. 100(4):333- 8. PMID: 20547083. </span></p>\\n<p style=\"top:661.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:671.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Amstutz U, Henricks LM, Offer SM, Barbarino J, Schellens JHM, Swen JJ, Klein TE, McLeod HL, Caudle KE, Diasio RB, Schwab M. 2018. Clinical Pharmacogenetics </span></p>\\n<p style=\"top:681.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Implementation Consortium (CPIC) Guideline for Dihydropyrimidine Dehydrogenase </span></p>\\n<p style=\"top:691.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Genotype and Fluoropyrimidine Dosing: 2017 Update. Clin. Pharmacol. Ther. 103(2):210-216. PMID: 29152729. </span></p>\\n<p style=\"top:700.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">6 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:78.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Antony D, Becker-Heck A, Zariwala MA, Schmidts M, Onoufriadis A, Forouhan M, Wilson R, Taylor-Cox T, Dewar A, Jackson C, Goggin P, Loges NT, Olbrich H, Jaspers </span></p>\\n<p style=\"top:88.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">M, Jorissen M, Leigh MW, Wolf WE, Daniels ML, Noone PG, Ferkol TW, Sagel SD, Rosenfeld M, Rutman A, Dixit A, O&apos;Callaghan C, Lucas JS, Hogg C, Scambler PJ, Emes </span></p>\\n<p style=\"top:98.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">RD, Uk10k, Chung EM, </span></p>\\n<p style=\"top:108.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:117.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Shoemark A, Knowles MR, Omran H, Mitchison HM. 2013. Mutations in CCDC39 and CCDC40 are the major cause of primary ciliary dyskinesia with axonemal </span></p>\\n<p style=\"top:127.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">disorganization and absent inner dynein arms. Hum. Mutat. 34(3):462-72. PMID: 23255504. </span></p>\\n<p style=\"top:137.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:147.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Becker-Heck A, Zohn IE, Okabe N, Pollock A, Lenhart KB, Sullivan-Brown J, McSheene J, Loges NT, Olbrich H, Haeffner K,  </span></p>\\n<p style=\"top:156.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Fliegauf M, Horvath J, Reinhardt R, Nielsen KG, Marthin JK, Baktai G, Anderson KV, Geisler R, Niswander L, Omran H, Burdine RD. 2011. The coiled-coil domain </span></p>\\n<p style=\"top:166.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">containing protein CCDC40 is essential for motile cilia function and left-right axis formation. Nat. Genet. 43(1):79-84. PMID: 21131974. </span></p>\\n<p style=\"top:176.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:186.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Birdwell KA, Decker B, Barbarino JM, Peterson JF, Stein CM, Sadee W, Wang D, Vinks AA, He Y, Swen JJ, Leeder JS, van Schaik R, Thummel KE, Klein TE, Caudle KE, </span></p>\\n<p style=\"top:195.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">MacPhee IA. 2015. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP3A5 Genotype and Tacrolimus Dosing. Clin. Pharmacol. Ther. </span></p>\\n<p style=\"top:205.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">98(1):19-24. PMID: 25801146. </span></p>\\n<p style=\"top:215.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:225.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Caudle KE, Dunnenberger HM, Freimuth RR, Peterson JF, Burlison JD, Whirl-Carrillo M, Scott SA, Rehm HL, Williams MS, Klein TE, Relling MV, Hoffman JM. 2017. </span></p>\\n<p style=\"top:235.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Standardizingterms for clinical pharmacogenetic test results: consensus terms from the Clinical Pharmacogenetics Implementation Consortium (CPIC). Genet. Med. </span></p>\\n<p style=\"top:244.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">19(2):215-223. PMID: 27441996. </span></p>\\n<p style=\"top:254.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:264.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Caudle KE, Rettie AE, Whirl-Carrillo M, Smith LH, Mintzer S, Lee MT, Klein TE, Callaghan JT, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical </span></p>\\n<p style=\"top:274.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">pharmacogenetics implementation consortium guidelines for CYP2C9 and HLA-B genotypes and phenytoin dosing. Clin. Pharmacol. Ther. 96(5):542-8. PMID: </span></p>\\n<p style=\"top:283.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">25099164. </span></p>\\n<p style=\"top:293.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:303.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, Schwab M. 2013. Clinical Pharmacogenetics Implementation Consortium guidelines for dihydropyrimidine </span></p>\\n<p style=\"top:313.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">dehydrogenase genotype and fluoropyrimidine dosing. Clin. Pharmacol. Ther. 94(6):640-5. PMID: 23988873. </span></p>\\n<p style=\"top:322.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:332.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Crews KR, Gaedigk A, Dunnenberger HM, Leeder JS, Klein TE, Caudle KE, Haidar CE, Shen DD, Callaghan JT, Sadhasivam S, Prows CA, Kharasch ED, Skaar TC, Clinical </span></p>\\n<p style=\"top:342.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacogenetics Implementation Consortium. 2014. Clinical Pharmacogenetics </span></p>\\n<p style=\"top:352.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Implementation Consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin. Pharmacol. Ther. 95(4):376-82. PMID: 24458010. </span></p>\\n<p style=\"top:362.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:371.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Gammal RS, Court MH, Haidar CE, Iwuchukwu OF, Gaur AH, Alvarellos M, Guillemette C, Lennox JL, Whirl-Carrillo M, Brummel SS, Ratain MJ, Klein TE, Schackman BR, </span></p>\\n<p style=\"top:381.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Caudle KE, Haas DW, Clinical Pharmacogenetics Implementation Consortium. 2016. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for </span></p>\\n<p style=\"top:391.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">UGT1A1 and Atazanavir </span></p>\\n<p style=\"top:401.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Prescribing. Clin Pharmacol Ther. 99(4):363-9. PMID: 26417955. </span></p>\\n<p style=\"top:410.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:420.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Gonsalves SG, Dirksen RT, Sangkuhl K, Pulk R, Alvarellos M, Vo T, Hikino K, Roden D, Klein TE, Poler SM, Patel S, Caudle KE, Gordon R, Brandom B, Biesecker LG. 2019. </span></p>\\n<p style=\"top:430.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for the Use of Potent Volatile Anesthetic Agents and Succinylcholine in the Context of RYR1 </span></p>\\n<p style=\"top:440.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">or CACNA1S Genotypes. Clin. Pharmacol. Ther. 105(6):1338-1344. PMID: 30499100. </span></p>\\n<p style=\"top:449.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:459.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Hicks JK, Bishop JR, Sangkuhl K, M&#xfc;ller DJ, Ji Y, Leckband SG, Leeder JS, Graham RL, Chiulli DL, LLerena A, Skaar TC, Scott SA, Stingl JC, Klein TE, Caudle KE, Gaedigk A, </span></p>\\n<p style=\"top:469.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Clinical Pharmacogenetics Implementation Consortium. 2015. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6 and CYP2C19 </span></p>\\n<p style=\"top:479.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Genotypes and Dosing </span></p>\\n<p style=\"top:489.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">of Selective Serotonin Reuptake Inhibitors. Clin. Pharmacol. Ther. 98(2):127-34. PMID: 25974703. </span></p>\\n<p style=\"top:498.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:508.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Hicks JK, Sangkuhl K, Swen JJ, Ellingrod VL, M&#xfc;ller DJ, Shimoda K, Bishop JR, Kharasch ED, Skaar TC, Gaedigk A, Dunnenberger HM, Klein TE, Caudle KE, Stingl JC. 2017. </span></p>\\n<p style=\"top:518.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. </span></p>\\n<p style=\"top:528.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacol. Ther. 102(1):37-44. PMID: 27997040. </span></p>\\n<p style=\"top:537.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:547.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Johnson JA, Caudle KE, Gong L, Whirl-Carrillo M, Stein CM, Scott SA, Lee MT, Gage BF, Kimmel SE, Perera MA, Anderson JL, Pirmohamed M, Klein TE, Limdi NA, Cavallari </span></p>\\n<p style=\"top:557.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">LH, Wadelius M. 2017. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for Pharmacogenetics-Guided Warfarin Dosing: 2017 Update. Clin. </span></p>\\n<p style=\"top:567.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacol. Ther. 102 </span></p>\\n<p style=\"top:576.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">(3):397-404. PMID: 28198005. </span></p>\\n<p style=\"top:586.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:596.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Knowles MR, Leigh MW, Carson JL, Davis SD, Dell SD, Ferkol TW, Olivier KN, Sagel SD, Rosenfeld M, Burns KA, Minnix SL, Armstrong MC, Lori A, Hazucha MJ, Loges NT, </span></p>\\n<p style=\"top:606.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Olbrich H, Becker-Heck A, Schmidts M, Werner C, Omran H, Zariwala MA, Genetic Disorders of Mucociliary Clearance Consortium. 2012. Mutations of DNAH11 in </span></p>\\n<p style=\"top:616.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">patients with primary ciliary dyskinesia with normal ciliary ultrastructure. Thorax. 67(5):433-41. PMID: 22184204. </span></p>\\n<p style=\"top:625.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:635.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Lima JJ, Thomas CD, Barbarino J, Desta Z, Van Driest SL, El Rouby N, Johnson JA, Cavallari LH, Shakhnovich V, Thacker DL, Scott SA, Schwab M, Uppugunduri CRS, </span></p>\\n<p style=\"top:645.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Formea CM, Franciosi JP, Sangkuhl K, Gaedigk A, Klein TE, Gammal RS, Furuta T. 2021. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for </span></p>\\n<p style=\"top:655.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">CYP2C19 and Proton Pump Inhibitor Dosing. Clin Pharmacol Ther. 109(6):1417-1423. PMID: 32770672. </span></p>\\n<p style=\"top:664.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:674.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Monnier N, Procaccio V, Stieglitz P, Lunardi J. 1997. Malignant-hyperthermia susceptibility is associated with a mutation of the alpha 1-subunit of the human </span></p>\\n<p style=\"top:684.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">dihydropyridine-sensitive L-type voltage-dependent calcium-channel receptor in </span></p>\\n<p style=\"top:694.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">skeletal muscle. Am. J. Hum. Genet. 60(6):1316-25. PMID: 9199552.  </span></p>\\n<p style=\"top:703.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">7 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:78.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Moriyama B, Obeng AO, Barbarino J, Penzak SR, Henning SA, Scott SA, Ag&#xfa;ndez J, Wingard JR, McLeod HL, Klein TE, Cross SJ, Caudle KE, Walsh TJ. 2017. Clinical </span></p>\\n<p style=\"top:88.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP2C19 and </span></p>\\n<p style=\"top:98.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Voriconazole Therapy. Clin. Pharmacol. Ther. 102(1):45-51. PMID: 27981572. </span></p>\\n<p style=\"top:108.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:117.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Muir AJ, Gong L, Johnson SG, Lee MT, Williams MS, Klein TE, Caudle KE, Nelson DR, Clinical Pharmacogenetics Implementation Consortium (CPIC). 2014. Clinical </span></p>\\n<p style=\"top:127.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacogenetics Implementation Consortium (CPIC) guidelines for IFNL3 (IL28B) genotype and PEG interferon--based regimens. Clin. Pharmacol. Ther. 95(2):141-6. </span></p>\\n<p style=\"top:137.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">PMID: 24096968. </span></p>\\n<p style=\"top:147.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:156.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Nakhleh N, Francis R, Giese RA, Tian X, Li Y, Zariwala MA, Yagi H, Khalifa O, Kureshi S, Chatterjee B, Sabol SL, Swisher M, Connelly PS, Daniels MP, Srinivasan A, Kuehl </span></p>\\n<p style=\"top:166.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">K, Kravitz N, Burns K, Sami I, Omran H, Barmada M, Olivier K, Chawla KK, Leigh M, Jonas R, Knowles M, Leatherbury L, Lo CW. 2012. High prevalence of respiratory </span></p>\\n<p style=\"top:176.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">ciliary dysfunction in congenital heart disease patients with heterotaxy. Circulation. 125(18): 2232-42. PMID: 22499950. </span></p>\\n<p style=\"top:186.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:195.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Porta F, Chiesa N, Martinelli D, Spada M. 2019. Clinical, biochemical, and molecular spectrum of short/branched-chain acyl-CoA dehydrogenase deficiency: two new </span></p>\\n<p style=\"top:205.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">cases and review of literature. J Pediatr Endocrinol Metab. 32(2):101-108. PMID: 30730842. </span></p>\\n<p style=\"top:215.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:225.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Ramsey LB, Johnson SG, Caudle KE, Haidar CE, Voora D, Wilke RA, Maxwell WD, McLeod HL, Krauss RM, Roden DM, Feng Q, Cooper-DeHoff RM, Gong L, Klein TE, </span></p>\\n<p style=\"top:235.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Wadelius M, Niemi M. 2014. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced myopathy: 2014 update. Clin. </span></p>\\n<p style=\"top:244.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacol. Ther. 96(4):423-8. </span></p>\\n<p style=\"top:254.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">PMID: 24918167. </span></p>\\n<p style=\"top:264.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:274.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Relling MV, Gardner EE, Sandborn WJ, Schmiegelow K, Pui CH, Yee SW, Stein CM, Carrillo M, Evans WE, Hicks JK, Schwab M, Klein TE, Clinical Pharmacogenetics </span></p>\\n<p style=\"top:283.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Implementation Consortium. 2013. Clinical pharmacogenetics implementation consortium guidelines for thiopurine methyltransferase genotype and thiopurine </span></p>\\n<p style=\"top:293.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">dosing: 2013 update. Clin. Pharmacol. Ther. 93(4):324-5. PMID: 23422873. </span></p>\\n<p style=\"top:303.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:313.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Relling MV, McDonagh EM, Chang T, Caudle KE, McLeod HL, Haidar CE, Klein T, Luzzatto L, Clinical Pharmacogenetics Implementation Consortium. 2014. Clinical </span></p>\\n<p style=\"top:322.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Pharmacogenetics Implementation Consortium (CPIC) guidelines for rasburicase therapy in the context of G6PD deficiency genotype. Clin. Pharmacol. Ther. 96(2):169-</span></p>\\n<p style=\"top:332.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">74. PMID: 24787449. </span></p>\\n<p style=\"top:342.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:352.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Relling MV, Schwab M, Whirl-Carrillo M, Suarez-Kurtz G, Pui CH, Stein CM, Moyer AM, Evans WE, Klein TE, Antillon-Klussmann FG, Caudle KE, Kato M, Yeoh AEJ, </span></p>\\n<p style=\"top:362.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Schmiegelow K, Yang JJ. 2019. Clinical Pharmacogenetics Implementation Consortium Guideline for Thiopurine Dosing Based on TPMT and NUDT15 Genotypes: 2018 </span></p>\\n<p style=\"top:371.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Update. Clin. Pharmacol. </span></p>\\n<p style=\"top:381.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Ther. 105(5):1095-1105. PMID: 30447069. </span></p>\\n<p style=\"top:391.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:401.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Schwabe GC, Hoffmann K, Loges NT, Birker D, Rossier C, de Santi MM, Olbrich H, Fliegauf M, Failly M, Liebers U, Collura M, Gaedicke G, Mundlos S, Wahn U, Blouin JL, </span></p>\\n<p style=\"top:410.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Niggemann B, Omran H, Antonarakis SE, Bartoloni L. 2008. Primary ciliary dyskinesia associated with normal axoneme ultrastructure is caused by DNAH11 mutations. </span></p>\\n<p style=\"top:420.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Hum Mutat. 29(2):289-98. PMID: 18022865. </span></p>\\n<p style=\"top:430.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:440.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Scott SA, Sangkuhl K, Stein CM, Hulot JS, Mega JL, Roden DM, Klein TE, Sabatine MS, Johnson JA, Shuldiner AR, Clinical Pharmacogenetics Implementation Consortium. </span></p>\\n<p style=\"top:449.9pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">2013. Clinical Pharmacogenetics Implementation Consortium guidelines for CYP2C19 genotype and clopidogrel therapy: 2013 update. Clin. Pharmacol. Ther. 94(3):317-</span></p>\\n<p style=\"top:459.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">23. PMID: 23698643. </span></p>\\n<p style=\"top:469.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:479.2pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Spedicati B, Cocca M, Palmisano R, Faletra F, Barbieri C, Francescatto M, Mezzavilla M, Morgan A, Pelliccione G, Gasparini P, Girotto G. 2021. Natural human knockouts </span></p>\\n<p style=\"top:489.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">and Mendelian disorders: deep phenotyping in Italian isolates. Eur J Hum Genet. 29(8):1272- 1281. PMID: 33727708. </span></p>\\n<p style=\"top:498.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:508.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Stewart SL, Hogan K, Rosenberg H, Fletcher JE. 2001. Identification of the Arg1086His mutation in the alpha subunit of the voltage-dependent calcium channel </span></p>\\n<p style=\"top:518.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">(CACNA1S) in a North American family with malignant hyperthermia. Clin. Genet. 59(3):178-84. PMID: 11260227. </span></p>\\n<p style=\"top:528.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:537.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Swen JJ, Nijenhuis M, de Boer A, Grandia L, Maitland-van der Zee AH, Mulder H, Rongen GA, van Schaik RH, Schalekamp T, Touw DJ, van der Weide J, Wilffert B, Deneer </span></p>\\n<p style=\"top:547.6pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">VH, Guchelaar HJ. 2011. Pharmacogenetics: from bench to byte--an update of guidelines. Clin. Pharmacol. Ther. 89(5):662-73. PMID: 21412232. </span></p>\\n<p style=\"top:557.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:567.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Theken KN, Lee CR, Gong L, Caudle KE, Formea CM, Gaedigk A, Klein TE, Ag&#xfa;ndez JAG, Grosser T. 2020. Clinical Pharmacogenetics Implementation Consortium </span></p>\\n<p style=\"top:576.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Guideline (CPIC) for CYP2C9 and Nonsteroidal Anti-Inflammatory Drugs. Clin Pharmacol Ther. 108(2):191- 200. PMID: 32189324. </span></p>\\n<p style=\"top:586.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:596.4pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Wilke RA, Ramsey LB, Johnson SG, Maxwell WD, McLeod HL, Voora D, Krauss RM, Roden DM, Feng Q, Cooper-Dehoff RM, Gong L, Klein TE, Wadelius M, Niemi M, </span></p>\\n<p style=\"top:606.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Clinical Pharmacogenomics Implementation Consortium (CPIC). 2012. The clinical pharmacogenomics implementation consortium: CPIC guideline for SLCO1B1 and </span></p>\\n<p style=\"top:616.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">simvastatin-induced myopathy. Clin. Pharmacol. Ther. 92(1):112-7. PMID: 22617227. </span></p>\\n<p style=\"top:625.7pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:635.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Witherspoon JW, Meilleur KG. 2016. Review of RyR1 pathway and associated pathomechanisms. Acta Neuropathol Commun. 4(1):121. PMID: 27855725. </span></p>\\n<p style=\"top:645.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:655.0pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Yang SK, Hong M, Baek J, Choi H, Zhao W, Jung Y, Haritunians T, Ye BD, Kim KJ, Park SH, Park SK, Yang DH, Dubinsky M, Lee I, McGovern DP, Liu J, Song K. 2014. A </span></p>\\n<p style=\"top:664.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat. Genet. 46(9):1017-20. PMID: 25108385. </span></p>\\n<p style=\"top:674.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:684.3pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">Zariwala MA, Gee HY, Kurkowiak M, Al-Mutairi DA, Leigh MW, Hurd TW, Hjeij R, Dell SD, Chaki M, Dougherty GW, Adan M, Spear PC, Esteve-Rudd J, Loges NT, Rosenfeld </span></p>\\n<p style=\"top:694.1pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">M, Diaz KA, Olbrich H, Wolf WE, Sheridan E, Batten TF, Halbritter J, Porath JD, Kohl S, Lovric S, Hwang DY, Pittman JE, Burns KA, Ferkol TW, Sagel SD, Olivier KN, Morgan </span></p>\\n<p style=\"top:703.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">LC, Werner C, Raidt J, Pennekamp P, Sun Z, Zhou W, Airik R, Natarajan S, Allen SJ, Amirav I, Wieczorek D, Landwehr K, Nielsen K, Schwerk N, Sertic J, K&#xf6;hler G, Washburn </span></p>\\n</div>\\n'\n",
      "b'<div id=\"page0\" style=\"width:612.0pt;height:792.0pt\">\\n<p style=\"top:30.4pt;left:460.3pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Accession ID: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">PM-23-NXXXX </span></b></p>\\n<p style=\"top:42.6pt;left:499.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Name: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">LAST, FIRST</span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\"> </span></b></p>\\n<p style=\"top:54.8pt;left:527.9pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#808080\">Page: </span></b><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\">8 of 8 </span></b></p>\\n<p style=\"top:67.0pt;left:576.1pt;line-height:10.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:10.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:729.7pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">______________________________________________________________________________________________________________________________________________________________________________________ </span></p>\\n<p style=\"top:739.1pt;left:36.0pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">Laboratory for Molecular Medicine                                                                                                                                                                Matthew S. Lebo, Ph.D., FACMG, Director </span></p>\\n<p style=\"top:750.1pt;left:36.0pt;line-height:6.0pt\"><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#0000ff\">https://www.massgeneralbrigham.org/en/research-and-innovation/centers-and-programs\\\\personalized-medicine\\\\molecular-medicine</span><span style=\"font-family:Cambria,serif;font-size:6.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:468.1pt;line-height:8.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:8.0pt;color:#000000\"> </span></p>\\n<p style=\"top:748.5pt;left:504.1pt;line-height:8.0pt\"><span style=\"font-family:Cambria,serif;font-size:8.0pt;color:#000000\">CLIA#: 22D1005307</span><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:758.3pt;left:36.0pt;line-height:12.0pt\"><span style=\"font-family:Times New Roman,serif;font-size:12.0pt;color:#000000\"> </span></p>\\n<p style=\"top:29.2pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#29a6a3\">Personalized Medicine </span></b></p>\\n<p style=\"top:42.4pt;left:37.7pt;line-height:8.0pt\"><b><span style=\"font-family:Arial,sans-serif;font-size:8.0pt;color:#2e74b5\">LABORATORY FOR MOLECULAR MEDICINE </span></b></p>\\n<p style=\"top:52.5pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">65 Landsdowne St, Cambridge, MA  02139 </span></p>\\n<p style=\"top:60.6pt;left:37.7pt;line-height:7.0pt\"><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">Phone: (617) 768-</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">8500 &#x2022; Fax: (617) 768</span><span style=\"font-family:Arial,sans-serif;font-size:7.0pt;color:#808080\">-8513 </span></p>\\n<p style=\"top:78.8pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">J, Levy S, Fan S, Koerner-Rettberg C, Amselem S, Williams DS, Mitchell BJ, Drummond IA, Otto EA, Omran H, Knowles MR, Hildebrandt F. 2013. ZMYND10 is mutated </span></p>\\n<p style=\"top:88.5pt;left:40.6pt;line-height:8.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:8.0pt;color:#000000\">in primary ciliary dyskinesia and interacts with LRRC6. Am. J. Hum. Genet. 93(2):336-45. PMID: 23891469. </span></p>\\n<p style=\"top:98.8pt;left:40.6pt;line-height:11.0pt\"><span style=\"font-family:Calibri,sans-serif;font-size:11.0pt;color:#000000\"> </span></p>\\n<p style=\"top:112.0pt;left:40.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:123.0pt;left:40.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Report Preparation by: </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">First Last</span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n<p style=\"top:133.9pt;left:40.6pt;line-height:9.0pt\"><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">Final Report by:  </span></b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\">First Last, PhD, FACMG</span><b><span style=\"font-family:Calibri,sans-serif;font-size:9.0pt;color:#000000\"> </span></b></p>\\n</div>\\n'\n"
     ]
    }
   ],
   "execution_count": 25
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
